<!DOCTYPE HTML PUBLIC "-//W3C//DTD XHTML 1.0 Transitional//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-transitional.dtd">
<html lang="en-US" xmlns="http://www.w3.org/1999/xhtml" xml:lang="en-US">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<!-- BEGIN: ftx_abs.tpl -->

<!-- BEGIN: journal-article.html-main.tpl -->



	<title>Wiley InterScience :: Article Full Text HTML</title>
	

	
	
	
	

 	
	
	<!-- JavaScript for Linking mechanism -->
	
	
	<!-- JavaScript for Embedded images -->
	


<link rel="stylesheet" type="text/css" href="index_4.css" media="all">
</head>
<body id="wileyinterscience" bgcolor="#ffffff">
<a id="pageTop" name="pageTop"></a>
<a id="top" name="top"></a>

<div id="contentFrame" class="journalArticleMain">
	<!-- BEGIN: main column -->
	<div class="mainCol">

		<!-- BEGIN: intro area -->
		<div id="intro" class="area" style="padding-left: 10px ! important;">

			<div class="item">
				<table border="0" cellpadding="0" cellspacing="0">
					<tbody><tr valign="top">
						<td width="100%">

							<!-- BEGIN: branding area -->
							<div class="banner">
								
								
								
								
<!-- BEGIN: product banner  -->

<table style="background-image: url(bannerbackground.gif); height: 58px;" width="100%" bgcolor="#eeeeee" border="0" cellpadding="0" cellspacing="0">
	<tbody><tr>
	<td><img src="bannerforeground.gif" alt="" border="0"></td>
	</tr>
</tbody></table>
<!-- END: product banner  -->


							</div>
							<!-- END: branding area -->

							<table border="0" cellpadding="0" cellspacing="0">
								<tbody><tr valign="top">
									<td width="100%">
										<!-- BEGIN: item info area -->
										<div class="info">
											<h2><a href="http://www3.interscience.wiley.com.ejproxy.a-star.edu.sg/journal/33945/home" target="_top">Cytometry Part A</a></h2>
											<p style="margin-top: 0.6em;">
										
											
												<a href="http://www3.interscience.wiley.com.ejproxy.a-star.edu.sg/rss/journal/33945">
													<img src="rss_feed.gif" alt="RSS feed for Cytometry Part A" title="RSS feed for Cytometry Part A" align="absbottom" border="0">
												</a> 
												<span class="text"><a href="http://www3.interscience.wiley.com.ejproxy.a-star.edu.sg/cgi-bin/help?page=HELP_syndication#RSS">What is RSS?</a></span>
											
										
											</p>
											
												<h6><a href="http://www3.interscience.wiley.com.ejproxy.a-star.edu.sg/journal/106592573/issue" target="_top"><strong>Early View</strong></a>
												&nbsp;&nbsp;(Articles online in advance of print)</h6>
											
											
											<p><strong>Published Online:</strong> 18 Jul 2008</p>

											

											


											
											  <p>Copyright Â© 2008 International Society for Analytical Cytology</p>
											
																						<div style="padding-top: 10px;"><a href="http://www3.interscience.wiley.com.ejproxy.a-star.edu.sg/journal/33945/home" title="Go to the homepage for this journal to access trials, sample copies, editorial and author information, news, and more." target="_top"><img src="link.journal.homepage.gif" alt="Journal Homepage Link" width="270" border="0" height="46"></a></div>
										</div>
										<!-- END: item info area -->
									</td>

									<td width="100">
										<!-- email / print tools -->
										<div class="tools"><img src="icons.email-print.gif" alt="E-Mail and Print Toolbar" usemap="#map_email-print" width="90" border="0" height="20"><map name="map_email-print" id="map_email-print"><area shape="rect" coords="2,4,44,16" href="javascript:mailTool(120839840,%20'',%20'');" id="email" alt="E-Mail" title="E-Mail This Page"><area shape="rect" coords="48,4,89,16" href="javascript:print();" id="print" alt="Print" title="Print This Page"></map></div>

										<!-- published for the society of ... -->
										<div class="published">
											
										</div>
									</td>
								</tr>
							</tbody></table>
						</td>
						<td>
							<div class="sideCol">
								<!-- BEGIN: search-combined.tpl -->
<table border="0" cellpadding="0" cellspacing="0">
	<!-- Show ADS Search -->
	

	<tbody><tr>
		<td class="BGpowderblue">
			<div id="searchBox">
				<div id="searchTitles" class="item">
					<form action="http://www3.interscience.wiley.com.ejproxy.a-star.edu.sg/journal/titlesearch" method="post" id="searchTitles" name="searchTitles" target="_top">
						<fieldset>
							<!-- NOTE: this span must be here or else ie/win adds left padding when FIELDSET is used instead of DIV -->
							<span>
							<input name="query" value="" type="hidden">
							<input name="pagenum" value="1" type="hidden">
							</span>
						</fieldset>
					</form>
					<form action="http://www3.interscience.wiley.com.ejproxy.a-star.edu.sg/search/allsearch" method="get" id="searchArticlesForm" name="allContentSearchForm" target="_top">
						<fieldset>
							<!-- NOTE: this span must be here or else ie/win adds left padding when FIELDSET is used instead of DIV -->
							<span>
								<input name="mode" value="quicksearch" type="hidden">
								<input name="WISindexid1" value="WISall" type="hidden">
								<input name="WISsearch1" value="" type="hidden">
							</span>
						</fieldset>
					</form>
					<form name="formSelector" action="javascript:selectAndSubmit();" target="_top">
						<fieldset>
							<table class="dataTable" style="padding-bottom: 0.2em;" border="0" cellpadding="0" cellspacing="0">
								<tbody><tr>
									<td rowspan="2" valign="top"><h3><img src="txt.search.gif" alt="Search" vspace="3" width="49" border="0" height="16"></h3></td>
									<td><input name="whichForm" value="allContentSearch" checked="checked" type="radio"><label for="allContent">All Content</label></td>
								</tr>
								<tr><td><input name="whichForm" value="searchTitles" type="radio"><label for="pubTitle">Publication Titles</label></td></tr>
								</tbody>
							</table>
							<div></div><table class="dataTable" borders="0" cellpadding="0" cellspacing="0">
								<tbody><tr>
								<td>
									<label for="articleKeywords" class="invisible"><span class="txt">Enter words or phrases</span><br></label>
									<input name="terms" id="articleKeywords" value="" style="width: 130px;" size="12" type="text">
								</td>
								<td>
									<a href="javascript:selectAndSubmit();"><img type="image" src="but.go.gif" alt="Go" border="0"></a>
								</td>
								</tr></tbody>
							</table>
						</fieldset>
						<ul class="noBullet" style="padding-bottom: 0pt;">
						<li><a href="http://www3.interscience.wiley.com.ejproxy.a-star.edu.sg/search/allsearch" target="_top">Advanced Search</a></li>
						<li><a href="http://www3.interscience.wiley.com.ejproxy.a-star.edu.sg/crossref.html">CrossRef / Google Search</a></li>
						<li><a href="http://www3.interscience.wiley.com.ejproxy.a-star.edu.sg/stasa/search.html">Acronym Finder</a></li>
						</ul>
					</form>
				</div>
			</div>
		</td>
	</tr>
</tbody></table>
<!-- END: search-combined.tpl -->

							</div>
						</td>
					</tr>
				</tbody></table>
			</div>
		</div>
		<!-- END: intro area -->


		<!-- BEGIN: content -->
		<div id="content" class="area">

			<div class="subContent">
				<div class="line"><img src="dot.ccc.gif" alt="" width="100%" border="0" height="1"></div>

				<!-- issue nav -->
				<table align="right" border="0" cellpadding="0" cellspacing="0"><tbody><tr>
					<td><p class="issueNav"><nobr>  <span>&lt;</span> <a href="http://www3.interscience.wiley.com.ejproxy.a-star.edu.sg/cgi-bin/fulltext/121358791/HTMLSTART" target="_top">Previous Article</a>   &nbsp;|&nbsp;    <a href="http://www3.interscience.wiley.com.ejproxy.a-star.edu.sg/cgi-bin/fulltext/120780541/HTMLSTART" target="_top">Next Article</a> <span>&gt;</span></nobr></p></td>
				</tr></tbody></table>

				<!-- BEGIN: abstract header-->
				<div class="item">
					<p><img src="icon.bullet.orange.gif" alt="" width="9" border="0" height="9"> <a href="http://www3.interscience.wiley.com.ejproxy.a-star.edu.sg/cgi-bin/savedarticleform?ID=120839840&amp;path_ok=/cgi-bin/fulltext/120839840/HTMLSTART" class="linkMore" target="_top">Save Article to My Profile</a>
					<!-- Link to Citation Export -->
					&nbsp;&nbsp;&nbsp;&nbsp;
					<img src="icon.bullet.orange.gif" alt="" width="9" border="0" height="9"> <a href="http://www3.interscience.wiley.com.ejproxy.a-star.edu.sg/tools/citex?clienttype=1&amp;subtype=1&amp;mode=1&amp;version=1&amp;id=120839840&amp;redirect=/cgi-bin/fulltext/120839840/HTMLSTART" class="linkMore" target="_top">Download Citation</a>
					</p><p>
						<a href="http://www3.interscience.wiley.com.ejproxy.a-star.edu.sg/cgi-bin/abstract/120839840/ABSTRACT" target="_top">Abstract</a> 
						
						&nbsp;|&nbsp; <a href="http://www3.interscience.wiley.com.ejproxy.a-star.edu.sg/cgi-bin/abstract/120839840/REFERENCES" target="_top">References</a> 
						
						&nbsp;|&nbsp; Full Text: <strong>HTML</strong>


					</p><div class="txt" style="text-align: right;"><a href="http://www3.interscience.wiley.com.ejproxy.a-star.edu.sg/cgi-bin/fulltext/120839840/HTMLSTARTW" class="linkMore" target="_top">View Full Width</a></div>
				</div>
				<!-- END: abstract header-->

				<div class="line"><img src="dot.ccc.gif" alt="" width="100%" border="0" height="2"></div>
			</div>

			<!-- BEGIN: prerendered content -->
			<table width="100%" border="0" cellpadding="0" cellspacing="0"><tbody><tr>
				<td width="10"><div class="spacer"><img src="dot.clear.gif" alt="" width="10" border="0" height="1"></div></td>
				<td>
					<div class="prerendered">

					
<!-- Rendered from source SGML using DSSSL -->
<!-- Rendering Developed By Andy Townsend. June-99 Aug-00 -->
<font size="+1">&nbsp;Original Article</font><br><table width="100%"><tbody><tr><td><div class="articleTitle">Segmentation and analysis of the three-dimensional redistribution of nuclear components in human mesenchymal stem cells</div><div class="articleSubTitle"></div></td></tr><tr><td>Bart J. Vermolen<sup><font size="-1"> 1 2</font><font size="-1"> *</font></sup>, Yuval Garini<sup><font size="-1"> 1 3</font></sup>, Ian T. Young<sup><font size="-1"> 1</font></sup>, Roeland W. Dirks<sup><font size="-1"> 4</font></sup>, Vered Raz<sup><font size="-1"> 4</font></sup></td></tr><tr><td><sup><font size="-1">1</font></sup>Department of Imaging Science and Technology, Delft University of Technology, Delft, The Netherlands<br><sup><font size="-1">2</font></sup>Faculty of Science and Technology, University of Twente, Enschede, The Netherlands<br><sup><font size="-1">3</font></sup>Department of Physics and Institute of Nanotechnology, Bar-Ilan University, Ramat-Gan, Israel<br><sup><font size="-1">4</font></sup>Department of Molecular Cell Biology, Leiden University Medical Center, Leiden, The Netherlands<br></td></tr><tr><td><b>email:</b> Bart J. Vermolen (<a href="mailto:b.j.vermolen@utwente.nl">b.j.vermolen@utwente.nl</a>)</td></tr></tbody></table><p><sup><font size="-1">*</font></sup>Correspondence to Bart J. Vermolen, P.O. Box 217, 7500 AE, Enschede, The Netherlands</p><p><b>Funded by:</b><br><img src="bull.gif" align="0">&nbsp;Cyttron Program; Grant Number: BSIK03036</p><p></p><table width="100%" border="0"><tbody><tr><td class="mainSectionHeader"><a name="keywords">Keywords</a></td></tr></tbody></table><table><tbody><tr><td>confocal microscopy â¢ image processing â¢ caspase-8 â¢ apoptosis â¢ telomeres â¢ centromeres</td></tr></tbody></table><p></p><table width="100%" border="0"><tbody><tr><td class="mainSectionHeader"><a name="abstract">Abstract</a></td></tr></tbody></table><table><tbody><tr><td><img src="sec_here_small.gif" alt="Abstract" border="0"></td><td><a href="#SEC1-1"><img src="sec_down_small.gif" alt="MATERIALS AND METHODS" border="0"></a></td><td><a href="#SEC1-2"><img src="sec_down_small.gif" alt="RESULTS" border="0"></a></td><td><a href="#SEC1-3"><img src="sec_down_small.gif" alt="DISCUSSION" border="0"></a></td><td><a href="#acknowledgements"><img src="sec_down_small.gif" alt="Acknowledgements" border="0"></a></td><td><a href="#references"><img src="sec_down_small.gif" alt="References" border="0"></a></td></tr></tbody></table><table width="100%" border="0"><tbody><tr><td>To better understand the impact of changes in nuclear architecture on nuclear functions, it is essential to quantitatively elucidate the three-dimensional organization of nuclear components using image processing tools. We have developed a novel image segmentation method, which involves a contrast enhancement and a subsequent thresholding step. In addition, we have developed a new segmentation method of the nuclear volume using the fluorescent background signal of a probe. After segmentation of the nucleus, a first-order normalization is performed on the signal positions of the component of interest to correct for the shape of the nucleus. This method allowed us to compare various signal positions within a single nucleus, and also on pooled data obtained from multiple nuclei, which may vary in size and shape. The algorithms have been tested by analyzing the spatial localization of nuclear bodies in relation to the nuclear center. Next, we used this new tool to study the change in the spatial distribution of nuclear components in cells before and after caspase-8 activation, which leads to cell death. Compared to the morphological TopHat method, this method gives similar but significantly faster results. A clear shift in the radial distribution of centromeres has been found, while the radial distribution of telomeres was changed much less. In addition, we have used this new tool to follow changes in the spatial distribution of two nuclear components in the same nucleus during activation of apoptosis. We show that after caspase-8 activation, when centromeres shift to a peripheral localization, the spatial distribution of PML-NBs does not change while that of centromeres did. We propose that the use of this new image segmentation method will contribute to a better understanding of the 3D spatial organization of the cell nucleus. Â© 2008 International Society for Advancement of Cytometry</td></tr></tbody></table><hr size="1" width="25%" align="left">Received: 6 February 2008; Revised: 14 April 2008; Accepted: 5 June 2008<p></p><table width="100%" border="0"><tbody><tr><td class="mainSectionHeader"><a name="doi">Digital Object Identifier (DOI)</a></td></tr></tbody></table><br>10.1002/cyto.a.20612&nbsp;&nbsp;<a target="help" href="http://www3.interscience.wiley.com.ejproxy.a-star.edu.sg/doiinfo.html">About DOI</a><br><p></p><table width="100%" border="0"><tbody><tr><td class="mainSectionHeader"><a name="articletext">Article Text</a></td></tr></tbody></table><br><p>T<font size="-1">HE</font> major function of the cell nucleus is to regulate gene activity, which depends on well-studied molecular mechanisms such as transcription, pre-mRNA splicing, and ribosome assembly. In contrast to what is known about the molecular regulation of these mechanisms, far less is understood about the extent that the dynamics of nuclear components and the three-dimensional (3D) structural organization of the nucleus contribute to the regulation of nuclear functions. Recent models of high-order genome organization suggest a nonrandom spatial localization of chromosome territories in the interphase nucleus (
[<a href="#BIB1">1</a>]). Also subchromosomal domains are suggested to be nonrandomly positioned. Centromeres containing pericentric satellite repeats show a preferential peripheral orientation in G0-arrested human cells ([<a href="#BIB2">2</a>]) as well as in differentiated cells ([<a href="#BIB2">2-4</a>]). Telomeres, which are satellite repeats at the ends of chromosomes, reveal a cell cycle-dependent localization in B-lymphocytes. Throughout the cell cycle, telomeres exhibit a spherical organization but in G2 they reorganize to a disk shape ([<a href="#BIB5">5</a>]). Together, these studies indicate a nonrandom organization of heterochromatic regions in the nucleus, and suggest that a functional correlation exists between the spatial organization of heterochromatic regions and gene activity ([<a href="#BIB4">4</a>]
[<a href="#BIB6">6</a>]
[<a href="#BIB7">7</a>]).</p><p>The mechanism by which heterochromatic regions are organized in the cell nucleus is currently unknown. There are emerging genomic and biochemical evidences that give a role for the nuclear lamina proteins (which support the structure of the nucleus) in chromatin organization and control of gene activity (
[<a href="#BIB8">8-10</a>]). Thus, if changes in lamina organization lead to changes in chromatin organization, correlation should be found using quantitative image analysis. In a recent study, we found that in cells, which are activated for apoptosis via the caspase-8 pathway, changes in lamina organization are followed by changes in the spatial organization of telomeres and centromeres ([<a href="#BIB11">11</a>]). Here, we have developed a new quantitative image analysis tool, which facilitates our studies at the 3D localization of multiple (e.g., two or three) nuclear components relative to the lamina structure in a single cell. To test whether this tool is able to quantify changes in spatial localization, we have compared the spatial organization of different nuclear components (i.e., telomeres, centromeres, and PML-NBs) in human mesenchymal stem cells (hMSCs) before and after caspase-8 activation. Image processing and analysis have been carried out in four basic steps: (1) segmentation of the nuclear bodies, (2) segmentation of the nucleus, (3) normalization of the nuclear body positions, and (4) analysis of the nuclear body radial distribution.</p><p>Several methods to detect spots, e.g., fluorescently marked telomere, centromere, and chromosomal loci signals, have been reported in the past. Most studies have been done in 2D (
[<a href="#BIB12">12</a>]) or semi-3D ([<a href="#BIB13">13</a>]); detection was done sequentially on the 2D slices of the 3D image stack. Few studies have implemented true 3D detection methods. In ([<a href="#BIB14">14</a>]
[<a href="#BIB15">15</a>]), a Gaussian model driven segmentation algorithm has been used with the assumption of a high intensity curvature and high intensity. Previously, we segmented spots of varying intensities using a TopHat algorithm ([<a href="#BIB5">5</a>]
[<a href="#BIB16">16</a>]
[<a href="#BIB17">17</a>]). In this study, we applied a model-driven segmentation approach, which is suitable for spots with varying intensities. To segment the nuclear bodies in three dimensions, we adapted a method developed by Olivo-Marin ([<a href="#BIB18">18</a>]) and extended it to 3D. Together with an interactive correction step, this method allowed us to accurately determine the spatial positioning of nuclear bodies. We also present and compare three segmentation algorithms to segment the nuclear volume. We show that the three methods give comparable results. Since one of the segmentation methods does not require imaging of the nuclear lamina, it is technically easier and therefore preferable. Furthermore, as the nucleus of hMSCs varies in size and shape; we introduce a normalization of the nuclear size, which makes it possible to perform quantitative image analysis on pooled data. After normalization, the radial distribution of different nuclear components within a single nucleus has been analyzed. This provided means to compare different components in a single cell and also to pool the different data from different cells, that is, a cell population. Using this new image processing tool, we show that changes in nuclear architecture can be monitored after activation of apoptosis by caspase-8.</p><p></p><div class="firstLevelHeading"><a name="SEC1-1">MATERIALS AND METHODS</a></div><table><tbody><tr><td><a href="#abstract"><img src="sec_up_small.gif" alt="Abstract" border="0"></a></td><td><img src="sec_here_small.gif" alt="MATERIALS AND METHODS" border="0"></td><td><a href="#SEC1-2"><img src="sec_down_small.gif" alt="RESULTS" border="0"></a></td><td><a href="#SEC1-3"><img src="sec_down_small.gif" alt="DISCUSSION" border="0"></a></td><td><a href="#acknowledgements"><img src="sec_down_small.gif" alt="Acknowledgements" border="0"></a></td><td><a href="#references"><img src="sec_down_small.gif" alt="References" border="0"></a></td></tr></tbody></table><p></p><div class="firstLevelHeading"><a name="SEC2-1">Segmentation</a></div><p></p><div class="secondLevelHeading"><a name="SEC3-1">Probe.</a></div><p>The identification of all probe signals in an image has been performed in two steps. First, a segmentation step has been performed where the object pixels (the probe signals) are distinguished from background pixels. As a result, a set of connected pixels are now defined as an object. Second, the center of the object has been determined using the fluorescence intensities as a weighting factor.</p><p>To segment the probe signal of interest in the image, a derivative scale-space method has been chosen. Previously (
[<a href="#BIB5">5</a>]
[<a href="#BIB16">16</a>]) we have used the morphological TopHat transformation ([<a href="#BIB17">17</a>]) to segment labeled telomeres in the mouse lymphocytes. Although this algorithm gave satisfactory results, we chose to implement a new and faster algorithm and compared it to the TopHat method. This method is based on a robust method developed by Olivo-Marin ([<a href="#BIB18">18</a>]), where spots in a highly variable and noisy background can be segmented. The kernel used in ([<a href="#BIB18">18</a>]) is [1/16,1/4,3/8,1/4,1/16], which is an approximation of the B3-spline function and also an approximation of the 1D Gaussian with standard deviation <img src="sigma.gif" border="0"> = 1. We therefore propose to adjust this method to find objects that resemble 3D Gaussian intensity profiles of spots with a lateral size around 200 nm, as described below. This choice has been made because the signals in the nucleus are smaller than the optical resolution of the microscope system, and therefore the intensity profiles resemble the point spread function (PSF). The 3D Gaussian is an appropriate approximation of the 3D confocal PSF ([<a href="#BIB19">19</a>]). Furthermore, the 3D Gaussian has a number of the same features as the proposed Ã  trous wavelet algorithm in ([<a href="#BIB18">18</a>]): it is translation-invariant, the images after convolution with the 3D Gaussian are correlated, and the implementation is fairly simple. The feature that it needs to be isotropic is let go, since the 3D PSF is anisotropic.</p><p>The algorithm is implemented using 3D Gaussian profiles with different widths characterized by the standard deviation, <img src="sigma.gif" border="0">, of the Gaussian function. In the axial (<i>z</i>) direction, the width is three times larger than that in the lateral (<i>x</i>, <i>y</i>) direction because of the fundamental anisotropy in the 3D PSF. The PSF is longer in the axial direction than in the lateral direction. The image, <i>g</i>, is now convolved with Gaussian profiles with different widths to produce a Gaussian scale space of the image:
</p><p><a name="EQN1"></a></p><center><table width="85%" border="0"><tbody><tr><td><img src="tex2gif.pl" align="absmiddle" border="0"></td><td align="right">1</td></tr></tbody></table></center>with <i>i</i> = 0<img src="hellip.gif" border="0">2 and where <img src="otimes.gif" border="0"> is the convolution operator, <i>G</i> is the Gaussian profile with width <img src="sigma.gif" border="0"><i><sub>i</sub></i> in the lateral direction, and 3<img src="sigma.gif" border="0"><i><sub>i</sub></i> in the axial direction, where the factor 3 corresponds to the typical ratio of axial to lateral dimensions in confocal microscopy:
<p><a name="EQN2"></a></p><center><table width="85%" border="0"><tbody><tr><td><img src="tex2gif_001.pl" align="absmiddle" border="0"></td><td align="right">2</td></tr></tbody></table></center>Next we choose a scale base to define the different widths:
<p><a name="EQN3"></a></p><center><table width="85%" border="0"><tbody><tr><td><img src="tex2gif_002.pl" align="absmiddle" border="0"></td><td align="right">3</td></tr></tbody></table></center>
with base <img src="ap.gif" border="0"> 100 nm; so, we are looking for spots of size <img src="ap.gif" border="0"> 200 nm, which is approximately the full width at half maximum of a confocal PSF with the high NA we use and will enhance the contrast of small spots. We have tested the algorithm on simulated images of two spots in close neighborhood with varying spots sizes and concluded that the contrast enhancement works for spot sizes within the range visually seen in our real data (data not shown). After multiplying the differences in the Gaussian scale space, we produce the result:
<p><a name="EQN4"></a></p><center><table width="85%" border="0"><tbody><tr><td><img src="tex2gif_003.pl" align="absmiddle" border="0"></td><td align="right">4</td></tr></tbody></table></center>For further noise reduction, we convolve <i>g</i><sub>product</sub> with a small (1 pixel wide) 3D Gaussian profile. The resulting image is thresholded at a value, <i>T</i>:
<p><a name="EQN5"></a></p><center><table width="85%" border="0"><tbody><tr><td><img src="tex2gif_004.pl" align="absmiddle" border="0"></td><td align="right">5</td></tr></tbody></table></center>We propose to define the threshold <i>T</i> by treating the high intensity values in <i>g</i><sub>product</sub> as outliers. This means that we will define <i>T</i> using the mean, <img src="mu.gif" align="absbottom" border="0">, and the standard deviation, <img src="sigma.gif" border="0">, of the intensity values of <i>g</i><sub>product</sub>:
<p><a name="EQN6"></a></p><center><table width="85%" border="0"><tbody><tr><td><img src="tex2gif_005.pl" align="absmiddle" border="0"></td><td align="right">6</td></tr></tbody></table></center>In most cases, this automatic thresholding gives a satisfactory result, indicating that the signal-to-noise ratio is sufficient for accurate segmentation of the probe. Otherwise, the user can alter this threshold level. This gives a binary mask, <i>g</i><sub>bin</sub>, with objects representing probe locations. We compute the center of gravity of intensities for every object in the resulting mask to estimate the sub-pixel (<i>x<sub>n</sub></i>, <i>y<sub>n</sub></i>, <i>z<sub>n</sub></i>) coordinate for the <i>n</i>th probe signal. This algorithm for spot detection and localization has been embedded in TeloView (
[<a href="#BIB16">16</a>]) (see <img src="ldquo.gif" border="0">Image acquisition and processing<img src="rdquo.gif" border="0"> section). After spot detection, the operator can visually check the spot detection within a matter of seconds, and manually add or remove the coordinates of probe signals if needed. This interactive step can also be used to manually separate touching objects as after segmentation some objects still touch. For instance, if a larger object visually looks like two smaller objects, one can delete the marker in the user interface and add two markers on the centre coordinates of the smaller objects.<p></p><div class="secondLevelHeading"><a name="SEC3-2">Nucleus.</a></div><p>To position the molecular markers within the nucleus, we had to define the nuclear sphere. We have used three methods to segment the nucleus (Fig. 
<a href="#FIG1">1</a>): <a name="L1"></a></p><table><tbody><tr><td valign="top" align="left">1</td><td valign="top" align="left">The lamina is labeled with lamin A or lamin B fused to a fluorescent gene product and the segmentation of the nucleus is carried out on the fluorescent signal of the lamina protein (Fig. 
<a href="#FIG1">1</a>Ai, labeled as <img src="ldquo.gif" border="0">la<img src="rdquo.gif" border="0">).</td></tr><tr><td valign="top" align="left">2</td><td valign="top" align="left">The lamina is labeled in the same way as in 1, but the segmentation of the nucleus is carried out after a logarithmic stretch of the lamina signal (Fig. 
<a href="#FIG1">1</a>Aii, labeled as <img src="ldquo.gif" border="0">nla<img src="rdquo.gif" border="0">).</td></tr><tr><td valign="top" align="left">3</td><td valign="top" align="left">The lamina is not labeled, but a logarithmic stretch is performed on the background signal of the probe. The background is probably caused by fluorescent molecules unbound to structural nuclear elements. This logarithm stretch highlights the probe background, which is sufficient to define the shape of the nucleus (Fig. 
<a href="#FIG1">1</a>Aiii, named as <img src="ldquo.gif" border="0">n<img src="rdquo.gif" border="0">); now, segmentation is carried out on this stretched background.</td></tr></tbody></table><p><a name="FIG1"></a></p><center><table width="85%" border="0"><tbody><tr><td valign="top" width="128" align="left"><a href="javascript:display_fig001(0)"><img src="tfig001" border="0"></a></td><td align="left"><font size="-1"><b>Figure 1. </b>Comparison of three methods for segmentation of the nuclear sphere. (<b>A</b>) Image representations of the different segmentation methods in one nucleus. The red line outlines the nuclear boundary, which was found by each of the segmentation methods. The <i>n</i> segmentation is calculated from the logarithmic stretched image of the background of the probe signal (i). The <i>nla</i> segmentation is calculated from the logarithmic stretched image of the lamina signal (ii). The <i>la</i> segmentation is calculated from the image of the lamina signal (iii). (<b>B</b>) Cumulative distribution function plot of Trf1-DsRed spatial distribution. Plots show the distribution of pooled data from five cells after activation of caspase-8 at passage 4 using the three segmentation methods, object background (<i>n</i>), lamina in background mode (<i>nla</i>), and the lamina (<i>la</i>). (<b>C</b>) Linear regression analyses of the pooled data (B) reveals that the <i>nla</i> and <i>la</i> segmentation methods are statistically equivalent. The <i>n</i> method, however, gives a good estimation of the spatial localization of the probes in the nucleus, as compared with the other two methods.<br>[<a href="javascript:display_fig001(0)">Normal View 54K</a> | <a href="javascript:display_fig001(1)">Magnified View 177K</a>]</font></td></tr></tbody></table></center><p>In all three methods, we segment the nucleus with an isodata thresholding algorithm (
[<a href="#BIB20">20</a>]) after noise reduction by a convolution with a Gaussian filter. This easy algorithm is sufficient, because there is only one nucleus per image and we do not need to separate touching objects.</p><p>While method 3 differs from method 1 and 2 by using a different probe, methods 1 and 2 differ from each other because the logarithmic stretch causes the isodata thresholding algorithm to choose a different thresholding level. Although method 3 is more sensitive to the signal-to-noise ratio compared to the other methods, nuclear segmentation worked for all our images with all three methods. This resulted in three different segmentations of the nucleus.</p><p></p><div class="firstLevelHeading"><a name="SEC2-2">Probe Distribution</a></div><p></p><div class="secondLevelHeading"><a name="SEC3-3">Radial position with a segmented nucleus.</a></div><p>To calculate the radial distribution of the components inside the nucleus, a method is required that eliminates the effect of the nuclear size and the nonspherical shape. We use a method that produces a normalized distance of a probe, <i>r</i><sub>norm</sub>, from the center of a nucleus to the nuclear boundary. Thus, <i>r</i><sub>norm</sub> = 0 means that a probe is at the center and <i>r</i><sub>norm</sub> = 1 means that a probe is at the boundary. Our method approximates the nucleus shape of the hMSCs by an ellipsoid. To normalize the probe position, we used the coordinates of the pixels obtained from the segmented nucleus to find the center of the nucleus. We also use these coordinates to transform the coordinates of the probe signals (<i>x<sub>i</sub></i>, <i>y<sub>i</sub></i>, <i>z<sub>i</sub></i>). From the nuclear coordinates, we calculate the covariance matrix (the second-order moments of inertia of the nuclear voxels). We then perform a singular value decomposition on this matrix. This gives a rotation matrix, <i>S<sub>n</sub></i>, and a matrix with the singular values, <i>V<sub>n</sub></i>. The singular values represent the variances, <img src="sigma.gif" border="0"><img src="latex2gif.pl" align="middle" border="0">, in the different principal directions given by the rotation matrix. The principal directions are the directions in which the variances are largest.</p><p>The singular matrix has the form:
</p><p><a name="EQN7"></a></p><center><table width="85%" border="0"><tbody><tr><td><img src="tex2gif_006.pl" align="absmiddle" border="0"></td><td align="right">7</td></tr></tbody></table></center>where
<p><a name="EQN8"></a></p><center><table width="85%" border="0"><tbody><tr><td><img src="tex2gif_007.pl" align="absmiddle" border="0"></td><td align="right">8</td></tr></tbody></table></center>For a sphere with radius one, the unit sphere, all three singular values will give 
<img src="tex2gif_008.pl" align="absmiddle" border="0">. Now, we rotate and normalize the variances of the probe signal coordinates to the unit sphere:
<p><a name="EQN9"></a></p><center><table width="85%" border="0"><tbody><tr><td><img src="tex2gif_009.pl" align="absmiddle" border="0"></td><td align="right">9</td></tr></tbody></table></center>A schematic presentation of this transformation is shown in Figure 
<a href="#FIG2">2</a>. After this transformation, the normalized radius, <i>r</i><sub>norm</sub>, is simply
<p><a name="EQN10"></a></p><center><table width="85%" border="0"><tbody><tr><td><img src="tex2gif_010.pl" align="absmiddle" border="0"></td><td align="right">10</td></tr></tbody></table></center>
<p><a name="FIG2"></a></p><center><table width="85%" border="0"><tbody><tr><td valign="top" width="128" align="left"><a href="javascript:display_fig002(0)"><img src="tfig002" border="0"></a></td><td align="left"><font size="-1"><b>Figure 2. </b>Schematic representation of the normalization procedure. (<b>A</b>) 2D example of a nucleus (red) with a signal (black dot). (<b>B</b>) The same nucleus (A) after normalization. The normalized radius, <i>r</i><sub>norm</sub>, is given by <i>r</i>/<i>R</i>. If <i>R</i> = 1 (green circle, the unit sphere), <i>r</i><sub>norm</sub> is given by Eq. 
<a href="#EQN9">(9)</a>.<br>[<a href="javascript:display_fig002(0)">Normal View 10K</a> | <a href="javascript:display_fig002(1)">Magnified View 24K</a>]</font></td></tr></tbody></table></center><p></p><div class="firstLevelHeading"><a name="SEC2-3">Cell Preparation and Molecular Labeling</a></div><p>Human MSCs were isolated from bone marrow samples of adult donors and were cultured as described in (
[<a href="#BIB11">11</a>]).</p><p>To visualize nuclear proteins, we expressed lamin A, lamin B (components of the nuclear lamina), Trf1, Trf2 (both associated with the telomeres), and CenpA (associated with the centromeres) as fusion proteins with GFP and DsRed in hMSCs as previously described (
[<a href="#BIB11">11</a>]). PML-NBs were visualized after fixation of the cells using a specific anti-PML antibody as described in ([<a href="#BIB11">11</a>]).</p><p></p><div class="firstLevelHeading"><a name="SEC2-4">Image Acquisition and Processing</a></div><p>3D images from live or fixed cells were acquired using a confocal microscope (Leica, model TCS-2) equipped with an argon/krypton laser and a 100Ã/1.3 NA Apo objective. The 3D images were loaded in TeloView, a custom noncommercial in-house developed analysis program for MatLab (The Mathworks, Natick, MA). TeloView makes use of DIPimage (
[<a href="#BIB21">21</a>]) developed at the Quantitative Imaging Group (TU-Delft, The Netherlands, <a href="http://www.diplib.org/" target="_top">http://www.diplib.org</a>). The typical sampling distances in the lateral direction were on the order of <img src="delta.gif" border="0"><i>x</i> = <img src="delta.gif" border="0"><i>y</i> = 40 nm and in the axial direction on the order of <img src="delta.gif" border="0"><i>z</i> = 160 nm. This anisotropy in the sampling distances makes weighing of pixel values and distances necessary during segmentation and analysis; we have solved this by interpolating the images to get isotropic sampling distances. Typical image size was 512 Ã 512 Ã 40. Before segmentation, the images are interpolated linearly in the <i>z</i> direction from <img src="delta.gif" border="0"><i>z</i> to <img src="delta.gif" border="0"><i>z</i><img src="prime.gif" border="0"> so that the sampling distance in all three directions is the same: <img src="delta.gif" border="0"><i>x</i> = <img src="delta.gif" border="0"><i>y</i> = <img src="delta.gif" border="0"><i>z</i><img src="prime.gif" border="0">. Note that our remark about the fundamental anisotropy in the confocal PSF in the section on probe segmentation still holds after this interpolation, since the PSF is an intrinsic property of the microscope system and is therefore independent of the sampling.</p><p></p><div class="firstLevelHeading"><a name="SEC2-5">Statistics</a></div><p>The spatial distribution of the probe is plotted in a cumulative distribution function (CDF) plot. We have chosen this representation to avoid the problem of choosing a binning size in a histogram (the conventional way of showing the empirical data distribution function), which can heavily influence the appearance of the distribution. The CDF of the radius, <i>F</i>(<i>r</i>), is defined as
</p><p><a name="EQN11"></a></p><center><table width="85%" border="0"><tbody><tr><td><img src="tex2gif_011.pl" align="absmiddle" border="0"></td><td align="right">11</td></tr></tbody></table></center>Distributions were compared with the Wilcoxon rank-sum test (
[<a href="#BIB22">22</a>]
[<a href="#BIB23">23</a>]) and the two sample Kolmogorov-Smirnov test ([<a href="#BIB24">24</a>]).<p></p><div class="firstLevelHeading"><a name="SEC1-2">RESULTS</a></div><table><tbody><tr><td><a href="#abstract"><img src="sec_up_small.gif" alt="Abstract" border="0"></a></td><td><a href="#SEC1-1"><img src="sec_up_small.gif" alt="MATERIALS AND METHODS" border="0"></a></td><td><img src="sec_here_small.gif" alt="RESULTS" border="0"></td><td><a href="#SEC1-3"><img src="sec_down_small.gif" alt="DISCUSSION" border="0"></a></td><td><a href="#acknowledgements"><img src="sec_down_small.gif" alt="Acknowledgements" border="0"></a></td><td><a href="#references"><img src="sec_down_small.gif" alt="References" border="0"></a></td></tr></tbody></table><p></p><div class="firstLevelHeading"><a name="SEC2-6">Probe Segmentation</a></div><p>We have compared the scale space segmentation algorithm with a TopHat transformation method. For the scale space segmentation we used three scales (<i>i</i> = 0, 1, 2) and base = 100 nm. The TopHat transformation uses parameters for the structuring element (SE) as previously described in (
[<a href="#BIB16">16</a>]); a spherical SE with radius 742 nm. A total of 14 images with 524 telomere signals have been analyzed. On average, we find 37 telomeres per cell. This number is lower than expected, since not all telomeres are labeled with this method and also possibly due to telomere merging. We have chosen a different threshold for the scale-space method, <i>T</i> (see <img src="ldquo.gif" border="0"><a href="#SEC1-1">Materials and Methods</a><img src="rdquo.gif" border="0"> section), for each image separately, so that the number of interactive corrections (signals added and signals removed) in each image is minimized. This resulted in 26 corrections (<img src="ap.gif" border="0">2 corrections per image). For the threshold in the TopHat method, we chose a threshold for each image in the same manner, resulting in 30 corrections (<img src="ap.gif" border="0">2 corrections per image). This amount of corrections is acceptable for practical use. While no significant differences in the number of corrections has been found between the two methods, the new method gives significant advantages in computation time. Typical computation time of the TopHat method is 5-15 min versus 1-3 min with the scale space method on an AMD Opteron<sup>TM</sup> 244 1.8 GHz processor with 8 GB RAM. Therefore, we chose to use the scale-space method.</p><p></p><div class="firstLevelHeading"><a name="SEC2-7">Nuclear Segmentation</a></div><p>We have used the fluorescent protein-tagged lamin A or lamin B proteins to visualize the nuclear envelope. During apoptosis, before degradation of the lamina proteins, the nuclear envelope changes from a round-flat to a convoluted shape. In addition, as previously described, the spatial organization of centromeres and telomeres changes (
[<a href="#BIB11">11</a>]). At this point, we are interested in developing a more accurate quantitative image analysis tool that will allow us to study the spatial distribution of nuclear components with respect to the nuclear shape. We therefore tested three methods for segmentation of the nucleus (Fig. 
<a href="#FIG1">1</a>A). The segmentation methods are described in <img src="ldquo.gif" border="0"><a href="#SEC1-1">Materials and Methods</a><img src="rdquo.gif" border="0"> section. The performance of the three segmentation methods has been tested on telomere signals. 3D confocal images were taken from hMSCs after activation of caspase-8 at passage 4, expressing Trf1-DsRed and lamin B-GFP. An example of a nucleus expressing both fluorescent fusion proteins is shown in Figure <a href="#FIG5">5</a>A. After nuclear segmentation, using each of the methods, nuclear boundaries were indicated by a red line (Fig. <a href="#FIG1">1</a>A, red line). The nuclear shape was determined by segmenting the Lamin B-GFP signal without contrast stretch (<i>la</i>), or after a logarithmic stretch (<i>nla</i>), or by segmenting the nuclear background signal derived from Trf1-DsRed (<i>n</i>) (Fig. <a href="#FIG1">1</a>Ai, ii and iii, respectively). Comparisons of the three methods revealed little differences (Figs. <a href="#FIG1">1</a>B and <a href="#FIG1">1</a>C). Therefore, when analyzing the change in distribution of nuclear bodies from the center of mass, it is possible to use the <i>n</i> method for nuclear segmentation, as it is significantly faster and simpler. During apoptosis, however, the shape of the nuclear envelope changes, and in addition, the lamina form intranuclear structures ([<a href="#BIB11">11</a>]). Thus, it would be interesting to analyze the change in the spatial distribution relative to the lamina. As changes in lamina organization precede changes in the spatial organization of centromeres and telomeres ([<a href="#BIB11">11</a>]), it is possible that changes in lamina spatial organization affect the spatial organization of nuclear bodies.</p><p>We emphasize that the images contain single nuclei, so that more sophisticated segmentation algorithms (
[<a href="#BIB25">25-27</a>]), where multiple cells or nuclei are present in one image, which need to be segmented, are unnecessary in this study.</p><p>Next, the spatial distribution of the telomeres was quantified within each of the segmented nuclei and the frequency of telomeres was plotted against the nuclear radius. To perform statistical tests on pooled data, it was essential to first normalize the radius of the nuclei, as described in Figure 
<a href="#FIG2">2</a> and in the <img src="ldquo.gif" border="0"><a href="#SEC1-1">Materials and Methods</a><img src="rdquo.gif" border="0"> section [Eq. 
<a href="#EQN9">(9)</a>]. The pooled data from five nuclei, representing 343 Trf1-DsRed fluorescent dots, revealed no significant differences between the three segmentation methods (Figs. <a href="#FIG1">1</a>B and <a href="#FIG1">1</a>C). However, a detailed comparison revealed that the <i>la</i> and the <i>nla</i> methods show a high level of correlation, <i>R</i> = 0.99 (Fig. <a href="#FIG1">1</a>C). This is not surprising since both the <i>la</i> and the <i>nla</i> methods are based on the lamina signal. The correlation of the <i>la</i> or <i>nla</i> methods with the <i>n</i> method was less, <i>R</i> = 0.78 and <i>R</i> = 0.77, respectively (Fig. <a href="#FIG1">1</a>C). We suggest that the <i>n</i> method for nuclear segmentation gives a good estimation for the spatial localization of nuclear components. This method, however, does not outline the nuclear envelop as precisely when we visually evaluate the resulting segmentation and compare it to the lamina segmentation methods. Finally, some general notes on the effect of radial positioning with respect to the accuracy of the segmentation. The segmentation might give two kinds of errors: with and without a bias. Statistically, these errors will not affect the final results in our application. For the nonbiasing errors, this is trivial: the ellipse fitting as explained in the section on probe distribution will reduce this error to a very small contribution. When a bias exists, it will not affect differences in radial position distributions we report. The accuracy of the nuclear segmentation should be further explored, depending on the specific application. Nevertheless, as the <i>n</i> segmentation method is the fastest and requires significantly less work, it can be used to estimate the nuclear shape.</p><p></p><div class="firstLevelHeading"><a name="SEC2-8">Changes in Spatial Distribution During Activation of Apoptosis</a></div><p>To test whether the <i>n</i> segmentation method can be used to study changes in the spatial organization of multiple nuclear components, we applied it to images from cells expressing both CenpA-GFP and Trf1-DsRed (Fig. 
<a href="#FIG3">3</a>A). These cells were induced to go into apoptosis by expressing the FKC8 gene, which was activated after AP20187 treatment (
[<a href="#BIB11">11</a>]
[<a href="#BIB28">28</a>]). To study eventual differences between centromere and telomere spatial organization, we developed a method that finds the radial distribution of multiple nuclear components in a single nucleus. Live hMSCs were imaged before and 4 and 6 h after AP20187 treatment.
</p><p><a name="FIG3"></a></p><center><table width="85%" border="0"><tbody><tr><td valign="top" width="128" align="left"><a href="javascript:display_fig003(0)"><img src="tfig003" border="0"></a></td><td align="left"><font size="-1"><b>Figure 3. </b>Distribution analysis of centromeres and telomeres in a single nucleus. (<b>A</b>) Maximum intensity projections of single hMSC expressing CenpA-GFP (green) and Trf1-DsRed (Red), before (mock-treated) and after caspase-8 activation (+AP20187, 4 h treatment). (<b>B</b>) Cumulative distribution function plots showing changes in the relative distribution of CenpA and Trf1 signals during caspase-8 activation in a single representative cell. Analyses were carried out in cells expressing the FKC8 vector before and after caspase-8 activation (+AP20187). Four hours after AP20187 treatment, the nucleus showed a round shape and after 6 h a convoluted shape. The plots show that in normal hMSCs, the spatial distribution of CenpA is more central compared with that of Trf1. Upon caspase-8 activation, the cumulative distribution function plots reveal a clear shift in CenpA localization towards the periphery in round-shape nuclei. A very similar distribution of CenpA and Trf1 is found in convoluted nuclei. The <i>P</i> values of the Wilcoxon rank-sum test and the two sample Kolmogorov- Smirnov test are given by <i>P</i>(W) and <i>P</i>(KS) respectively.<br>[<a href="javascript:display_fig003(0)">Normal View 88K</a> | <a href="javascript:display_fig003(1)">Magnified View 294K</a>]</font></td></tr></tbody></table></center><p>First, we tested this method on single living cells. For a mock-treated cell, the CDF plots show that the centromeres have a higher preference for a central localization as compared with telomere positioning (Fig. 
<a href="#FIG3">3</a>B, mock-treated). This observation confirms previous studies done on fixed cells, showing a more central localization of centromeres as compared to telomeres in interphase nuclei in various cell types (
[<a href="#BIB29">29</a>]
[<a href="#BIB30">30</a>]). The CDF plot revealed that few telomeres are localized at the nuclear periphery in untreated hMSCs (Fig. 
<a href="#FIG3">3</a>B). This observation is consistent with previous studies showing that few telomeres are physically associated with the nuclear lamina ([<a href="#BIB31">31</a>]).</p><p>After apoptosis activation by AP20187 treatment, we observed in the round-shaped nucleus that the centromeres were positioned near the nuclear periphery [Fig. 
<a href="#FIG3">3</a>B, round, and (
[<a href="#BIB9">9</a>])]. At that time, in the same nucleus, the spatial distribution of telomeres was more central than that of the centromeres (Fig. <a href="#FIG3">3</a>B, round). At a later stage, in the convoluted-shaped nucleus, the radial distribution of centromeres and telomeres did not differ, as shown by the statistical tests. These analyses suggest that both centromeres and telomeres change position within the three-dimensional space of the cell nucleus at different rates. In addition, the shift in centromere localization is more significant than the telomere shift.</p><p>Unlike centromeres and telomeres that show changes in spatial organization upon caspase-8 activation, PML-NB organization is initially unchanged, even in cells showing a convoluted nuclear shape. The distribution of PML-NBs is disrupted only when the lamina shows massive degradation (
[<a href="#BIB11">11</a>]). Here, we used our newly developed method to quantitatively compare the spatial localization of centromeres and PML-NBs in mock-treated cells showing a round nuclear shape and in caspase-8-activated cells showing a convoluted nuclear shape. To visualize PML-NBs, cells were fixed and incubated with the appropriate antibodies (Fig. 
<a href="#FIG4">4</a>A). The frequency distributions of both centromeres and PML-NBs were plotted in histograms against radial rings. Representative results are shown in Figure <a href="#FIG4">4</a>B. The analysis of this one cell reveals a peripheral localization of several CenpA molecules in caspase-8-activated cells, while the spatial distribution of PML-NBs does not change. This observation is consistent with previous results, showing shift in CenpA distributions in apoptotic cells ([<a href="#BIB11">11</a>]).
</p><p><a name="FIG4"></a></p><center><table width="85%" border="0"><tbody><tr><td valign="top" width="128" align="left"><a href="javascript:display_fig004(0)"><img src="tfig004" border="0"></a></td><td align="left"><font size="-1"><b>Figure 4. </b>PML-NB and CenpA distribution in a single nucleus. (<b>A</b>) Maximum intensity projection of CenpA-GFP (green) and PML-NB (red) within single hMSCs expressing FKC8, before (mock-treated) and after caspase-8 activation (+AP20187, 6-h treatment). (B) Histograms show the fraction of centromeres (green) or PML-NBs (red) distributed in a normalized nucleus of a representative cell. Analyses were carried out on mock-treated cells and on cells 6 h after AP20187 treatment. In mock-treated hMSCs, CenpA and PML-NBs have a similar spatial distribution pattern, but upon caspase-8 activation, some CenpA molecules show a peripheral localization while the distribution of PML-NBs is not affected.<br>[<a href="javascript:display_fig004(0)">Normal View 62K</a> | <a href="javascript:display_fig004(1)">Magnified View 185K</a>]</font></td></tr></tbody></table></center><p>Next, we analyzed pooled data obtained from multiple cells expressing both CenpA-GFP and Trf1-DsRed before or 4 h after caspase-8 activation. When we visually analyzed the localization of centromeres with respect to the center of the nucleus, centromeres were shown to be centrally localized, while after caspase-8 activation, their spatial distribution shifted towards the nuclear periphery (Fig. 
<a href="#FIG5">5</a>A). A clear shift in centromere localization towards the nuclear periphery is also shown by the CDF plot of the pooled data of four cells before and after treatment with <i>P</i> &lt; 1e-7 calculated by the Wilcoxon rank sum test, using the nuclear background segmentation method (Fig. <a href="#FIG5">5</a>B). In contrast, the <i>P</i> values calculated by the Wilcoxon rank sum test did not show a significant change in Trf1 localization between normal, nine cells, and convoluted-shaped nuclei, five cells, in caspase-8-activated cells. From the CDF plot, we estimated that 20% of the telomeres are localized to the nuclear periphery (<i>r</i> &gt; 0.8) in both untreated and AP20187-treated cells. Thus, the radial distribution of centromeres shifts to the periphery in convoluted-shaped nuclei, whereas the telomere distribution is not changed.
</p><p><a name="FIG5"></a></p><center><table width="85%" border="0"><tbody><tr><td valign="top" width="128" align="left"><a href="javascript:display_fig005(0)"><img src="tfig005" border="0"></a></td><td align="left"><font size="-1"><b>Figure 5. </b>Changes in the spatial distribution of telomeres and centromeres during activation of apoptosis. (<b>A</b>) Left: maximum intensity projection of a mock-treated cell expressing lamin A-DsRed (blue) together with CenpA-GFP (green). Right: maximum intensity projection of a cell expressing Lamin B-GFP (green), Ttrf1-DsRed (red), and FKC8, after AP20187 treatment. Lamin A and B expression marks the inner nuclear membrane. (<b>B</b>) Cumulative distribution function plot of the spatial distribution of CenpA-GFP before (blue line) and 4 h after caspase-8 activation with 100 nM AP20187 (red line). Segmentation of the nuclear sphere was carried out using the <i>nla</i> segmentation. The graph shows pooled data from four cells for each treatment. The <i>P</i> value of the Wilcoxon rank-sum test is given by <i>P</i>(W). (<b>C</b>) Cumulative distribution function plot of the spatial distribution of Trf1-DsRed analyzed in cells expressing lamin B-GFP before (blue line) and 6 h after caspase-8 activation (green line). Segmentation of the nuclear sphere was carried out using the <i>nla</i> segmentation method. The graph shows pooled data from nine cells and five cells before and after caspase-8 activation, respectively. The <i>P</i> value of the Wilcoxon rank-sum test is given by <i>P</i>(W).<br>[<a href="javascript:display_fig005(0)">Normal View 56K</a> | <a href="javascript:display_fig005(1)">Magnified View 207K</a>]</font></td></tr></tbody></table></center><p></p><div class="firstLevelHeading"><a name="SEC1-3">DISCUSSION</a></div><table><tbody><tr><td><a href="#abstract"><img src="sec_up_small.gif" alt="Abstract" border="0"></a></td><td><a href="#SEC1-1"><img src="sec_up_small.gif" alt="MATERIALS AND METHODS" border="0"></a></td><td><a href="#SEC1-2"><img src="sec_up_small.gif" alt="RESULTS" border="0"></a></td><td><img src="sec_here_small.gif" alt="DISCUSSION" border="0"></td><td><a href="#acknowledgements"><img src="sec_down_small.gif" alt="Acknowledgements" border="0"></a></td><td><a href="#references"><img src="sec_down_small.gif" alt="References" border="0"></a></td></tr></tbody></table><p>We have presented a new segmentation method to analyze probe distribution in the cell nucleus. This method shows a clear improvement over the conventional tophat method, especially with respect to computation time. We have also developed an improved segmentation method for the nucleus itself, which is based on background signals derived from expressed fluorescent fusion proteins. Using this method, the preparation of samples and the acquisition of images is technically easier and therefore superior. Segmented images allow us to quantify the spatial distribution of centromeres and telomeres relative to the nuclear center. This tool also allows us to quantitatively relate the radial distribution between two or three nuclear components in a single cell nucleus. It also allows a quantitative comparison of the distribution of nuclear components between cells and between treatments and various biological situations. As an example, we showed here how the distribution of centromeres and telomeres changes after activation of caspase-8, which results in cell death.</p><p>Nuclear architecture does not only describe the organization of the nucleus, but recent studies indicate that the spatial and temporal organization of the genome is likely to have functional consequences. Changes in nuclear architecture are amongst the most dramatic hallmarks of development and differentiation processes and defects in architectural elements of the cell nucleus are responsible for several human diseases. Yet, it is not clear how the spatial localization of the various nuclear structures is changed during change in cell function. Two major models have been suggested (
[<a href="#BIB32">32</a>]). In the structural scaffold concept, structural proteins, like the lamina proteins, confine the activation regions of the genome. The self-reinforcing concept suggests that the spatial organization of nuclear structures reflects the transcriptional activity of the genome. Following the self-reinforcing concept, we tested here the spatial localization of nuclear probes relative to the center of mass. However, it is possible that changes in the nuclear lamina also contribute to the spatial localization of nuclear structures, like in human diseases with mutation in lamina genes ([<a href="#BIB33">33</a>]) or in cell death ([<a href="#BIB11">11</a>]
[<a href="#BIB32">32</a>]
[<a href="#BIB33">33</a>]). Therefore, it would be interesting to test the change in the spatial localization of nuclear probes using the distance transform approach ([<a href="#BIB34">34</a>]), where the smallest distance to the lamina will be found. The distance from the center of the nucleus to the nuclear component, as described in this study, and the distance from the lamina to the nuclear component are fundamentally different metrics and can give complementary information. Finally, it would be mostly interesting to compare between the two methods with respect to the biological models. A broad application of both tools will eventually lead to a better understanding of the nuclear architecture and its relation to nuclear function.</p><p></p><div class="firstLevelHeading"><a name="acknowledgements">Acknowledgements</a></div><table><tbody><tr><td><a href="#abstract"><img src="sec_up_small.gif" alt="Abstract" border="0"></a></td><td><a href="#SEC1-1"><img src="sec_up_small.gif" alt="MATERIALS AND METHODS" border="0"></a></td><td><a href="#SEC1-2"><img src="sec_up_small.gif" alt="RESULTS" border="0"></a></td><td><a href="#SEC1-3"><img src="sec_up_small.gif" alt="DISCUSSION" border="0"></a></td><td><img src="sec_here_small.gif" alt="Acknowledgements" border="0"></td><td><a href="#references"><img src="sec_down_small.gif" alt="References" border="0"></a></td></tr></tbody></table><p>We thank the members of Prof. H. Tanke's group for fruitful discussions.</p><p></p><table width="100%" border="0"><tbody><tr><td class="mainSectionHeader"><a name="references">References</a></td></tr></tbody></table><table><tbody><tr><td><a href="#abstract"><img src="sec_up_small.gif" alt="Abstract" border="0"></a></td><td><a href="#SEC1-1"><img src="sec_up_small.gif" alt="MATERIALS AND METHODS" border="0"></a></td><td><a href="#SEC1-2"><img src="sec_up_small.gif" alt="RESULTS" border="0"></a></td><td><a href="#SEC1-3"><img src="sec_up_small.gif" alt="DISCUSSION" border="0"></a></td><td><a href="#acknowledgements"><img src="sec_up_small.gif" alt="Acknowledgements" border="0"></a></td><td><img src="sec_here_small.gif" alt="References" border="0"></td></tr></tbody></table><table width="100%"><tbody><tr valign="baseline"><td class="referenceNum">1</td><td><a name="BIB1">

CremerT,KÃ¼pperK,DietzelS,FakanS.
Higher order chromatin architecture in the cell nucleus: On the way from structure to function.
<i>Biol Cell</i> 
2004;

<b>96</b>:
 555-567. </a><a href="http://www3.interscience.wiley.com.ejproxy.a-star.edu.sg/cgi-bin/reflink?object=TYPE%3DJCIT%26BIBID%3D1%26SNM%3DCremer%26FNM%3DT%26SNM%3DK%FCpper%26FNM%3DK%26SNM%3DDietzel%26FNM%3DS%26SNM%3DFakan%26FNM%3DS%26ATL%3DHigher%20order%20chromatin%20architecture%20in%20the%20cell%20nucleus%3A%20On%20the%20way%20from%20structure%20to%20function%26JTL%3DBiol%20Cell%26PYR%3D2004%26VID%3D96%26PPF%3D555%26PPL%3D567%26ADOI%3D10.1002/cyto.a.20612%26EVIEW%3DY%26ENABLEISI%3DN%26ENABLECAS%3DY%26ENABLEPM%3DY" target="_top"><span style="color: rgb(51, 102, 153);">Links</span></a>&nbsp;&nbsp;</td></tr><tr valign="baseline"><td class="referenceNum">2</td><td><a name="BIB2">

SoloveiI,SchermellehL,DÃ¼ringK,EngelhardtA,SteinS,CremerC,CremerT.
Differences in centromere positioning of cycling and postmitotic human cell types.
<i>Chromosoma</i> 
2004;

<b>112</b>:
 410-423. </a><a href="http://www3.interscience.wiley.com.ejproxy.a-star.edu.sg/cgi-bin/reflink?object=TYPE%3DJCIT%26BIBID%3D2%26SNM%3DSolovei%26FNM%3DI%26SNM%3DSchermelleh%26FNM%3DL%26SNM%3DD%FCring%26FNM%3DK%26SNM%3DEngelhardt%26FNM%3DA%26SNM%3DStein%26FNM%3DS%26SNM%3DCremer%26FNM%3DC%26SNM%3DCremer%26FNM%3DT%26ATL%3DDifferences%20in%20centromere%20positioning%20of%20cycling%20and%20postmitotic%20human%20cell%20types%26JTL%3DChromosoma%26PYR%3D2004%26VID%3D112%26PPF%3D410%26PPL%3D423%26ADOI%3D10.1002/cyto.a.20612%26EVIEW%3DY%26ENABLEISI%3DN%26ENABLECAS%3DY%26ENABLEPM%3DY" target="_top"><span style="color: rgb(51, 102, 153);">Links</span></a>&nbsp;&nbsp;</td></tr><tr valign="baseline"><td class="referenceNum">3</td><td><a name="BIB3">

WiblinAE,CuiW,ClarkAJ,BickmoreWA.
Distinctive nuclear organisation of centromeres and regions involved in pluripotency in human embryonic stem cells.
<i>J Cell Sci</i> 
2005;

<b>118</b>:
 3861-3868. </a><a href="http://www3.interscience.wiley.com.ejproxy.a-star.edu.sg/cgi-bin/reflink?object=TYPE%3DJCIT%26BIBID%3D3%26SNM%3DWiblin%26FNM%3DAE%26SNM%3DCui%26FNM%3DW%26SNM%3DClark%26FNM%3DAJ%26SNM%3DBickmore%26FNM%3DWA%26ATL%3DDistinctive%20nuclear%20organisation%20of%20centromeres%20and%20regions%20involved%20in%20pluripotency%20in%20human%20embryonic%20stem%20cells%26JTL%3DJ%20Cell%20Sci%26PYR%3D2005%26VID%3D118%26PPF%3D3861%26PPL%3D3868%26ADOI%3D10.1002/cyto.a.20612%26EVIEW%3DY%26ENABLEISI%3DN%26ENABLECAS%3DY%26ENABLEPM%3DY" target="_top"><span style="color: rgb(51, 102, 153);">Links</span></a>&nbsp;&nbsp;</td></tr><tr valign="baseline"><td class="referenceNum">4</td><td><a name="BIB4">

StadlerS,SchnappV,MayerR,SteinS,CremerC,BoniferC,CremerT,DietzelS.
The architecture of chicken chromosome territories changes during differentiation.
<i>BMC Cell Biol</i> 
2004;

<b>5</b>:
 44. </a><a href="http://www3.interscience.wiley.com.ejproxy.a-star.edu.sg/cgi-bin/reflink?object=TYPE%3DJCIT%26BIBID%3D4%26SNM%3DStadler%26FNM%3DS%26SNM%3DSchnapp%26FNM%3DV%26SNM%3DMayer%26FNM%3DR%26SNM%3DStein%26FNM%3DS%26SNM%3DCremer%26FNM%3DC%26SNM%3DBonifer%26FNM%3DC%26SNM%3DCremer%26FNM%3DT%26SNM%3DDietzel%26FNM%3DS%26ATL%3DThe%20architecture%20of%20chicken%20chromosome%20territories%20changes%20during%20differentiation%26JTL%3DBMC%20Cell%20Biol%26PYR%3D2004%26VID%3D5%26PPF%3D44%26ADOI%3D10.1002/cyto.a.20612%26EVIEW%3DY%26ENABLEISI%3DN%26ENABLECAS%3DY%26ENABLEPM%3DY" target="_top"><span style="color: rgb(51, 102, 153);">Links</span></a>&nbsp;&nbsp;</td></tr><tr valign="baseline"><td class="referenceNum">5</td><td><a name="BIB5">

ChuangT,MoshirS,GariniY,ChuangA,YoungI,VermolenB,van den DoelR,MougeyV,PerrinM,BraunM,KerrPD,FestT,BoukampP,MaiS.
The three-dimensional organization of telomeres in the nucleus of mammalian cells.
<i>BMC Biol</i> 
2004;

<b>2</b>:
 12. </a><a href="http://www3.interscience.wiley.com.ejproxy.a-star.edu.sg/cgi-bin/reflink?object=TYPE%3DJCIT%26BIBID%3D5%26SNM%3DChuang%26FNM%3DT%26SNM%3DMoshir%26FNM%3DS%26SNM%3DGarini%26FNM%3DY%26SNM%3DChuang%26FNM%3DA%26SNM%3DYoung%26FNM%3DI%26SNM%3DVermolen%26FNM%3DB%26SNM%3Dvan%20den%20Doel%26FNM%3DR%26SNM%3DMougey%26FNM%3DV%26SNM%3DPerrin%26FNM%3DM%26SNM%3DBraun%26FNM%3DM%26SNM%3DKerr%26FNM%3DPD%26SNM%3DFest%26FNM%3DT%26SNM%3DBoukamp%26FNM%3DP%26SNM%3DMai%26FNM%3DS%26ATL%3DThe%20three-dimensional%20organization%20of%20telomeres%20in%20the%20nucleus%20of%20mammalian%20cells%26JTL%3DBMC%20Biol%26PYR%3D2004%26VID%3D2%26PPF%3D12%26ADOI%3D10.1002/cyto.a.20612%26EVIEW%3DY%26ENABLEISI%3DN%26ENABLECAS%3DY%26ENABLEPM%3DY" target="_top"><span style="color: rgb(51, 102, 153);">Links</span></a>&nbsp;&nbsp;</td></tr><tr valign="baseline"><td class="referenceNum">6</td><td><a name="BIB6">

ParadaLA,McQueenPG,MisteliT.
Tissue-specific spatial organization of genomes.
<i>Genome Biol</i> 
2004;

<b>5</b>:
 R44. </a><a href="http://www3.interscience.wiley.com.ejproxy.a-star.edu.sg/cgi-bin/reflink?object=TYPE%3DJCIT%26BIBID%3D6%26SNM%3DParada%26FNM%3DLA%26SNM%3DMcQueen%26FNM%3DPG%26SNM%3DMisteli%26FNM%3DT%26ATL%3DTissue-specific%20spatial%20organization%20of%20genomes%26JTL%3DGenome%20Biol%26PYR%3D2004%26VID%3D5%26PPF%3DR44%26ADOI%3D10.1002/cyto.a.20612%26EVIEW%3DY%26ENABLEISI%3DN%26ENABLECAS%3DY%26ENABLEPM%3DY" target="_top"><span style="color: rgb(51, 102, 153);">Links</span></a>&nbsp;&nbsp;</td></tr><tr valign="baseline"><td class="referenceNum">7</td><td><a name="BIB7">

KimSH,McQueenPG,LichtmanMK,ShevachEM,ParadaLA,MisteliT.
Spatial genome organization during T-cell differentiation.
<i>Cytogenet Genome Res</i> 
2004;

<b>105</b>(2-4):
 292-301. </a><a href="http://www3.interscience.wiley.com.ejproxy.a-star.edu.sg/cgi-bin/reflink?object=TYPE%3DJCIT%26BIBID%3D7%26SNM%3DKim%26FNM%3DSH%26SNM%3DMcQueen%26FNM%3DPG%26SNM%3DLichtman%26FNM%3DMK%26SNM%3DShevach%26FNM%3DEM%26SNM%3DParada%26FNM%3DLA%26SNM%3DMisteli%26FNM%3DT%26ATL%3DSpatial%20genome%20organization%20during%20T-cell%20differentiation%26JTL%3DCytogenet%20Genome%20Res%26PYR%3D2004%26VID%3D105%26IID%3D2-4%26PPF%3D292%26PPL%3D301%26ADOI%3D10.1002/cyto.a.20612%26EVIEW%3DY%26ENABLEISI%3DN%26ENABLECAS%3DY%26ENABLEPM%3DY" target="_top"><span style="color: rgb(51, 102, 153);">Links</span></a>&nbsp;&nbsp;</td></tr><tr valign="baseline"><td class="referenceNum">8</td><td><a name="BIB8">

GruenbaumY,MargalitA,GoldmanRD,ShumakerDK,WilsonKL.
The nuclear lamina comes of age.
<i>Nat Rev Mol Cell Biol</i> 
2005;

<b>6</b>:
 21-31. </a><a href="http://www3.interscience.wiley.com.ejproxy.a-star.edu.sg/cgi-bin/reflink?object=TYPE%3DJCIT%26BIBID%3D8%26SNM%3DGruenbaum%26FNM%3DY%26SNM%3DMargalit%26FNM%3DA%26SNM%3DGoldman%26FNM%3DRD%26SNM%3DShumaker%26FNM%3DDK%26SNM%3DWilson%26FNM%3DKL%26ATL%3DThe%20nuclear%20lamina%20comes%20of%20age%26JTL%3DNat%20Rev%20Mol%20Cell%20Biol%26PYR%3D2005%26VID%3D6%26PPF%3D21%26PPL%3D31%26ADOI%3D10.1002/cyto.a.20612%26EVIEW%3DY%26ENABLEISI%3DN%26ENABLECAS%3DY%26ENABLEPM%3DY" target="_top"><span style="color: rgb(51, 102, 153);">Links</span></a>&nbsp;&nbsp;</td></tr><tr valign="baseline"><td class="referenceNum">9</td><td><a name="BIB9">

KosakST,GroudineM.
Gene order and dynamic domains.
<i>Science</i> 
2004;

<b>306</b>:
 644-647. </a><a href="http://www3.interscience.wiley.com.ejproxy.a-star.edu.sg/cgi-bin/reflink?object=TYPE%3DJCIT%26BIBID%3D9%26SNM%3DKosak%26FNM%3DST%26SNM%3DGroudine%26FNM%3DM%26ATL%3DGene%20order%20and%20dynamic%20domains%26JTL%3DScience%26PYR%3D2004%26VID%3D306%26PPF%3D644%26PPL%3D647%26ADOI%3D10.1002/cyto.a.20612%26EVIEW%3DY%26ENABLEISI%3DN%26ENABLECAS%3DY%26ENABLEPM%3DY" target="_top"><span style="color: rgb(51, 102, 153);">Links</span></a>&nbsp;&nbsp;</td></tr><tr valign="baseline"><td class="referenceNum">10</td><td><a name="BIB10">

PickersgillH,KalverdaB,de WitE,TalhoutW,FornerodM,van SteenselB.
Characterization of the <i>Drosophila melanogaster</i> genome at the nuclear lamina.
<i>Nat Genet</i> 
2006;

<b>38</b>:
 1005-1014. </a><a href="http://www3.interscience.wiley.com.ejproxy.a-star.edu.sg/cgi-bin/reflink?object=TYPE%3DJCIT%26BIBID%3D10%26SNM%3DPickersgill%26FNM%3DH%26SNM%3DKalverda%26FNM%3DB%26SNM%3Dde%20Wit%26FNM%3DE%26SNM%3DTalhout%26FNM%3DW%26SNM%3DFornerod%26FNM%3DM%26SNM%3Dvan%20Steensel%26FNM%3DB%26ATL%3DCharacterization%20of%20the%20Drosophila%20melanogaster%20genome%20at%20the%20nuclear%20lamina%26JTL%3DNat%20Genet%26PYR%3D2006%26VID%3D38%26PPF%3D1005%26PPL%3D1014%26ADOI%3D10.1002/cyto.a.20612%26EVIEW%3DY%26ENABLEISI%3DN%26ENABLECAS%3DY%26ENABLEPM%3DY" target="_top"><span style="color: rgb(51, 102, 153);">Links</span></a>&nbsp;&nbsp;</td></tr><tr valign="baseline"><td class="referenceNum">11</td><td><a name="BIB11">

RazV,CarlottiF,VermolenBJ,van der PoelE,SloosWCR,Knaan-ShanzerS,de VriesAAF,HoebenRC,YoungIT,TankeHJ,GariniY,DirksRW.
Changes in lamina structure are followed by spatial reorganization of heterochromatic regions in caspase-8-activated human mesenchymal stem cells.
<i>J Cell Sci</i> 
2006;

<b>119</b>:
 4247-4256. </a><a href="http://www3.interscience.wiley.com.ejproxy.a-star.edu.sg/cgi-bin/reflink?object=TYPE%3DJCIT%26BIBID%3D11%26SNM%3DRaz%26FNM%3DV%26SNM%3DCarlotti%26FNM%3DF%26SNM%3DVermolen%26FNM%3DBJ%26SNM%3Dvan%20der%20Poel%26FNM%3DE%26SNM%3DSloos%26FNM%3DWCR%26SNM%3DKnaan-Shanzer%26FNM%3DS%26SNM%3Dde%20Vries%26FNM%3DAAF%26SNM%3DHoeben%26FNM%3DRC%26SNM%3DYoung%26FNM%3DIT%26SNM%3DTanke%26FNM%3DHJ%26SNM%3DGarini%26FNM%3DY%26SNM%3DDirks%26FNM%3DRW%26ATL%3DChanges%20in%20lamina%20structure%20are%20followed%20by%20spatial%20reorganization%20of%20heterochromatic%20regions%20in%20caspase-8-activated%20human%20mesenchymal%20stem%20cells%26JTL%3DJ%20Cell%20Sci%26PYR%3D2006%26VID%3D119%26PPF%3D4247%26PPL%3D4256%26ADOI%3D10.1002/cyto.a.20612%26EVIEW%3DY%26ENABLEISI%3DN%26ENABLECAS%3DY%26ENABLEPM%3DY" target="_top"><span style="color: rgb(51, 102, 153);">Links</span></a>&nbsp;&nbsp;</td></tr><tr valign="baseline"><td class="referenceNum">12</td><td><a name="BIB12">

NettenH,YoungIT,van VlietLJ,TankeHJ,VrolijkH,SloosWCR.
FISH and chips: Automation of fluorescent dot counting in interphase cell nuclei.
<i>Cytometry</i> 
1997;

<b>28</b>:
 1-10. </a><a href="http://www3.interscience.wiley.com.ejproxy.a-star.edu.sg/cgi-bin/reflink?object=TYPE%3DJCIT%26BIBID%3D12%26SNM%3DNetten%26FNM%3DH%26SNM%3DYoung%26FNM%3DIT%26SNM%3Dvan%20Vliet%26FNM%3DLJ%26SNM%3DTanke%26FNM%3DHJ%26SNM%3DVrolijk%26FNM%3DH%26SNM%3DSloos%26FNM%3DWCR%26ATL%3DFISH%20and%20chips%3A%20Automation%20of%20fluorescent%20dot%20counting%20in%20interphase%20cell%20nuclei%26JTL%3DCytometry%26PYR%3D1997%26VID%3D28%26PPF%3D1%26PPL%3D10%26ADOI%3D10.1002/cyto.a.20612%26EVIEW%3DY%26ENABLEISI%3DN%26ENABLECAS%3DY%26ENABLEPM%3DY" target="_top"><span style="color: rgb(51, 102, 153);">Links</span></a>&nbsp;&nbsp;</td></tr><tr valign="baseline"><td class="referenceNum">13</td><td><a name="BIB13">

GrigoryanAM,DoughertyER,KononenJ,BubendorfL,HostetterG,KallioniemiO.
Morphological spot counting from stacked images for automated analysis of gene copy numbers by fluorescence <i>in situ</i> hybridization.
<i>J Biomed Opt</i> 
2002;

<b>7</b>:
 109-122. </a><a href="http://www3.interscience.wiley.com.ejproxy.a-star.edu.sg/cgi-bin/reflink?object=TYPE%3DJCIT%26BIBID%3D13%26SNM%3DGrigoryan%26FNM%3DAM%26SNM%3DDougherty%26FNM%3DER%26SNM%3DKononen%26FNM%3DJ%26SNM%3DBubendorf%26FNM%3DL%26SNM%3DHostetter%26FNM%3DG%26SNM%3DKallioniemi%26FNM%3DO%26ATL%3DMorphological%20spot%20counting%20from%20stacked%20images%20for%20automated%20analysis%20of%20gene%20copy%20numbers%20by%20fluorescence%20in%20situ%20hybridization%26JTL%3DJ%20Biomed%20Opt%26PYR%3D2002%26VID%3D7%26PPF%3D109%26PPL%3D122%26ADOI%3D10.1002/cyto.a.20612%26EVIEW%3DY%26ENABLEISI%3DN%26ENABLECAS%3DY%26ENABLEPM%3DY" target="_top"><span style="color: rgb(51, 102, 153);">Links</span></a>&nbsp;&nbsp;</td></tr><tr valign="baseline"><td class="referenceNum">14</td><td><a name="BIB14">

ThomannD,RinesDR,SorgerPK,DanuserG.
Automatic fluorescent tag detection in 3D with super-resolution: Application to the analysis of chromosome movement.
<i>J Microsc</i> 
2002;

<b>208</b>:
 49-64. </a><a href="http://www3.interscience.wiley.com.ejproxy.a-star.edu.sg/cgi-bin/reflink?object=TYPE%3DJCIT%26BIBID%3D14%26SNM%3DThomann%26FNM%3DD%26SNM%3DRines%26FNM%3DDR%26SNM%3DSorger%26FNM%3DPK%26SNM%3DDanuser%26FNM%3DG%26ATL%3DAutomatic%20fluorescent%20tag%20detection%20in%203D%20with%20super-resolution%3A%20Application%20to%20the%20analysis%20of%20chromosome%20movement%26JTL%3DJ%20Microsc%26PYR%3D2002%26VID%3D208%26PPF%3D49%26PPL%3D64%26ADOI%3D10.1002/cyto.a.20612%26EVIEW%3DY%26ENABLEISI%3DN%26ENABLECAS%3DY%26ENABLEPM%3DY" target="_top"><span style="color: rgb(51, 102, 153);">Links</span></a>&nbsp;&nbsp;</td></tr><tr valign="baseline"><td class="referenceNum">15</td><td><a name="BIB15">

ThomannD,DornJ,SorgerPK,DanuserG.
Automatic fluorescent tag localization II: Improvement in super-resolution by relative tracking.
<i>J Microsc</i> 
2003;

<b>211</b>:
 230-248. </a><a href="http://www3.interscience.wiley.com.ejproxy.a-star.edu.sg/cgi-bin/reflink?object=TYPE%3DJCIT%26BIBID%3D15%26SNM%3DThomann%26FNM%3DD%26SNM%3DDorn%26FNM%3DJ%26SNM%3DSorger%26FNM%3DPK%26SNM%3DDanuser%26FNM%3DG%26ATL%3DAutomatic%20fluorescent%20tag%20localization%20II%3A%20Improvement%20in%20super-resolution%20by%20relative%20tracking%26JTL%3DJ%20Microsc%26PYR%3D2003%26VID%3D211%26PPF%3D230%26PPL%3D248%26ADOI%3D10.1002/cyto.a.20612%26EVIEW%3DY%26ENABLEISI%3DN%26ENABLECAS%3DY%26ENABLEPM%3DY" target="_top"><span style="color: rgb(51, 102, 153);">Links</span></a>&nbsp;&nbsp;</td></tr><tr valign="baseline"><td class="referenceNum">16</td><td><a name="BIB16">

VermolenBJ,GariniY,MaiS,MougeyV,FestT,ChuangTC-Y,ChuangAY-C,WarkL,YoungIT.
Characterizing the three-dimensional organization of telomeres.
<i>Cytometry A</i> 
2005;

<b>67A</b>:
 144-150. </a><a href="http://www3.interscience.wiley.com.ejproxy.a-star.edu.sg/cgi-bin/reflink?object=TYPE%3DJCIT%26BIBID%3D16%26SNM%3DVermolen%26FNM%3DBJ%26SNM%3DGarini%26FNM%3DY%26SNM%3DMai%26FNM%3DS%26SNM%3DMougey%26FNM%3DV%26SNM%3DFest%26FNM%3DT%26SNM%3DChuang%26FNM%3DTC-Y%26SNM%3DChuang%26FNM%3DAY-C%26SNM%3DWark%26FNM%3DL%26SNM%3DYoung%26FNM%3DIT%26ATL%3DCharacterizing%20the%20three-dimensional%20organization%20of%20telomeres%26JTL%3DCytometry%20A%26PYR%3D2005%26VID%3D67A%26PPF%3D144%26PPL%3D150%26ADOI%3D10.1002/cyto.a.20612%26EVIEW%3DY%26ENABLEISI%3DN%26ENABLECAS%3DY%26ENABLEPM%3DY" target="_top"><span style="color: rgb(51, 102, 153);">Links</span></a>&nbsp;&nbsp;</td></tr><tr valign="baseline"><td class="referenceNum">17</td><td><a name="BIB17">

MeyerF.
Iterative image transformations for an automatic screening of cervical smears.
<i>J Histochem Cytochem</i> 
1979;

<b>27</b>:
 128-135. </a><a href="http://www3.interscience.wiley.com.ejproxy.a-star.edu.sg/cgi-bin/reflink?object=TYPE%3DJCIT%26BIBID%3D17%26SNM%3DMeyer%26FNM%3DF%26ATL%3DIterative%20image%20transformations%20for%20an%20automatic%20screening%20of%20cervical%20smears%26JTL%3DJ%20Histochem%20Cytochem%26PYR%3D1979%26VID%3D27%26PPF%3D128%26PPL%3D135%26ADOI%3D10.1002/cyto.a.20612%26EVIEW%3DY%26ENABLEISI%3DN%26ENABLECAS%3DY%26ENABLEPM%3DY" target="_top"><span style="color: rgb(51, 102, 153);">Links</span></a>&nbsp;&nbsp;</td></tr><tr valign="baseline"><td class="referenceNum">18</td><td><a name="BIB18">

Olivo-MarinJ-C.
Extraction of spots in biological images using multiscale products.
<i>Pattern Recognit</i> 
2002;

<b>35</b>:
 1989-1996. </a><a href="http://www3.interscience.wiley.com.ejproxy.a-star.edu.sg/cgi-bin/reflink?object=TYPE%3DJCIT%26BIBID%3D18%26SNM%3DOlivo-Marin%26FNM%3DJ-C%26ATL%3DExtraction%20of%20spots%20in%20biological%20images%20using%20multiscale%20products%26JTL%3DPattern%20Recognit%26PYR%3D2002%26VID%3D35%26PPF%3D1989%26PPL%3D1996%26ADOI%3D10.1002/cyto.a.20612%26EVIEW%3DY%26ENABLEISI%3DN%26ENABLECAS%3DY%26ENABLEPM%3DY" target="_top"><span style="color: rgb(51, 102, 153);">Links</span></a>&nbsp;&nbsp;</td></tr><tr valign="baseline"><td class="referenceNum">19</td><td><a name="BIB19">

ZhangB,ZerubiaJ,Olivo-MarinJ-C.
Gaussian approximations of fluorescence microscope point-spread function models.
<i>Appl Opt</i> 
2007;

<b>46</b>:
 1819-1829. </a><a href="http://www3.interscience.wiley.com.ejproxy.a-star.edu.sg/cgi-bin/reflink?object=TYPE%3DJCIT%26BIBID%3D19%26SNM%3DZhang%26FNM%3DB%26SNM%3DZerubia%26FNM%3DJ%26SNM%3DOlivo-Marin%26FNM%3DJ-C%26ATL%3DGaussian%20approximations%20of%20fluorescence%20microscope%20point-spread%20function%20models%26JTL%3DAppl%20Opt%26PYR%3D2007%26VID%3D46%26PPF%3D1819%26PPL%3D1829%26ADOI%3D10.1002/cyto.a.20612%26EVIEW%3DY%26ENABLEISI%3DN%26ENABLECAS%3DY%26ENABLEPM%3DY" target="_top"><span style="color: rgb(51, 102, 153);">Links</span></a>&nbsp;&nbsp;</td></tr><tr valign="baseline"><td class="referenceNum">20</td><td><a name="BIB20">

RidlerTW,CalvardS.
Picture thresholding using an iterative selection method.
<i>IEEE Trans Syst Man Cybern</i> 
1978;

<b>8</b>:
 630-632. </a><a href="http://www3.interscience.wiley.com.ejproxy.a-star.edu.sg/cgi-bin/reflink?object=TYPE%3DJCIT%26BIBID%3D20%26SNM%3DRidler%26FNM%3DTW%26SNM%3DCalvard%26FNM%3DS%26ATL%3DPicture%20thresholding%20using%20an%20iterative%20selection%20method%26JTL%3DIEEE%20Trans%20Syst%20Man%20Cybern%26PYR%3D1978%26VID%3D8%26PPF%3D630%26PPL%3D632%26ADOI%3D10.1002/cyto.a.20612%26EVIEW%3DY%26ENABLEISI%3DN%26ENABLECAS%3DY%26ENABLEPM%3DY" target="_top"><span style="color: rgb(51, 102, 153);">Links</span></a>&nbsp;&nbsp;</td></tr><tr valign="baseline"><td class="referenceNum">21</td><td><a name="BIB21">

Luengo HendriksCL,RiegerB,van GinkelM,van KempenGMP,van VlietLJ.
<i>DIPimage, a scientific image processing toolbox for MATLAB</i>. Pattern Recognition Group,
Department of Applied Physics, Delft University of Technology,
1999. </a></td></tr><tr valign="baseline"><td class="referenceNum">22</td><td><a name="BIB22">

MannHB,WhitneyDR.
On a test whether one of two random variables is stochastically larger than the other.
<i>Ann Math Stat</i> 
1947;

<b>18</b>:
 50-60. </a><a href="http://www3.interscience.wiley.com.ejproxy.a-star.edu.sg/cgi-bin/reflink?object=TYPE%3DJCIT%26BIBID%3D22%26SNM%3DMann%26FNM%3DHB%26SNM%3DWhitney%26FNM%3DDR%26ATL%3DOn%20a%20test%20whether%20one%20of%20two%20random%20variables%20is%20stochastically%20larger%20than%20the%20other%26JTL%3DAnn%20Math%20Stat%26PYR%3D1947%26VID%3D18%26PPF%3D50%26PPL%3D60%26ADOI%3D10.1002/cyto.a.20612%26EVIEW%3DY%26ENABLEISI%3DN%26ENABLECAS%3DY%26ENABLEPM%3DY" target="_top"><span style="color: rgb(51, 102, 153);">Links</span></a>&nbsp;&nbsp;</td></tr><tr valign="baseline"><td class="referenceNum">23</td><td><a name="BIB23">

WilcoxonF.
Individual comparisons by ranking methods.
<i>Biomet Bull</i> 
1945;

<b>1</b>:
 80-83. </a><a href="http://www3.interscience.wiley.com.ejproxy.a-star.edu.sg/cgi-bin/reflink?object=TYPE%3DJCIT%26BIBID%3D23%26SNM%3DWilcoxon%26FNM%3DF%26ATL%3DIndividual%20comparisons%20by%20ranking%20methods%26JTL%3DBiomet%20Bull%26PYR%3D1945%26VID%3D1%26PPF%3D80%26PPL%3D83%26ADOI%3D10.1002/cyto.a.20612%26EVIEW%3DY%26ENABLEISI%3DN%26ENABLECAS%3DY%26ENABLEPM%3DY" target="_top"><span style="color: rgb(51, 102, 153);">Links</span></a>&nbsp;&nbsp;</td></tr><tr valign="baseline"><td class="referenceNum">24</td><td><a name="BIB24">

HollanderM,WolfeDA.
Nonparametric statistical methods. In:
BradleyRA ,HunterJS ,KendallDG ,WatsonGS , editors.
<i>Wiley Series in Probability and Mathematical Statistics</i>.
New York:
Wiley;
1973. </a></td></tr><tr valign="baseline"><td class="referenceNum">25</td><td><a name="BIB25">

Ortiz de SolÃ³rzanoC,RodriguezEG,JonesA,PinkelD,GrayJW,SudarD,LockettSJ.
Segmentation of confocal microscope images of cell nuclei in thick tissue sections.
<i>J Microsc</i> 
1999;

<b>193</b>:
 212-226. </a><a href="http://www3.interscience.wiley.com.ejproxy.a-star.edu.sg/cgi-bin/reflink?object=TYPE%3DJCIT%26BIBID%3D25%26SNM%3DOrtiz%20de%20Sol%F3rzano%26FNM%3DC%26SNM%3DRodriguez%26FNM%3DEG%26SNM%3DJones%26FNM%3DA%26SNM%3DPinkel%26FNM%3DD%26SNM%3DGray%26FNM%3DJW%26SNM%3DSudar%26FNM%3DD%26SNM%3DLockett%26FNM%3DSJ%26ATL%3DSegmentation%20of%20confocal%20microscope%20images%20of%20cell%20nuclei%20in%20thick%20tissue%20sections%26JTL%3DJ%20Microsc%26PYR%3D1999%26VID%3D193%26PPF%3D212%26PPL%3D226%26ADOI%3D10.1002/cyto.a.20612%26EVIEW%3DY%26ENABLEISI%3DN%26ENABLECAS%3DY%26ENABLEPM%3DY" target="_top"><span style="color: rgb(51, 102, 153);">Links</span></a>&nbsp;&nbsp;</td></tr><tr valign="baseline"><td class="referenceNum">26</td><td><a name="BIB26">

SartiA,Ortiz de SolÃ³rzanoC,LockettS,MalladiR.
A geometric model for 3-D confocal image analysis.
<i>IEEE Trans Biomed Eng</i> 
2000;

<b>47</b>:
 1600-1609. </a><a href="http://www3.interscience.wiley.com.ejproxy.a-star.edu.sg/cgi-bin/reflink?object=TYPE%3DJCIT%26BIBID%3D26%26SNM%3DSarti%26FNM%3DA%26SNM%3DOrtiz%20de%20Sol%F3rzano%26FNM%3DC%26SNM%3DLockett%26FNM%3DS%26SNM%3DMalladi%26FNM%3DR%26ATL%3DA%20geometric%20model%20for%203-D%20confocal%20image%20analysis%26JTL%3DIEEE%20Trans%20Biomed%20Eng%26PYR%3D2000%26VID%3D47%26PPF%3D1600%26PPL%3D1609%26ADOI%3D10.1002/cyto.a.20612%26EVIEW%3DY%26ENABLEISI%3DN%26ENABLECAS%3DY%26ENABLEPM%3DY" target="_top"><span style="color: rgb(51, 102, 153);">Links</span></a>&nbsp;&nbsp;</td></tr><tr valign="baseline"><td class="referenceNum">27</td><td><a name="BIB27">

ParvinB,YangQ,HanJ,ChangH,RydbergB,Barcellos-HoffMH.
Iterative voting for inference of structural saliency and characterization of subcellular events.
<i>IEEE Trans Image Process</i> 
2007;

<b>16</b>:
 615-623. </a><a href="http://www3.interscience.wiley.com.ejproxy.a-star.edu.sg/cgi-bin/reflink?object=TYPE%3DJCIT%26BIBID%3D27%26SNM%3DParvin%26FNM%3DB%26SNM%3DYang%26FNM%3DQ%26SNM%3DHan%26FNM%3DJ%26SNM%3DChang%26FNM%3DH%26SNM%3DRydberg%26FNM%3DB%26SNM%3DBarcellos-Hoff%26FNM%3DMH%26ATL%3DIterative%20voting%20for%20inference%20of%20structural%20saliency%20and%20characterization%20of%20subcellular%20events%26JTL%3DIEEE%20Trans%20Image%20Process%26PYR%3D2007%26VID%3D16%26PPF%3D615%26PPL%3D623%26ADOI%3D10.1002/cyto.a.20612%26EVIEW%3DY%26ENABLEISI%3DN%26ENABLECAS%3DY%26ENABLEPM%3DY" target="_top"><span style="color: rgb(51, 102, 153);">Links</span></a>&nbsp;&nbsp;</td></tr><tr valign="baseline"><td class="referenceNum">28</td><td><a name="BIB28">

CarlottiF,ZaldumbideA,MartinP,BoulukosKE,HoebenRC,PognonecP.
Development of an inducible suicide gene system based on human caspase 8.
<i>Cancer Gene Ther</i> 
2005;

<b>12</b>:
 627-639. </a><a href="http://www3.interscience.wiley.com.ejproxy.a-star.edu.sg/cgi-bin/reflink?object=TYPE%3DJCIT%26BIBID%3D28%26SNM%3DCarlotti%26FNM%3DF%26SNM%3DZaldumbide%26FNM%3DA%26SNM%3DMartin%26FNM%3DP%26SNM%3DBoulukos%26FNM%3DKE%26SNM%3DHoeben%26FNM%3DRC%26SNM%3DPognonec%26FNM%3DP%26ATL%3DDevelopment%20of%20an%20inducible%20suicide%20gene%20system%20based%20on%20human%20caspase%208%26JTL%3DCancer%20Gene%20Ther%26PYR%3D2005%26VID%3D12%26PPF%3D627%26PPL%3D639%26ADOI%3D10.1002/cyto.a.20612%26EVIEW%3DY%26ENABLEISI%3DN%26ENABLECAS%3DY%26ENABLEPM%3DY" target="_top"><span style="color: rgb(51, 102, 153);">Links</span></a>&nbsp;&nbsp;</td></tr><tr valign="baseline"><td class="referenceNum">29</td><td><a name="BIB29">

AmrichovaJ,LukasovaE,KozubekS,KozubekM.
Nuclear and territorial topography of chromosome telomeres in human lymphocytes.
<i>Exp Cell Res</i> 
2003;

<b>289</b>:
 11-26. </a><a href="http://www3.interscience.wiley.com.ejproxy.a-star.edu.sg/cgi-bin/reflink?object=TYPE%3DJCIT%26BIBID%3D29%26SNM%3DAmrichova%26FNM%3DJ%26SNM%3DLukasova%26FNM%3DE%26SNM%3DKozubek%26FNM%3DS%26SNM%3DKozubek%26FNM%3DM%26ATL%3DNuclear%20and%20territorial%20topography%20of%20chromosome%20telomeres%20in%20human%20lymphocytes%26JTL%3DExp%20Cell%20Res%26PYR%3D2003%26VID%3D289%26PPF%3D11%26PPL%3D26%26ADOI%3D10.1002/cyto.a.20612%26EVIEW%3DY%26ENABLEISI%3DN%26ENABLECAS%3DY%26ENABLEPM%3DY" target="_top"><span style="color: rgb(51, 102, 153);">Links</span></a>&nbsp;&nbsp;</td></tr><tr valign="baseline"><td class="referenceNum">30</td><td><a name="BIB30">

WeierichC,BreroA,SteinS,HaseJv,CremerC,CremerT,SoloveiI.
Three-dimensional arrangements of centromeres and telomeres in nuclei of human and murine lymphocytes.
<i>Chromosome Res</i> 
2003;

<b>11</b>:
 485-502. </a><a href="http://www3.interscience.wiley.com.ejproxy.a-star.edu.sg/cgi-bin/reflink?object=TYPE%3DJCIT%26BIBID%3D30%26SNM%3DWeierich%26FNM%3DC%26SNM%3DBrero%26FNM%3DA%26SNM%3DStein%26FNM%3DS%26SNM%3DHase%26FNM%3DJv%26SNM%3DCremer%26FNM%3DC%26SNM%3DCremer%26FNM%3DT%26SNM%3DSolovei%26FNM%3DI%26ATL%3DThree-dimensional%20arrangements%20of%20centromeres%20and%20telomeres%20in%20nuclei%20of%20human%20and%20murine%20lymphocytes%26JTL%3DChromosome%20Res%26PYR%3D2003%26VID%3D11%26PPF%3D485%26PPL%3D502%26ADOI%3D10.1002/cyto.a.20612%26EVIEW%3DY%26ENABLEISI%3DN%26ENABLECAS%3DY%26ENABLEPM%3DY" target="_top"><span style="color: rgb(51, 102, 153);">Links</span></a>&nbsp;&nbsp;</td></tr><tr valign="baseline"><td class="referenceNum">31</td><td><a name="BIB31">

LudÃ©rusME,van SteenselB,ChongL,SibonOC,CremersFF,de LangeT.
Structure, subnuclear distribution, and nuclear matrix association of the mammalian telomeric complex.
<i>J Cell Biol</i> 
1996;

<b>135</b>:
 867-881. </a><a href="http://www3.interscience.wiley.com.ejproxy.a-star.edu.sg/cgi-bin/reflink?object=TYPE%3DJCIT%26BIBID%3D31%26SNM%3DLud%E9rus%26FNM%3DME%26SNM%3Dvan%20Steensel%26FNM%3DB%26SNM%3DChong%26FNM%3DL%26SNM%3DSibon%26FNM%3DOC%26SNM%3DCremers%26FNM%3DFF%26SNM%3Dde%20Lange%26FNM%3DT%26ATL%3DStructure%2C%20subnuclear%20distribution%2C%20and%20nuclear%20matrix%20association%20of%20the%20mammalian%20telomeric%20complex%26JTL%3DJ%20Cell%20Biol%26PYR%3D1996%26VID%3D135%26PPF%3D867%26PPL%3D881%26ADOI%3D10.1002/cyto.a.20612%26EVIEW%3DY%26ENABLEISI%3DN%26ENABLECAS%3DY%26ENABLEPM%3DY" target="_top"><span style="color: rgb(51, 102, 153);">Links</span></a>&nbsp;&nbsp;</td></tr><tr valign="baseline"><td class="referenceNum">32</td><td><a name="BIB32">

MisteliT.
Concepts in nuclear architecture.
<i>Bioessays</i> 
2005;

<b>27</b>:
 477-487. </a><a href="http://www3.interscience.wiley.com.ejproxy.a-star.edu.sg/cgi-bin/reflink?object=TYPE%3DJCIT%26BIBID%3D32%26SNM%3DMisteli%26FNM%3DT%26ATL%3DConcepts%20in%20nuclear%20architecture%26JTL%3DBioessays%26PYR%3D2005%26VID%3D27%26PPF%3D477%26PPL%3D487%26ADOI%3D10.1002/cyto.a.20612%26EVIEW%3DY%26ENABLEISI%3DN%26ENABLECAS%3DY%26ENABLEPM%3DY" target="_top"><span style="color: rgb(51, 102, 153);">Links</span></a>&nbsp;&nbsp;</td></tr><tr valign="baseline"><td class="referenceNum">33</td><td><a name="BIB33">

BroersJLV,RamaekersFCS,BonneG,YaouRB,HutchisonCJ.
Nuclear lamins: Laminopathies and their role in premature ageing.
<i>Physiol Rev</i> 
2006;

<b>86</b>:
 967-1008. </a><a href="http://www3.interscience.wiley.com.ejproxy.a-star.edu.sg/cgi-bin/reflink?object=TYPE%3DJCIT%26BIBID%3D33%26SNM%3DBroers%26FNM%3DJLV%26SNM%3DRamaekers%26FNM%3DFCS%26SNM%3DBonne%26FNM%3DG%26SNM%3DYaou%26FNM%3DRB%26SNM%3DHutchison%26FNM%3DCJ%26ATL%3DNuclear%20lamins%3A%20Laminopathies%20and%20their%20role%20in%20premature%20ageing%26JTL%3DPhysiol%20Rev%26PYR%3D2006%26VID%3D86%26PPF%3D967%26PPL%3D1008%26ADOI%3D10.1002/cyto.a.20612%26EVIEW%3DY%26ENABLEISI%3DN%26ENABLECAS%3DY%26ENABLEPM%3DY" target="_top"><span style="color: rgb(51, 102, 153);">Links</span></a>&nbsp;&nbsp;</td></tr><tr valign="baseline"><td class="referenceNum">34</td><td><a name="BIB34">

MullikinJC.
The vector distance transform in two and three dimensions.
<i>CVGIP: Graph Models Image Process</i> 
1992;

<b>54</b>:
 526-535. </a><a href="http://www3.interscience.wiley.com.ejproxy.a-star.edu.sg/cgi-bin/reflink?object=TYPE%3DJCIT%26BIBID%3D34%26SNM%3DMullikin%26FNM%3DJC%26ATL%3DThe%20vector%20distance%20transform%20in%20two%20and%20three%20dimensions%26JTL%3DCVGIP%3A%20Graph%20Models%20Image%20Process%26PYR%3D1992%26VID%3D54%26PPF%3D526%26PPL%3D535%26ADOI%3D10.1002/cyto.a.20612%26EVIEW%3DY%26ENABLEISI%3DN%26ENABLECAS%3DY%26ENABLEPM%3DY" target="_top"><span style="color: rgb(51, 102, 153);">Links</span></a>&nbsp;&nbsp;</td></tr></tbody></table>

					</div>
				</td>
				<td width="10"><div class="spacer"><img src="dot.clear.gif" alt="" width="10" border="0" height="1"></div></td>
			</tr></tbody></table>
			<!-- END: prerendered content -->

		</div>
		<!-- END: content -->

	</div>
	<!-- END: main column -->

</div>

<!-- END: journal-article.html-main.tpl -->

<!-- END: ftx_abs.tpl -->
</body>
</html>
