<!DOCTYPE html PUBLIC "-//W3C//DTD XHTML 1.0 Strict//EN" "http://www.w3.org/TR/xhtml1/DTD/xhtml1-strict.dtd">
<html xmlns:m="http://www.w3.org/1998/Math/MathML">
<head>
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">
<title>BioMed Central | Full text | A multiresolution approach to automated classification of protein subcellular location images</title>

<meta name="citation_pdf_url" content="http://www.biomedcentral.com/content/pdf/1471-2105-8-210.pdf">
<meta name="citation_abstract_html_url" content="http://www.biomedcentral.com/1471-2105/8/210/abstract/">
<meta name="citation_fulltext_html_url" content="http://www.biomedcentral.com/1471-2105/8/210">

<link rel="search" type="application/opensearchdescription+xml" title="BioMed Central Search" href="http://www.biomedcentral.com/download/toolbar/firefox/bmc.xml">

<!--[if IE 7]><link rel='stylesheet' type='text/css' href="/stylesheets/ie7.css"/><![endif]--><!--[if lte IE 6]><link rel='stylesheet' type='text/css' href="/stylesheets/ie56.css"/><![endif]-->

<meta name="dc.title" content="A multiresolution approach to automated classification of protein subcellular location images">
<meta name="dc.creator" content="Amina Chebira">
<meta name="dc.creator" content="Yann Barbotin">
<meta name="dc.creator" content="Charles Jackson">
<meta name="dc.creator" content="Thomas Merryman">
<meta name="dc.creator" content="Gowri Srinivasa">
<meta name="dc.creator" content="Robert Murphy">
<meta name="dc.creator" content="Jelena Kovačević">
<meta name="dc.type" content="Research article">
<meta name="dc.source" content="BMC Bioinformatics 2007 8:210">
<meta name="dc.date" content="2007-6-19">
<meta name="dc.identifier" content="10.1186/1471-2105-8-210">
<meta name="dc.publisher" content="Biomed Central">
<meta name="dc.rights" content="http://creativecommons.org/licenses/by/2.0/">
<meta name="dc.format" content="text/html">
<meta name="dc.language" content="en">

<meta name="prism.publicationName" content="BMC Bioinformatics">
<meta name="prism.publicationDate" content="2007-6-19">
<meta name="prism.volume" content="8">
<meta name="prism.number" content="1">
<meta name="prism.section" content="Research article">
<meta name="prism.startingPage" content="210">



<link rel="alternate" type="application/pdf" title="PDF" href="http://www.biomedcentral.com/content/pdf/1471-2105-8-210.pdf">
<link rel="alternate" type="text/xml" title="XML version" href="http://www.biomedcentral.com/content/xml/1471-2105-8-210.xml">
<link rel="stylesheet" type="text/css" href="index.css" media="all">
</head>
<body alink="#6699ff" link="#003399" vlink="#3366cc"><!--skip nav start--><div class="skip"><a href="#maincontent">Skip to main content</a></div><table align="center" border="0" cellpadding="0" cellspacing="0" width="100%"><tbody><tr><td class="topad" align="center">

<center>

<!------ OAS SETUP begin ------>



<!------ OAS SETUP end ------>
<!------ OAS AD 'Top' begin ------>
<a href="http://oas.biomedcentral.com/5c/www.biomedcentral.com/article/bmcbioinformatics/1615400274/Top/biomed/house_BIL_jan08_leaderboard/Biology_Image_Library_Leaderboard_v2_gif_design_01_v8.gif/63613036663366633436653634656530" target="_top"><img src="biology_image_library_leaderboard_v2_gif_design_01_v8.gif" alt="" border="0" height="90" width="728"></a>
 <a href="http://www.biomedcentral.com/info/mediainformation/"><img src="advhere.gif" alt="" border="0"></a>
<!------ OAS AD end ------>
</center>

</td></tr></tbody></table><table border="0" cellpadding="5" cellspacing="0" width="100%"><tbody><tr valign="bottom"><td width="50"><a href="http://www.biomedcentral.com/bmcbioinformatics"><img src="logo.gif" alt="" border="0" hspace="3"></a></td><td align="left"><a href="http://www.biomedcentral.com/info/about/faq?name=impactfactor"><img src="1002.gif" alt="" border="0" hspace="3"></a></td><td id="topnav" align="right"><strong>Welcome </strong>Bioinformatics Institute Singapore (<a href="http://www.biomedcentral.com/registration/">Log on / register</a>)<br><a href="http://www.biomedcentral.com/support/feedback/" onclick="popup(this.href,'Feedback',650,500); return false;">Feedback</a> | <a href="http://www.biomedcentral.com/support/">Support</a></td></tr></tbody></table><table border="0" cellpadding="0" cellspacing="0" width="100%"><tbody><tr valign="top"><td colspan="3" bgcolor="#003399"><div id="TopBar"><font color="#ffffff">&nbsp;<a href="http://www.biomedcentral.com/home/" class="xtoplinks"><b> home </b></a>|<a href="http://www.biomedcentral.com/browse/journals/" class="xtoplinks"><b> journals A-Z </b></a>|<a href="http://www.biomedcentral.com/browse/bysubject/" class="xtoplinks"><b> subject areas </b></a>|<a href="http://www.biomedcentral.com/search/" class="xtoplinks"><b> advanced search </b></a>|<a href="http://www.biomedcentral.com/info/authors/" class="xtoplinks"><b> authors </b></a>|<a href="http://www.biomedcentral.com/info/about/peerreview/" class="xtoplinks"><b> reviewers </b></a>|<a href="http://www.biomedcentral.com/info/libraries" class="xtoplinks"><b> libraries </b></a>|<a href="http://careers.the-scientist.com/" class="xtoplinks"><b> jobs </b></a>|<a href="http://www.biomedcentral.com/info/" class="xtoplinks"><b> about </b></a>|<a href="http://www.biomedcentral.com/my/" class="xtoplinks"><b> my BioMed Central </b></a></font></div></td></tr></tbody></table><div id="article"><table cellpadding="0" cellspacing="0"><tbody valign="top"><tr><td id="leftcolumn"><ul style="position: fixed; top: 0px; left: 89px;" id="outline">
<li><a href="#">Top</a></li>
<li><a href="#IDA5NMIN">Abstract</a></li>
<li><a href="#IDAMNMIN">Background</a></li><li><a href="#IDAUMBTD">Results and discussion</a></li><li><a href="#IDAO3CON">Conclusion</a></li><li><a href="#IDAS3CON">Methods</a></li><li class="long"><a href="#IDAHRXPCB" class="clipped" onmouseover="outlineover(this)" onmouseout="outlineout(this)" title="Reproducible research">Reproducible ...</a></li>
<li><a href="#IDAVRXPCB">Authors' contributions</a></li><li><a href="#IDA1RXPCB">Acknowledgements</a></li>
<li class="lowest"><a href="#refs">References</a></li>
</ul>
<table width="120"><tbody><tr><td>&nbsp;</td></tr></tbody></table></td><td id="content"><div id="navbox">
<table cellpadding="5" cellspacing="2">
<tbody><tr>
<td><a href="http://www.biomedcentral.com/bmcbioinformatics"><strong>BMC Bioinformatics</strong></a><br><a href="http://www.biomedcentral.com/bmcbioinformatics/8">Volume 8</a><br></td>
</tr><tr><td><strong>Viewing options:</strong><ul>
<li>&nbsp;<a href="http://www.biomedcentral.com/1471-2105/8/210/abstract/">Abstract</a></li>
<li>&nbsp;<strong>Full text</strong></li>
<li>&nbsp;<a href="http://www.biomedcentral.com/content/pdf/1471-2105-8-210.pdf">PDF </a>
		(377KB)
				</li>
<li>&nbsp;<a href="http://www.biomedcentral.com/1471-2105/8/210/additional/">Additional Files</a></li>
</ul><strong>Associated material:</strong><ul><li>&nbsp;<a href="http://www.biomedcentral.com/1471-2105/8/210/comments">Readers' comments</a></li><li>&nbsp;<a href="http://www.biomedcentral.com/pubmed/17578580" onclick="largepopup(this.href,'pubmed');return false">PubMed record</a></li></ul><strong>Related literature:</strong><ul><li>&nbsp;Articles citing this article<div><a href="http://scholar.google.com/scholar?hl=en&amp;lr=&amp;safe=off&amp;c2coff=1&amp;q=link%3Ahttp://www.biomedcentral.com/1471-2105/8/210&amp;btnG=Search" onclick="largepopup(this.href,'on_google');return false">on Google Scholar</a></div><div><a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcAuth=BMC_sci&amp;SrcApp=BMC_sci&amp;DestLinkType=FullRecord&amp;DestApp=WOS&amp;KeyUT=000248499400001" onclick="largepopup(this.href,'on_isi');return false">on&nbsp;ISI&nbsp;Web&nbsp;of&nbsp;Science</a></div><div><a href="http://www.biomedcentral.com/search/citedby/pmc/17578580">on PubMed Central</a></div></li>
<li>&nbsp;Other articles by authors<br><div><a href="javascript://" onclick="toggleauthors('authg','authbuttong'); return false"><img src="expand2.gif" id="authbuttong" alt="">on Google Scholar
				</a></div>
<div style="display: none;" id="authg"><a href="http://scholar.google.com/scholar?q=author%3A%22A+Chebira%22">Chebira A</a><br><a href="http://scholar.google.com/scholar?q=author%3A%22Y+Barbotin%22">Barbotin Y</a><br><a href="http://scholar.google.com/scholar?q=author%3A%22C+Jackson%22">Jackson C</a><br><a href="http://scholar.google.com/scholar?q=author%3A%22T+Merryman%22">Merryman T</a><br><a href="http://scholar.google.com/scholar?q=author%3A%22G+Srinivasa%22">Srinivasa G</a><br><a href="http://scholar.google.com/scholar?q=author%3A%22RF+Murphy%22">Murphy RF</a><br><a href="http://scholar.google.com/scholar?q=author%3A%22J+Kovacevic%22">Kovačević J</a></div>
<div><a href="javascript://" onclick="toggleauthors('authpm','authbuttonpm'); return false"><img src="expand2.gif" id="authbuttonpm" alt="">on PubMed
				</a></div>
<div style="display: none;" id="authpm"><a href="http://www.biomedcentral.com/search/results.asp?db=pm&amp;terms=Chebira_A&amp;field=AU" onclick="largepopup(this.href,'pubmed',800,600);return false">Chebira A</a><br><a href="http://www.biomedcentral.com/search/results.asp?db=pm&amp;terms=Barbotin_Y&amp;field=AU" onclick="largepopup(this.href,'pubmed',800,600);return false">Barbotin Y</a><br><a href="http://www.biomedcentral.com/search/results.asp?db=pm&amp;terms=Jackson_C&amp;field=AU" onclick="largepopup(this.href,'pubmed',800,600);return false">Jackson C</a><br><a href="http://www.biomedcentral.com/search/results.asp?db=pm&amp;terms=Merryman_T&amp;field=AU" onclick="largepopup(this.href,'pubmed',800,600);return false">Merryman T</a><br><a href="http://www.biomedcentral.com/search/results.asp?db=pm&amp;terms=Srinivasa_G&amp;field=AU" onclick="largepopup(this.href,'pubmed',800,600);return false">Srinivasa G</a><br><a href="http://www.biomedcentral.com/search/results.asp?db=pm&amp;terms=Murphy_RF&amp;field=AU" onclick="largepopup(this.href,'pubmed',800,600);return false">Murphy RF</a><br><a href="http://www.biomedcentral.com/search/results.asp?db=pm&amp;terms=Kovacevic_J&amp;field=AU" onclick="largepopup(this.href,'pubmed',800,600);return false">Kovačević J</a></div>
</li>
<li>&nbsp;Related articles/pages<br><div><a href="http://www.google.co.uk/search?hl=en&amp;q=related:http://www.biomedcentral.com/1471-2105/8/210">on Google</a></div>
<div><a href="http://scholar.google.com/scholar?q=related:http://www.biomedcentral.com/1471-2105/8/210">on Google Scholar</a></div>
<div><a href="http://www.biomedcentral.com/pubmed/related/17578580">on PubMed</a></div></li></ul><strong>Tools:</strong><ul><li>&nbsp;<a href="http://www.biomedcentral.com/1471-2105/8/210/citation">Download references</a></li><li>&nbsp;<a href="http://www.biomedcentral.com/content/download/xml/1471-2105-8-210.xml">Download XML</a></li><li>&nbsp;<a href="http://www.biomedcentral.com/1471-2105/8/210/email">Email to a friend</a></li><li>&nbsp;<a href="http://www.biomedcentral.com/info/about/reprintsandperm">Order reprints</a></li><li>&nbsp;<a href="http://www.biomedcentral.com/1471-2105/8/210/postcomment">Post a comment</a></li>
<li>&nbsp;<a href="http://www.biomedcentral.com/bmcbioinformatics/alerts/">Sign up for article alerts</a></li></ul><strong>Post to:</strong><br><ul class="iconlist">
<li><a href="http://www.citeulike.org/posturl?url=http://www.biomedcentral.com/1471-2105/8/210" onclick="largepopup(this.href,'citeulike_popup_post',800,600);return false"><img alt="" src="citeulink.gif"></a>&nbsp;<a href="http://www.citeulike.org/posturl?url=http://www.biomedcentral.com/1471-2105/8/210" onclick="largepopup(this.href,'citeulike_popup_post',800,600);return false">Citeulike				
				</a></li>
<li><a onclick="largepopup(this.href,'Connotea',800,600);return false" href="http://www.connotea.org/addpopup?continue=confirm&amp;uri=http://dx.doi.org/doi:10.1186/1471-2105-8-210"><img alt="" src="connotea.gif"></a>&nbsp;<a href="http://www.connotea.org/addpopup?continue=confirm&amp;uri=http://dx.doi.org/doi:10.1186/1471-2105-8-210" onclick="largepopup(this.href,'Connotea',800,600);return false">Connotea
					</a></li>
<li><a href="http://del.icio.us/post?url=http://www.biomedcentral.com/1471-2105/8/210&amp;title=A+multiresolution+approach+to+automated+classification+of+protein+subcellular+location+images" onclick="largepopup(this.href,'Del.icio.us',800,600);return false"><img alt="" src="delicious.gif"></a>&nbsp;<a href="http://del.icio.us/post?url=http://www.biomedcentral.com/1471-2105/8/210&amp;title=A+multiresolution+approach+to+automated+classification+of+protein+subcellular+location+images" onclick="largepopup(this.href,'Del.icio.us',800,600);return false">Del.icio.us</a></li>
<li><a href="http://digg.com/submit?phase=2&amp;url=http://www.biomedcentral.com/1471-2105/8/210&amp;title=A+multiresolution+approach+to+automated+classification+of+protein+subcellular+location+images&amp;topic=general_sciences" onclick="largepopup(this.href,'Digg',800,600);return false"><img alt="" src="digg.gif"></a>&nbsp;<a href="http://digg.com/submit?phase=2&amp;url=http://www.biomedcentral.com/1471-2105/8/210&amp;title=A+multiresolution+approach+to+automated+classification+of+protein+subcellular+location+images&amp;topic=general_sciences" onclick="largepopup(this.href,'Digg',800,600);return false">Digg
				</a></li>
<li><a href="http://www.facebook.com/sharer.php?u=http://www.biomedcentral.com/1471-2105/8/210" onclick="largepopup(this.href,'facebook',800,600);return false"><img alt="" src="facebook.gif"></a>&nbsp;<a href="http://www.facebook.com/sharer.php?u=http://www.biomedcentral.com/1471-2105/8/210" onclick="largepopup(this.href,'facebook',800,600);return false">Facebook</a></li>
</ul></td></tr></tbody></table>
</div><a name="maincontent"></a><div id="topmatter"><a name="text"></a><a href="http://www.biomedcentral.com/info/about/openaccess/"><img class="access" src="openaccess-large.gif" alt="Open Access" height="24" width="89"></a><span class="articletype">Research article</span><h1>A multiresolution approach to automated classification of protein subcellular location images</h1><p><strong>Amina Chebira</strong><sup>1</sup> <a href="http://www.biomedcentral.com/logon/logon.asp?msg=ce"><img src="email-ca.gif" title="Corresponding author" alt="email"></a>, <strong>Yann Barbotin</strong><sup>2</sup><sup>,4</sup> <a href="http://www.biomedcentral.com/logon/logon.asp?msg=ce"><img src="email.gif" alt="email" title="Email"></a>, <strong>Charles Jackson</strong><sup>1</sup> <a href="http://www.biomedcentral.com/logon/logon.asp?msg=ce"><img src="email.gif" alt="email" title="Email"></a>, <strong>Thomas Merryman</strong><sup>2</sup> <a href="http://www.biomedcentral.com/logon/logon.asp?msg=ce"><img src="email.gif" alt="email" title="Email"></a>, <strong>Gowri Srinivasa</strong><sup>1</sup> <a href="http://www.biomedcentral.com/logon/logon.asp?msg=ce"><img src="email.gif" alt="email" title="Email"></a>, <strong>Robert F Murphy</strong><sup>1</sup><sup>,3</sup> <a href="http://www.biomedcentral.com/logon/logon.asp?msg=ce"><img src="email.gif" alt="email" title="Email"></a> and <strong>Jelena Kovačević</strong><sup>1</sup><sup>,2</sup> <a href="http://www.biomedcentral.com/logon/logon.asp?msg=ce"><img src="email.gif" alt="email" title="Email"></a></p><sup>1</sup>Center for Bioimage Informatics and Dept. of Biomedical Engineering, Carnegie Mellon University, Pittsburgh, PA, USA<br><sup>2</sup>Dept. of Electrical and Computer Engineering, Carnegie Mellon University, Pittsburgh, PA, USA<br><sup>3</sup>Depts. of Biological Sciences and Machine Learning, Carnegie Mellon University, Pittsburgh, PA, USA<br><sup>4</sup>Dept. of Communication Systems, Swiss Federal Institute of Technology, Lausanne, Switzerland<br><p><img src="email_001.gif" border="0"> author email
					<img src="email-ca_001.gif"> corresponding author email
					</p><p><em>BMC Bioinformatics</em> 2007,
			 <strong>8</strong><strong>:</strong>210<span class="pseudotab">doi:10.1186/1471-2105-8-210</span></p><p>The electronic version of this article is the complete one and can be found online at: <a href="http://www.biomedcentral.com/1471-2105/8/210">http://www.biomedcentral.com/1471-2105/8/210</a></p>
<table cellpadding="0" cellspacing="0">
<tbody>
<tr>
<td>Received:</td>
<td>1&nbsp;February&nbsp;2007</td>
</tr>
<tr>
<td>Accepted:</td>
<td>19&nbsp;June&nbsp;2007</td>
</tr>
<tr>
<td>Published:</td>
<td>19&nbsp;June&nbsp;2007</td>
</tr>
</tbody>
</table>
<p>
						©
						 2007 Chebira et al; licensee BioMed Central Ltd.<br>This is an Open Access article distributed under the terms of the Creative Commons Attribution License (<a href="http://creativecommons.org/licenses/by/2.0">http://creativecommons.org/licenses/by/2.0</a>), which permits unrestricted use, distribution, and reproduction in any medium, provided the original work is properly cited.</p></div><a name="IDA5NMIN"></a><h3>Abstract</h3><h4>Background</h4>
<p>Fluorescence microscopy is widely used to determine the subcellular location of proteins. Efforts to determine location on a proteome-wide basis create a need for automated methods to analyze the resulting images. Over the past ten years, the feasibility of using machine learning methods to recognize all major subcellular location patterns has been convincingly demonstrated, using diverse feature sets and classifiers. On a well-studied data set of 2D HeLa single-cell images, the best performance to date, 91.5%, was obtained by including a set of multiresolution features. This demonstrates the value of multiresolution approaches to this important problem.</p>
<h4>Results</h4>
<p>We report here a novel approach for the classification of subcellular location patterns by classifying in multiresolution subspaces. Our system is able to work with any feature set and any classifier. It consists of multiresolution (MR) decomposition, followed by feature computation and classification in each MR subspace, yielding local decisions that are then combined into a global decision. With 26 texture features alone and a neural network classifier, we obtained an increase in accuracy on the 2D HeLa data set to 95.3%.</p>
<h4>Conclusion</h4>
<p>We demonstrate that the space-frequency localized information in the multiresolution subspaces adds significantly to the discriminative power of the system. Moreover, we show that a vastly reduced set of features is sufficient, consisting of our novel modified Haralick texture features. Our proposed system is general, allowing for any combinations of sets of features and any combination of classifiers.</p><a name="IDAMNMIN"></a><h3>Background</h3><h4>Automated interpretation of protein subcellular location</h4>
<p>Among the most important goals in biological sciences today is to understand the function of all proteins. One of the critical characteristics of a protein is its subcellular location, that is, its spatial distribution within the cell. Knowledge of the location of all proteins will be essential for building accurate models that capture and simulate cell behavior, and eventually can be expected to be useful for early diagnosis of disease and/or monitoring of therapeutic effectiveness. The most widely used method for determining protein subcellular location is fluorescence microscopy. Given that mammalian cells are believed to express tens of thousands of proteins, comprehensive analysis of protein location will require acquisition of numbers of images that are beyond our ability to analyze visually.</p>
<p>Fortunately, the feasibility of automated interpretation of subcellular patterns in fluorescence microscope images has been clearly demonstrated over the past ten years, initially by our group <a name="IDASKMIN"></a><a name="IDACNMIN"></a><a name="IDAENMIN"></a>[<a href="#B1">1-3</a>] and then by others <a name="IDA2MMIN"></a><a name="IDA1MMIN"></a><a name="IDATMMIN"></a>[<a href="#B4">4-6</a>]. The result is systems that can classify protein location patterns with well-characterized reliability and better sensitivity than human observers (for reviews, please see <a name="IDASMMIN"></a><a name="IDALMMIN"></a>[<a href="#B7">7</a>,<a href="#B8">8</a>]). The heart of such systems is a set of numerical features – Subcellular Location Features (SLFs) – to describe the spatial distribution of proteins in each cell image. The SLFs include Haralick texture features, morphological features, and Zernike moments. Of particular relevance to the work described here is that the addition of simple multiresolution features resulted in a significant improvement of classification accuracy, to the highest reported accuracy of 91.5% for the 2D HeLa data set. This dataset contains images of all major subcellular patterns and is a well-established testbed for evaluating subcellular pattern analysis approaches. Note that with the aid of a parallel DNA channel, that accuracy climbed to 92.0%. It is important to have methods that work well when DNA images are available and also when they are not. We focus here on analysis of patterns without parallel DNA images and on improving performance relative to the best previous results.</p>
<h4>Multiresolution techniques</h4>
<p>As the introduction of the simplest <em>multiresolution (MR) </em>features produced a statistically significant jump in classification accuracy, our aim is to explore more sophisticated multiresolution techniques. In particular, the following are the three characteristics of multiresolution <a name="IDADMMIN"></a><a name="IDAEMMIN"></a>[<a href="#B9">9</a>,<a href="#B10">10</a>] we wish to explore: </p>
<p>(a) Localization: Fluorescence microscope images have highly localized structures both in space and frequency. This leads us to MR tools, as they have been found to be the most appropriate tools for computing and isolating such localized structures <a name="IDAKIMIN"></a>[<a href="#B11">11</a>].</p>
<p>(b) Adaptivity: Given that we are designing a system to distinguish between classes of proteins, it is clear that an ideal solution is to use adaptive transforms, a property provided by MR techniques. The reasoning is that if there is a different MR transform for each different class, then the transform itself would help in distinguishing the class.</p>
<p>(c) Fast and Efficient Computation: It is well known that MR techniques such as wavelets have a computational cost of the order <em>O</em>(<em>N</em>), where <em>N </em>is the input size, as opposed to <em>O</em>(<em>N </em>log <em>N</em>) typical for other linear transforms including the FFT. This is one of the major reasons for the phenomenal success of MR techniques in real applications and one of the reasons to incorporate MR features into the system. </p>
<p>MR transforms are many; we now give a brief overview. The basic idea behind MR is that we can look at a signal at different scales or resolutions and at different points in time. This should give us information about hidden structures in the signal, with a particular behavior across scales.</p>
<p>The main building block of any MR transform is a <em>filter </em>bank <a name="IDA4OMIN"></a>[<a href="#B10">10</a>]; it is a device that splits the signal into <em>MR subspaces </em>(also called subbands, wavelet coefficients or transform coefficients), where each MR subspace contains one part of the signal's spectrum. As an example, in the top part of Figure <a name="IDANPMIN"></a><a href="http://www.biomedcentral.com/1471-2105/8/210/figure/F1">1</a>, a two-channel filter bank is given, operating on a 1D signal. For images, on both rows and columns (horizontal and vertical directions), the signal is filtered, followed by downsampling by two (discarding every other sample, allowed because there is filtering beforehand). In the simplest case, this produces four subbands; one extracting lowpass information in both directions, one extracting highpass information in both direction and the remaining two extracting lowpass information in one direction and the highpass information in the other. The example in the lower part of the figure shows such a filter bank applied to each subband again (two levels), as we will use in this paper.</p><div class="figs"><div class="fig"><p><a href="http://www.biomedcentral.com/1471-2105/8/210/figure/F1"><img src="1471-2105-8-210-1.gif" alt="thumbnail" class="thumbnail" align="top"><strong>Figure 1.</strong></a> <strong>Basic multiresolution block</strong>. Top: Two-channel analysis filter bank. The filter <em>h </em>is a highpass filter and <em>g </em>is a lowpass filter. Bottom: A 2-level filter bank decomposition of actin. If the original image is of size <em>N </em>× <em>N</em>, the ones in the middle are of sizes <em>N</em>/2 × <em>N</em>/2 and the ones on the right are of sizes <em>N</em>/4 × <em>N</em>/4. Each branch has either the lowpass filter <em>g </em>or the highpass filter <em>h </em>followed by downsampling by 2 as in the top figure. Filtering and sampling are performed along the horizontal direction (rows) followed by the same operations along the vertical direction (columns).</p>
</div></div><p>Adaptivity of MR transforms manifests itself in many guises, including a number of popular transforms: (a) Growing a full tree to <em>L </em>levels with specific filters of the same length as the downsampling factor yields the Discrete Fourier Transform (DFT) of size 2<sup><em>L</em></sup>. (b) Growing a full tree to <em>L </em>levels but allowing the filters to be longer, leads to the Short-Time Fourier Transform, or, the Gabor Transform. (c) Growing the tree only on the lowpass branch to <em>L </em>levels leads to the <em>L</em>-level Discrete Wavelet Transform (DWT). (d) Growing an arbitrary tree leads to Wavelet Packets (WP). (e) Splitting the signal into more than two channels, allowing filters in the above transforms to be orthogonal and/or linear phase, allowing for true multidimensional filters and/or samplers, etc., leads to even more degrees of freedom.</p>
<h4>Towards multiresolution classification</h4>
<p>We now summarize our initial MR classification effort <a name="IDAZLBTD"></a><a name="IDAQLBTD"></a>[<a href="#B12">12</a>,<a href="#B13">13</a>]. We started with a simple classification system consisting of Haralick texture feature computation followed by a maximum-likelihood classifier, and demonstrated that, by adding an MR block in front, we were able to raise the classification accuracy by roughly 10% (from 71.8% to 82.2%) as compared to the system with no MR. This fits within our generic framework shown in Figure <a name="IDA2LBTD"></a><a href="http://www.biomedcentral.com/1471-2105/8/210/figure/F2">2</a>, where the feature computation block uses Haralick texture features and the classification block is maximum likelihood. We concluded that selecting features in MR subspaces allows us to custom-build discriminative feature sets. However, although the multiresolution block substantially increased classification accuracy, the accuracy of the overall system was still not high enough, and thus, in this work, we reexamined each step of the system: the features used, the classifier, and the weighting process.</p><div class="figs"><div class="fig"><p><a href="http://www.biomedcentral.com/1471-2105/8/210/figure/F2"><img src="1471-2105-8-210-2.gif" alt="thumbnail" class="thumbnail" align="top"><strong>Figure 2.</strong></a> <strong>Multiresolution (MR) classification system</strong>. The generic classification system (GCS) consists of feature extraction followed by classification (inside the dashed box). We add an MR block in front of GCS and compute features in MR subspaces (subbands). Classification is then performed on each of the subbands yielding local decisions which are then weighed and combined to give a final decision.</p>
</div></div><a name="IDAUMBTD"></a><h3>Results and discussion</h3><h4>Problem statement and philosophy</h4>
<p>The problem we are addressing is that of classifying the spatial distribution patterns of selected proteins within the cell. Assume that the images are of size <em>N </em>× <em>N </em>and let ℝ denote the set of intensities covered by all the images in the given dataset, compactly represented as an image belonging to ℝ<sup><em>N </em>× <em>N</em></sup>. Then, the problem can be formulated as designing a map from the <em>signal space </em>of protein localization images <a title="View MathML" href="http://www.biomedcentral.com/render/render.asp?equation=1471-2105-8-210-i1"><img class="mathimg" src="1471-2105-8-210-i1.gif" alt="Math"></a> ⊂ ℝ<sup><em>N </em>× <em>N</em></sup>, to a <em>response space </em><a title="View MathML" href="http://www.biomedcentral.com/render/render.asp?equation=1471-2105-8-210-i2"><img class="mathimg" src="1471-2105-8-210-i2.gif" alt="Math"></a>⊆{1, 2,..., <em>C</em>} of class labels. Thus, decision <em>d </em>is the map, <em>d</em>: <a title="View MathML" href="http://www.biomedcentral.com/render/render.asp?equation=1471-2105-8-210-i1"><img class="mathimg" src="1471-2105-8-210-i1.gif" alt="Math"></a> ↦ <a title="View MathML" href="http://www.biomedcentral.com/render/render.asp?equation=1471-2105-8-210-i2"><img class="mathimg" src="1471-2105-8-210-i2.gif" alt="Math"></a> that associates an input image with a class label <a name="IDA5OBTD"></a>[<a href="#B14">14</a>]. To reduce the dimensionality of the problem, one sets up a feature space <a title="View MathML" href="http://www.biomedcentral.com/render/render.asp?equation=1471-2105-8-210-i3"><img class="mathimg" src="1471-2105-8-210-i3.gif" alt="Math"></a>⊂ ℝ<sup><em>f</em></sup>, <em>f </em>≤ <em>N</em><sup>2</sup>, between the input space and the response space. The feature extractor <em><span class="entity">θ</span> </em>is the map <em><span class="entity">θ</span></em>:<a title="View MathML" href="http://www.biomedcentral.com/render/render.asp?equation=1471-2105-8-210-i1"><img class="mathimg" src="1471-2105-8-210-i1.gif" alt="Math"></a> ↦ <a title="View MathML" href="http://www.biomedcentral.com/render/render.asp?equation=1471-2105-8-210-i3"><img class="mathimg" src="1471-2105-8-210-i3.gif" alt="Math"></a>, and the classifier <em><span class="entity">ψ</span> </em>is the map <em><span class="entity">ψ</span></em>: <a title="View MathML" href="http://www.biomedcentral.com/render/render.asp?equation=1471-2105-8-210-i3"><img class="mathimg" src="1471-2105-8-210-i3.gif" alt="Math"></a> ↦ <a title="View MathML" href="http://www.biomedcentral.com/render/render.asp?equation=1471-2105-8-210-i2"><img class="mathimg" src="1471-2105-8-210-i2.gif" alt="Math"></a>. The goal is to find a (<em><span class="entity">θ</span></em>, <em><span class="entity">ψ</span></em>) pair that maximizes the classification accuracy.</p>
<p>To evaluate MR approaches, we use the well-characterized 2D HeLa set described previously <a name="IDAQJCON"></a>[<a href="#B3">3</a>]. The proteins in the data set were labeled using immunofluorescence, and thus, we know the ground truth, that is, which protein was labeled in each cell and subsequently imaged. This is useful for algorithm development as we can test the accuracy of classification schemes.</p>
<p>The challenge in this data set is that images from the same class may look different while those from different classes may look very similar (see Figure <a name="IDAWJCON"></a><a href="http://www.biomedcentral.com/1471-2105/8/210/figure/F2">2</a> in <a name="IDA1JCON"></a>[<a href="#B13">13</a>]). Based on the above discussion, we would like to extract discriminative features within space-frequency localized subspaces. These are obtained by MR decomposition; that is, instead of adding MR features as in <a name="IDAAKCON"></a>[<a href="#B15">15</a>], we compute features in the MR-decomposed subspaces. Thus, our system is a generic system with an MR decomposition block in front (see Figure <a name="IDAEKCON"></a><a href="http://www.biomedcentral.com/1471-2105/8/210/figure/F2">2</a>), followed by feature computation and classification in each of the subspaces. These are then combined through a weighting process. The hypothesis we test here is that adaptive classification in MR subspaces will improve the classification accuracy.</p>
<h4>Base system (nMR)</h4>
<p>We denote as <em>no MR </em>(<em>nMR</em>) the system consisting of the feature computation and the classifier blocks (see inside the dashed box in Figure <a name="IDARKCON"></a><a href="http://www.biomedcentral.com/1471-2105/8/210/figure/F2">2</a>). In our previous MR work, we used a maximum likelihood classifier that assumed the data to be well-separated Gaussian distributions, an assumption we found not to fit the data well. Instead, due to their simplicity and generality, we decided to use a two-layer neural network classifier. The first layer contains a node for each of the input features, each node using the Tan-Sigmoid transfer function. The second layer contains a node for each output and uses a linear transfer function (no hidden layers are used). We then train the neural network using a one-hot design, that is, since each output from the second layer corresponds to a class, when training, each training image will have an output of 1 for the class of which it is a member and a 0 for all other classes. To maximize the use of our data, our training process of the neural network block uses five-fold cross validation.</p>
<p>We ran the classifier with selected combinations of the three feature types used previously <a name="IDAYKCON"></a>[<a href="#B15">15</a>] (we did not use wavelet and Gabor features since they are inherently MR). These are: morphological features (<em>M</em>), Haralick texture features (<em>T</em><sub>1</sub>) and Zernike moment features (<em>Z</em>). The results obtained are given in the first row of Table <a name="IDADLCON"></a><a href="http://www.biomedcentral.com/1471-2105/8/210/table/T1">1</a>. We can see that the most powerful features on their own are texture features <em>T</em><sub>1</sub>, yielding 85.49% classification accuracy. Because of that, we looked into other texture feature sets, such as the second Haralick set from <a name="IDALLCON"></a>[<a href="#B16">16</a>] we termed <em>T</em><sub>2</sub>, which produced a slightly better result of 85.76% (second row of Table <a name="IDASLCON"></a><a href="http://www.biomedcentral.com/1471-2105/8/210/table/T1">1</a>). By examining these feature sets more closely, we developed a novel version of Haralick texture features, termed <em>T</em><sub>3 </sub>(details are given in <strong>Methods</strong>). With this set, the classification accuracy of the nMR system jumped to 87.46% (third row of Table <a name="IDA1LCON"></a><a href="http://www.biomedcentral.com/1471-2105/8/210/table/T1">1</a>). We also ran the experiment for all possible combinations of feature sets, as shown in the first three rows of Table <a name="IDA5LCON"></a><a href="http://www.biomedcentral.com/1471-2105/8/210/table/T1">1</a>. We found that, while the addition of other feature sets to <em>T</em><sub>3 </sub>did not increase the accuracy significantly, it did increase the number of features and thus computational complexity. This "flat" trend will turn out be general, as we will show later.</p><div class="figs"><div class="table">
<p><a href="http://www.biomedcentral.com/1471-2105/8/210/table/T1"><strong>Table 1.</strong></a> Classification accuracy per class. <em>Z</em>, <em>M </em>and <em>T </em>stand for Zernike, morphological and texture features. </p>
</div></div><h4>MR Bases (MRB)</h4>
<p>We now implement our main idea of adding an MR block in front of feature computation and classification, as in Figure <a name="IDAEYCON"></a><a href="http://www.biomedcentral.com/1471-2105/8/210/figure/F2">2</a>. We start with the MR decomposition being a basis expansion (details are given in <strong>Methods</strong>). We grow a full tree to two levels with Haar filters (see the bottom part of Figure <a name="IDAKYCON"></a><a href="http://www.biomedcentral.com/1471-2105/8/210/figure/F1">1</a>). We then test the system with all feature combinations, a neural network classifier as well as two versions of the weighting algorithm (open-form and closed-form, details are given in <strong>Methods</strong>).</p>
<p>The classifier is evaluated using nested cross validations (five-fold cross validation in the neural networks block and ten-fold during the weighting process). One problem with this technique is that the initial ordering of the images determines which images are grouped together for training and testing in each fold of the cross validation. A different original ordering of the images would result in different groupings, which would be equivalent to presenting different data sets to the classifier, and would thus result in a different overall result. We solve this problem by running multiple trials, each with a random initial ordering of the images. The mean result of these trials is taken as our true classification accuracy. In our experiments, we perform ten-fold cross validation on the weight calculation.</p>
<p>We note the following trends: (a) For all feature combinations, MRB significantly outperforms nMR, thus demonstrating that classifying in MR subspaces indeed improves classification accuracy. (b) For the two versions of the weighting algorithm, open form and closed form, the closed-form algorithm slightly outperforms the open-form one for all feature combinations except for <em>M </em>alone (fourth and fifth rows of Table <a name="IDAVYCON"></a><a href="http://www.biomedcentral.com/1471-2105/8/210/table/T1">1</a>). In particular, for texture features <em>T</em><sub>3</sub>, the accuracy rose slightly, from 91.82% to 92.32%. (c) While a slightly higher classification accuracy is obtained by using all three feature sets (92.54%) as well as both <em>T </em>and <em>M </em>(92.62%), the larger number of features and additional complexity of using <em>M </em>and <em>Z </em>features do not justify the slight improvement in accuracy (texture features <em>T</em><sub>3 </sub>alone achieve 92.32%). As for nMR, this "flat" trend is good news as we can use a significantly reduced feature set and still obtain a fairly high classification accuracy.</p>
<p>While we were satisfied that our hypothesis seems to be true, that is, classifying in MR subspaces increases classification accuracy significantly, we decided to look more closely into how we can improve the system even more. A known issue with MRB is that they are not shift invariant (rather, they are periodically shift invariant). This is due to downsampling used and can create problems as shifted versions of data can lead to different features in MR subspaces.</p>
<p>Our hypothesis is that shifts in the testing set produce reduced classification accuracy. We test this hypothesis by running the algorithm with <em>T</em><sub>3 </sub>features alone and with shifts of <em>t </em>= 0, 1, 2, 3 horizontally and vertically in the testing set (these shifts are chosen because we use 2 levels of the MR transform, so it is shift invariant to shifts of 2<sup>2</sup><em>t</em>, but not to shifts of 2<sup>2</sup><em>t </em>+ 1, 2<sup>2</sup><em>t </em>+ 2, 2<sup>2</sup><em>t </em>+ 3). As expected, the classification accuracy drops by 0.22%.</p>
<p>This experiment strongly indicates the use of MR techniques which are shift invariant (or almost shift invariant). These are called <em>frames </em>and we examine them next.</p>
<h4>MR Frames (MRF)</h4>
<p>The simplest MR frame which is completely shift-invariant is called à trous <a name="IDAF0CON"></a>[<a href="#B10">10</a>] and is obtained by removing downsamplers (which introduce shift variance) from the scheme. This leads to redundancy but avoids the problem of shift variance. The results of the experiments with MR frames (MRF) are given in the last two rows of Table <a name="IDAJ0CON"></a><a href="http://www.biomedcentral.com/1471-2105/8/210/table/T1">1</a> (for the two versions of the weighting algorithm again).</p>
<p>As for MRB, the three trends are similar: (a) MRF outperforms MRB (the only set showing no improvement is <em>Z </em>alone). (b) The closed-form algorithm outperforms the open-form one whenever <em>T</em><sub>3 </sub>set is involved. (c) Again, the "flat" trend continues: the difference between using <em>T</em><sub>3 </sub>only as opposed to <em>T</em><sub>3</sub>, <em>M </em>or all feature sets is so minor that the added number of features is not worth the complexity. We highlight these trends pictorially in Figure <a name="IDA20CON"></a><a href="http://www.biomedcentral.com/1471-2105/8/210/figure/F3">3</a>.</p><div class="figs"><div class="fig"><p><a href="http://www.biomedcentral.com/1471-2105/8/210/figure/F3"><img src="1471-2105-8-210-3.gif" alt="thumbnail" class="thumbnail" align="top"><strong>Figure 3.</strong></a> <strong>Pictorial representation of classification accuracy results</strong>. The diagram shows results from Table 1 for those sets involving <em>T</em><sub>3</sub>, namely (<em>T</em><sub>3</sub>), (<em>T</em><sub>3</sub>, <em>M</em>) and (<em>T</em><sub>3</sub>, <em>M</em>, <em>Z</em>). Diamond markers represent the nMR system (no MR block), circles represent the MRB system (MR bases, no redundancy) and squares represent the MRF system (MR frames, redundancy). Filled markers denote the closed-form weighting algorithm (CF), while empty ones denote the open-form weighting algorithm (OF). The following trends are noteworthy: (a) Introducing MR (both MRB and MRF) significantly outperforms nMR, thus demonstrating that classifying in MR subspaces indeed improves classification accuracy. (b) MRF outperform MRB. (b) For the two versions of the weighting algorithm, open form and closed form, the closed-form algorithm slightly outperforms the open-form one. (d) The trend in each case is almost flat across various feature set combinations, indicating that the texture set <em>T</em><sub>3 </sub>alone (26 features) is sufficient for high classification accuracy.</p>
</div></div><h4>Discussion and future work</h4>
<p>Classification of protein subcellular images was indeed significantly improved by classifying in MR subspaces. One reason for this improved performance over the system using the inherently MR features is that those features are simply energies in the subbands, while here, the features can be any suitable set, leading to a more general space of solutions. A reason for the improved performance of the MR systems over the nMR one could be intuitively understood if we assumed that this data set is highly "texture"-like. For example, it is possible for two different textures to have the same set of Haralick texture features (they have the same co-occurrence matrices), while when decomposed, even at the first level, their co-occurrence matrices would be different, leading to different Haralick texture features, and thus discriminative power. An example of this is given in the compendium to the paper (see Additional file <a name="IDAG2CON"></a><a href="http://www.biomedcentral.com/1471-2105/8/210/suppl/S1">1</a> and <a name="IDAL2CON"></a>[<a href="#B17">17</a>]).</p><div class="figs"><div class="addfile">
<p><a href="http://www.biomedcentral.com/1471-2105/8/210/suppl/S1"></a><strong>Additional file 1.</strong> Compendium. 07_ChebiraBJMSMK_compendium.zip. This file is a compressed archive that contains the code that generated the results in this paper, the pseudo-code for the weighting algorithms, Table <a name="IDAW2CON"></a><a href="http://www.biomedcentral.com/1471-2105/8/210/table/T1">1</a> with detailed results and index files of the web site containing all of this material <a name="IDA12CON"></a>[<a href="#B17">17</a>].</p><p>
	Format: ZIP 
	Size: 474KB <a href="http://www.biomedcentral.com/content/supplementary/1471-2105-8-210-s1.zip">Download file</a> or <a href="http://www.biomedcentral.com/content/supplementary/1471-2105-8-210-s1/" onclick="largepopup(this.href,'mini_website',800,600);return false"> display content in a new window</a></p>
</div></div><p>We plan on exploring a number of issues in our future work. (a) For example, our system effectively builds an adapted MR decomposition (via subband weights) for the whole data set; we want to adapt that decomposition to each class, arguing that a different MR decomposition for a different class would be a discriminative feature in itself. We are currently working on this by adapting the closed-form algorithm. (b) We would also like to explore whether improved performance can be obtained by incorporating feature selection methods during classifier training for each subband, as was done in the original work in <a name="IDAF3CON"></a>[<a href="#B15">15</a>]. (c) It will also be of interest to explore how and whether to include information from parallel DNA images, since this information improved nMR-based classification accuracy in <a name="IDAK3CON"></a>[<a href="#B15">15</a>] from 91.5% to 92.0%. This improvement is because the parallel DNA image provides a frame of reference for distinguishing proteins that are inside or near the nucleus from those with similar patterns that are not. (d) Lastly, we would like to find a cost function that would allow us to explicitly build wavelet packets. While we implicitly do this now using weights, it would lead to improved computational efficiency if we had a method for building a subtree as opposed to using all the subbands.</p><a name="IDAO3CON"></a><h3>Conclusion</h3><p>This paper addresses automated and robust classification of major protein subcellular location patterns. With the introduction of a multiresolution approach, we are able to obtain a high classification accuracy of 95.26% with only 26 texture features, proving that adaptive MR techniques improve the classification of the 2D HeLa data set.</p><a name="IDAS3CON"></a><h3>Methods</h3><h4>Data set</h4>
<p>We used the collection of 2D images of HeLa cells described previously <a name="IDA13CON"></a>[<a href="#B3">3</a>] and publicly available <a name="IDAA4CON"></a>[<a href="#B18">18</a>]. It contains approximately 90 single-cell images of size 512 × 512, in each of <em>C </em>= 10 classes. The 10 classes of subcellular location patterns were obtained by labeling an endoplasmic reticulum protein, two Golgi proteins (giantin and gpp130), a lysosomal protein, a mitochondrial protein, a nucleolar protein, two cytoskeletal proteins (actin and tubulin), an endosomal protein, and DNA. The best previously described overall classification accuracy on this data set, without the use of the parallel DNA channel, is 91.5% <a name="IDAH4CON"></a>[<a href="#B15">15</a>].</p>
<h4>Base system (nMR)</h4>
<h4>Feature Sets</h4><p>As in <a name="IDAU4CON"></a>[<a href="#B15">15</a>], we start with Haralick texture features (set <em>T</em><sub>1</sub>, 13 features), morphological (set <em>M</em>, 16 features) and Zernike moments (set <em>Z</em>, 49 features). Unlike in <a name="IDAA5CON"></a>[<a href="#B15">15</a>], we do not use wavelet/Gabor features because the MR advantage given by these will be achieved by our MR decomposition. Therefore, our total number of features is 78, as opposed to 174 in <a name="IDAF5CON"></a>[<a href="#B15">15</a>].</p>
<h4>MR classification</h4>
<p>We argued at the beginning of the paper that the nature of our data sets requires tools which offer localization in space and frequency as well as adaptivity, and we further argued that those tools are MR in nature. Thus, the novelty here is classifying in MR subspaces as opposed on the original image itself. The idea is that certain features will react well at a certain scale but not at another. Thus, we add an <em>MR </em>block in front of the standard feature extraction and classification blocks, as in Figure <a name="IDAQ5CON"></a><a href="http://www.biomedcentral.com/1471-2105/8/210/figure/F2">2</a>.</p>
<h4>MR block</h4><p>The basic MR block is the so-called <em>two-channel filter bank </em>(see top part of Figure <a name="IDA15CON"></a><a href="http://www.biomedcentral.com/1471-2105/8/210/figure/F1">1</a>). It, and its extensions, can be used to build decompositions custom-tailored to the image at hand. This is done by using this filter bank in a tree, iterating on any of the two-channels and its children. Moreover, the filter bank can have more than two channels, and can have more channels than the sampling factor (leading to redundant representations), etc.</p>
<p>Amongst the possible trees that one can use to analyze an image, the wavelet packets mentioned previously <a name="IDAFADON"></a>[<a href="#B19">19</a>] adapt themselves to the signal at hand. However, this is possible only if a suitable <em>cost function </em>is available. That is, in order to adaptively build the tree, we need to find a suitable "measure" that will indicate whether a subband (a node in the tree) contains useful information or not. If it does, then we keep the node, otherwise, we prune it.</p>
<p>Adaptive flavors of MR have been explored for their utility in classification in various domains <a name="IDAOADON"></a>[<a href="#B20">20</a>]. These studies have used the transform domain coefficients themselves as features and so had a natural cost function in selecting the tree most adapted to the signal. In <a name="IDATADON"></a>[<a href="#B21">21</a>], we used wavelet packets for fingerprint identification and verification with remarkable results.</p>
<p>To get a fairly general set of possibilities MR toolbox offers, we define the following matrix</p>
<p class="inlinenumber"><em>MR</em><sub><em>l</em>,<em>b</em>,<em>m</em>,<em>n </em></sub>= [<em><span class="entity">ψ</span></em><sub><em>l</em>,<em>b</em>,<em>o</em>,<em>p</em></sub>],<span>(1)</span></p>
<p>whose elements <em><span class="entity">ψ</span></em><sub><em>l</em>,<em>b</em>,<em>o</em>,<em>p </em></sub>go from <em>o </em>= 0, ..., <em>m </em>- 1 and <em>p </em>= 0, ..., <em>n </em>- 1. In the above, <em>l </em>denotes the level at which the block is applied, <em>b </em>denotes the particular child branch to which it is applied, <em>m </em>denotes the number of channels and <em>n </em>the sampling factor.</p>
<p>If <em>m </em>= <em>n </em>for every block, the above transforms would implement a basis (nonredundant) expansion. If at least for one block <em>m </em>&gt; <em>n</em>, the resulting decomposition is a frame and is redundant. The standard discrete dyadic wavelet transform is obtained with <em>m </em>= <em>n </em>= 2 and the MR block being applied only to the first branch of every preceding block.</p>
<p>As an example, we will show how our MR basis with Haar filters is represented. We have <em>m </em>= <em>n </em>= 2, and assume that we have a 1D system (a 2D system is obtained by applying the 1D one on rows and then on columns) with 1 level. Thus, <em>l </em>= 1 and the block is applied to the input branch only (we number that branch 0). The corresponding matrix is:</p>
<p class="inlinenumber"><a title="View MathML" href="http://www.biomedcentral.com/render/render.asp?equation=1471-2105-8-210-i4"><img class="mathimg" src="1471-2105-8-210-i4.gif" alt="Math"></a><span>(2)</span></p>
<p>where the lowpass filter is given in the first column (roughly averaging) and the highpass is given in the second column (roughly differencing). This matrix describes the operation of the system on the input of length 2. Since the input sequence is in general infinite, the matrix describing the operation of the system on such a sequence is given by</p>
<p class="inlinenumber"><a title="View MathML" href="http://www.biomedcentral.com/render/render.asp?equation=1471-2105-8-210-i5"><img class="mathimg" src="1471-2105-8-210-i5.gif" alt="Math"></a><span>(3)</span></p>
<p>The effect of downsampling is seen in the movement of the block <em>MR</em><sub>1 </sub>by 2 each time (if the downsampling were not present, the blocks would be moving by 1).</p>
<p>For <em>l </em>= 2 levels, and using the same transform <em>MR</em><sub>1 </sub>at each branch and at each level, we obtain the matrix <em>MR</em><sub>2</sub>:</p>
<p class="inlinenumber"><a title="View MathML" href="http://www.biomedcentral.com/render/render.asp?equation=1471-2105-8-210-i6"><img class="mathimg" src="1471-2105-8-210-i6.gif" alt="Math"></a><span>(4)</span></p>
<p>where ⊗ denotes the Kronecker product. The matrix describing the operation of the whole system is now an infinite block-diagonal matrix with <em>MR</em><sub>2 </sub>on the diagonal (similarly to (3)).</p>
<h4>Feature extraction block</h4><p>Instead of combining all features into a single probability vector, we allow each feature set its own probability vector per subband. For example, for 2 levels as in (4), we have a total of 21 subbands (original image + 4 subbands at the first level + 16 subbands at the second level), effectively bringing the number of subbands to 3·<em>S </em>= 3·21 = 63 if all three feature sets are used, where <em>S </em>is the number of subbands per level. Note that although we have decreased the number of features significantly, we have also increased the number of classifiers, because we now have one classifier per subband. Evaluating this computational trade-off is a task for future work.</p>
<h4>New texture feature set <em>T</em><sub>3</sub></h4><p>As the Haralick texture features seem to possess the most discriminative power, we looked more closely into these. Haralick texture features are calculated using four co-occurrence matrices <a name="IDARKDON"></a>[<a href="#B16">16</a>]: 1) <em>P</em><sub><em>H </em></sub>(horizontal nearest neighbors), 2) <em>P</em><sub><em>V </em></sub>(vertical nearest neighbors), 3) <em>P</em><sub><em>LD </em></sub>(left diagonal nearest neighbors), and 4) <em>P</em><sub><em>RD </em></sub>(right diagonal nearest neighbors). We now calculate 13 measures on each of these four matrices, as defined by Haralick. For example, the first two features on <em>P</em><sub><em>H </em></sub>are:</p>
<p class="inlinenumber"><a title="View MathML" href="http://www.biomedcentral.com/render/render.asp?equation=1471-2105-8-210-i7"><img class="mathimg" src="1471-2105-8-210-i7.gif" alt="Math"></a><span>(5)</span></p>
<p class="inlinenumber"><a title="View MathML" href="http://www.biomedcentral.com/render/render.asp?equation=1471-2105-8-210-i8"><img class="mathimg" src="1471-2105-8-210-i8.gif" alt="Math"></a><span>(6)</span></p>
<p>where <em>N</em><sub><em>g </em></sub>is the number of gray levels in the image and <em>R</em><sub><em>H </em></sub>is a normalizing constant equal to the sum of all the elements in <em>P</em><sub><em>H</em></sub>.</p>
<p>The other measures follow in similar fashion, giving us four sets of 13 measures: <em>f</em><sub>(<em>H</em>, 1–13), </sub><em>f</em><sub>(<em>V</em>, 1–13), </sub><em>f</em><sub>(<em>LD</em>, 1–13) </sub>and <em>f</em><sub>(<em>RD</em>, 1–13)</sub>. Haralick's original method reduces these to a single set of 13 by calculating the mean of each measure across the four sets (feature set <em>T</em><sub>1</sub>):</p>
<p class="inlinenumber"><a title="View MathML" href="http://www.biomedcentral.com/render/render.asp?equation=1471-2105-8-210-i9"><img class="mathimg" src="1471-2105-8-210-i9.gif" alt="Math"></a></p>
<p>for <em>i </em>= 1, . . ., 13. An alternative method <a name="IDA3SDON"></a>[<a href="#B16">16</a>] that we have used previously <a name="IDACTDON"></a>[<a href="#B22">22</a>], is to use both the mean and the range of the 13 measures, thus resulting in two sets of 13 features (26 features overall, feature set <em>T</em><sub>2</sub>). </p>
<p>We can significantly improve upon these results by making a change in the way that we combine our initial four sets of features. We note that <em>P</em><sub><em>H </em></sub>and <em>P</em><sub><em>V </em></sub>are fundamentally different from <em>P</em><sub><em>LD </em></sub>and <em>P</em><sub><em>HD </em></sub>because adjacent neighboring pixels are spatially closer than diagonal neighboring pixels. Therefore, instead of averaging the features from all four sets, we create our first set of 13 features by averaging <em>f</em><sub>(<em>H</em>,<em>i</em>) </sub>and <em>f</em><sub>(<em>V</em>,<em>i</em>)</sub>, and a second set of 13 features by averaging <em>f</em><sub>(<em>LD</em>,<em>i</em>) </sub>and <em>f</em><sub>(<em>RD</em>,<em>i</em>)</sub>. Thus, we end up with two sets of 13 features, which are concatenated into one feature set (<em>T</em><sub>3</sub>) of 26 features:</p>
<p class="inlinenumber"><a title="View MathML" href="http://www.biomedcentral.com/render/render.asp?equation=1471-2105-8-210-i10"><img class="mathimg" src="1471-2105-8-210-i10.gif" alt="Math"></a></p>
<p>for <em>i </em>= 1, ..., 13.</p>
<h4>Weighting algorithms</h4><p>Figure <a name="IDA0XDON"></a><a href="http://www.biomedcentral.com/1471-2105/8/210/figure/F2">2</a> shows a graphical representation of a generic MR classification system, including the process of combining all of the subband decisions into one. We use weights for each subband to adjust the importance that a particular subband has on the overall decision made by the classification system. If the weights are chosen such that the no decomposition weight is equal to 1, and all other weights are 0, we will achieve the same output vector as we would have without using the adaptive MR system. Therefore, we know that there exists a weight combination that will do at least as well as the generic classifier (when no MR is involved) in the training phase. Our goal is to decide how to find the weight vector that achieves the highest overall classification accuracy on a given data set. We developed two versions of the weighting algorithm: open-form and closed-form.</p>
<p>The difference between the open- and closed-form algorithms is that in the open-form version we optimize classification accuracy on the training set as opposed to the closed-form where we look for the least-squares solution. The open-form algorithm for the training and the testing phases are given in <a name="IDAEYDON"></a>[<a href="#B13">13</a>] under Algorithms 1 and 2, respectively.</p>
<p>The neural network block outputs a series of decision vectors for each subband of each training image. Each decision vector <a title="View MathML" href="http://www.biomedcentral.com/render/render.asp?equation=1471-2105-8-210-i11"><img class="mathimg" src="1471-2105-8-210-i11.gif" alt="Math"></a> contains <em>C </em>= 10 numbers (because we have 10 classes) that correspond to the "local" decisions made by the subband <em>s </em>for a specific image <em>r</em>.</p>
<h4>Open-form algorithm</h4><p>If using the open-form algorithm, we initialize all the weights (see Algorithm 1 in <a name="IDALZDON"></a>[<a href="#B13">13</a>] for details), and a global decision vector is computed using a weighted sum of the local decisions. An initial class label will be given to an image using this global decision vector. If that class label is correct, we go to the next image. If it is incorrect, we look at the local decisions of each subband and adjust the weights of each subband <em>s </em>as follows:</p>
<p class="inlinenumber"><a title="View MathML" href="http://www.biomedcentral.com/render/render.asp?equation=1471-2105-8-210-i12"><img class="mathimg" src="1471-2105-8-210-i12.gif" alt="Math"></a></p>
<p>where <em>iter </em>is the iteration number and <em>ε </em>is a small positive constant. This can be viewed as a reward/punishment method where the subbands taking the correct decisions will have their weights increased, and those taking wrong decisions will have their weights decreased. We continue cycling through the images until there is no increase in classification accuracy on the training set for a given number of iterations.</p>
<h4>Closed-form algorithm</h4><p>The closed-form solution does not use an iterative algorithm. Instead, it finds the weight vector by solving a minimization problem in the least-square sense. We now explain how this is accomplished.</p>
<p>Assume that we have <em>R </em>training images. For each training image <em>r </em>= 1, . . ., <em>R</em>, the vector <a title="View MathML" href="http://www.biomedcentral.com/render/render.asp?equation=1471-2105-8-210-i13"><img class="mathimg" src="1471-2105-8-210-i13.gif" alt="Math"></a>, is the <em>C </em>× 1 decision vector at the output of each subband classifier <em>s</em>, where <a title="View MathML" href="http://www.biomedcentral.com/render/render.asp?equation=1471-2105-8-210-i14"><img class="mathimg" src="1471-2105-8-210-i14.gif" alt="Math"></a> indicates the confidence of subband <em>s </em>that the training image <em>r </em>belongs to class <em>c</em>. For each training image <em>r</em>, the weighting block takes as input the subband (local) decision vectors <a title="View MathML" href="http://www.biomedcentral.com/render/render.asp?equation=1471-2105-8-210-i11"><img class="mathimg" src="1471-2105-8-210-i11.gif" alt="Math"></a> and combines them into a single output decision vector as follows:</p>
<p class="inlinenumber"><a title="View MathML" href="http://www.biomedcentral.com/render/render.asp?equation=1471-2105-8-210-i15"><img class="mathimg" src="1471-2105-8-210-i15.gif" alt="Math"></a></p>
<p>We can rewrite the above by, for each training image <em>r</em>, forming a matrix <em>D</em><sup>(<em>r</em>) </sup>of size <em>C </em>× <em>S</em>, where each element <a title="View MathML" href="http://www.biomedcentral.com/render/render.asp?equation=1471-2105-8-210-i16"><img class="mathimg" src="1471-2105-8-210-i16.gif" alt="Math"></a> is the value at position <em>c </em>of the decision vector <a title="View MathML" href="http://www.biomedcentral.com/render/render.asp?equation=1471-2105-8-210-i11"><img class="mathimg" src="1471-2105-8-210-i11.gif" alt="Math"></a> of subband classifier <em>s</em>. We can now compute:</p>
<p class="inlinenumber"><em>D</em><sup>(<em>r</em>)</sup><em>w</em>,</p>
<p>where <em>w </em>= (<em>w</em><sub>1</sub>, . . ., <em>w</em><sub><em>S</em></sub>)<sup><em>T </em></sup>is of size <em>S </em>× 1. Thus, we want to find a weight vector <em>w </em>common to all training images <em>r </em>= 1, . . ., <em>R</em>. A possible solution for <em>w </em>is the one that minimizes the error in the least-square sense:</p>
<p class="inlinenumber"><a title="View MathML" href="http://www.biomedcentral.com/render/render.asp?equation=1471-2105-8-210-i17"><img class="mathimg" src="1471-2105-8-210-i17.gif" alt="Math"></a><span>(7)</span></p>
<p>where <em>d</em><sup>(<em>r</em>) </sup>is the desired target decision vector of size <em>C </em>× 1. It has a 1 in the position of the true class, and 0 elsewhere.</p>
<p>We need to rewrite the above in a direct error-minimization form. We thus define a target output vector <em>d </em>of size <em>CR </em>× 1, as a vector which concatenates all the target decision vectors <em>d</em><sup>(<em>r</em>) </sup>as follows:</p>
<p class="inlinenumber"><a title="View MathML" href="http://www.biomedcentral.com/render/render.asp?equation=1471-2105-8-210-i18"><img class="mathimg" src="1471-2105-8-210-i18.gif" alt="Math"></a></p>
<p>and a <em>CR </em>× <em>S </em>matrix <em>D </em>consisting of the all the decision matrices <em>D</em><sup>(<em>r</em>) </sup>of all the training data stacked on top of each other:</p>
<p class="inlinenumber"><a title="View MathML" href="http://www.biomedcentral.com/render/render.asp?equation=1471-2105-8-210-i19"><img class="mathimg" src="1471-2105-8-210-i19.gif" alt="Math"></a></p>
<p>We can now rewrite (7) as:</p>
<p class="inlinenumber"><a title="View MathML" href="http://www.biomedcentral.com/render/render.asp?equation=1471-2105-8-210-i20"><img class="mathimg" src="1471-2105-8-210-i20.gif" alt="Math"></a></p>
<p>which possesses a closed-form solution and can be computed efficiently.</p>
<p>Then, for a testing image <em>t</em>, we compute its decision vector <em><span class="entity">δ</span> </em>= (<em><span class="entity">δ</span></em><sub>1</sub>, <em><span class="entity">δ</span></em><sub>2</sub>, . . ., <em><span class="entity">δ</span></em><sub><em>C</em></sub>) as follows:</p>
<p class="inlinenumber"><a title="View MathML" href="http://www.biomedcentral.com/render/render.asp?equation=1471-2105-8-210-i21"><img class="mathimg" src="1471-2105-8-210-i21.gif" alt="Math"></a></p>
<p>where <a title="View MathML" href="http://www.biomedcentral.com/render/render.asp?equation=1471-2105-8-210-i22"><img class="mathimg" src="1471-2105-8-210-i22.gif" alt="Math"></a> are the local decision vectors for <em>t</em>. The classification decision is then made as</p>
<p class="inlinenumber"><a title="View MathML" href="http://www.biomedcentral.com/render/render.asp?equation=1471-2105-8-210-i23"><img class="mathimg" src="1471-2105-8-210-i23.gif" alt="Math"></a></p>
<p>that is, the winning class corresponds to the index of the highest coefficient in <em><span class="entity">δ</span></em>.</p>
<h4>MR Bases</h4>
<p>Among all possible combinations given in (1), we now confine ourselves to those implementing bases, that is, the resulting decompositions are nonredundant. Thus, in each MR subblock, <em>m </em>= <em>n</em>.</p>
<p>We grow a full MR tree with 2 levels. The classification system uses all the subspaces from the root (the original image) to the leaves of the tree. Hence, the total number of subbands used is 21 (1 + 4 + 4<sup>2</sup>). We used the simplest, Haar filters in the decomposition, where the lowpass is given by <a title="View MathML" href="http://www.biomedcentral.com/render/render.asp?equation=1471-2105-8-210-i24"><img class="mathimg" src="1471-2105-8-210-i24.gif" alt="Math"></a> whereas the highpass is <a title="View MathML" href="http://www.biomedcentral.com/render/render.asp?equation=1471-2105-8-210-i25"><img class="mathimg" src="1471-2105-8-210-i25.gif" alt="Math"></a>. Given a 1D input sequence <em>x</em>, the MR transform we apply to each block of 4 elements (advancing each time by 4) is given by the matrix defined in (4). This is done first in the horizontal direction and then in the vertical one, producing 16 outputs (subbands). There are many other MRB blocks possible, the investigation of which is left for future work.</p>
<h4>MR Frames</h4>
<p>We now lift the restriction of no redundancy and allow <em>m </em>and <em>n </em>to be different (<em>m </em>&gt; <em>n</em>). The resulting decompositions are called frames <a name="IDA0QXPCB"></a>[<a href="#B23">23</a>].</p>
<p>We use again the full MR tree with 2 levels, but remove downsamplers, as in the à trous algorithm <a name="IDABRXPCB"></a>[<a href="#B10">10</a>]. Given a 1D input sequence <em>x</em>, the MR transform we apply to each block of 4 elements is identical to the one in (4) except that it is applied to every block of 4 elements (there is no downsampling). There are many other MRF blocks possible, the investigation of which is left for future work.</p><a name="IDAHRXPCB"></a><h3>Reproducible research</h3><p>All the material needed to reproduce results in this paper is available at the web site address <a name="IDANRXPCB"></a>[<a href="#B17">17</a>] and provided in the Additional file <a name="IDARRXPCB"></a><a href="http://www.biomedcentral.com/1471-2105/8/210/suppl/S1">1</a> as well.</p><a name="IDAVRXPCB"></a><h3>Authors' contributions</h3><p>AC coordinated the work, helped conceive the approach and helped extensively with writing the manuscript. YB was in charge of unifying the code and designing and running the experiments involving multiresolution and frames. CJ designed new texture features and helped extensively with writing the manuscript. TM conceived the enhanced multiresolution approach. GS was involved in the original multiresolution approach to classification, the discussions throughout, and helped extensively with writing the manuscript. RFM helped throughout with discussions and editing the manuscript. JK conceived the multiresolution approach and was fully involved in every step of the work, from designing the experiments through interpretation as well as drafting the manuscript. All of the authors read and approved the final manuscript.</p><a name="IDA1RXPCB"></a><h3>Acknowledgements</h3><p>We gratefully acknowledge the comments we received from the anonymous reviewers which greatly improved the quality of the manuscript. We are also indebted to our colleagues in the Center for Bioimage Informatics at Carnegie Mellon University for their help and support. This work was supported in part by NSF grants CCF-0515152 and EF-0331657, as well as the PA State Tobacco Settlement, Kamlet-Smith Bioinformatics Grant.</p><a name="refs"></a><h3>References</h3><ol id="references">
<li>
<p><a name="B1"></a> Boland M,  Markey M,  Murphy R:
	<strong>Classification of Protein Localization Patterns Obtained via Fluorescence Light Microscopy. </strong>In <em>Proc IEEE Int Conf EMBS Society</em>. Chicago, IL;  1997:594-597.  <a href="http://www.biomedcentral.com/sfx_links.asp?ui=1471-2105-8-210&amp;bibl=B1"><img src="sfx_links.asp" alt="OpenURL"></a></p><p><a href="javascript:turn();">Return to text</a></p>

</li>
<li>
<p><a name="B2"></a> Boland M,  Markey M,  Murphy R:
	<strong>Automated Recognition of Patterns Characteristic of Subcellular Structures in Fluorescence Microscopy Images. </strong></p><p><em>Cytometry</em> 1998,  <strong>33</strong><strong>:</strong>366-375. <a target="_top" href="http://www.biomedcentral.com/pubmed/9822349">PubMed Abstract</a> | <a target="_top" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=9822349">Publisher Full Text</a> |  <a href="http://www.biomedcentral.com/sfx_links.asp?ui=1471-2105-8-210&amp;bibl=B2"><img src="sfx_links.asp" alt="OpenURL"></a></p><p><a href="javascript:turn();">Return to text</a></p>

</li>
<li>
<p><a name="B3"></a> Boland M,  Murphy R:
	<strong>A neural network classifier capable of recognizing the patterns of all major subcellular structures in fluorescence microscope images of HeLa cells. </strong></p><p><em>Bioinformatics</em> 2001,  <strong>17</strong>(12)<strong>:</strong>1213-1223. <a target="_top" href="http://www.biomedcentral.com/pubmed/11751230">PubMed Abstract</a> | <a target="_top" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=11751230">Publisher Full Text</a> |  <a href="http://www.biomedcentral.com/sfx_links.asp?ui=1471-2105-8-210&amp;bibl=B3"><img src="sfx_links.asp" alt="OpenURL"></a></p><p><a href="javascript:turn();">Return to text</a></p>

</li>
<li>
<p><a name="B4"></a> Perner P,  Perner H,  Muller B:
	<strong>Mining Knowledge for Hep-2 Cell Image Classification. </strong></p><p><em>Journ Artificial Intelligence in Medicine</em> 2002,  <strong>26</strong>(1-2)<strong>:</strong>161-173.  <a href="http://www.biomedcentral.com/sfx_links.asp?ui=1471-2105-8-210&amp;bibl=B4"><img src="sfx_links.asp" alt="OpenURL"></a></p><p><a href="javascript:turn();">Return to text</a></p>

</li>
<li>
<p><a name="B5"></a> Danckaert A,  Gonzalez-Couto E,  Bollondi L,  Thompson N,  Hayes B:
	<strong>Automated Recognition of Intracellular Organelles in Confocal Microscope Images. </strong></p><p><em>Traffic</em> 2002,  <strong>3</strong><strong>:</strong>66-73. <a target="_top" href="http://www.biomedcentral.com/pubmed/11872144">PubMed Abstract</a> | <a target="_top" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=11872144">Publisher Full Text</a> |  <a href="http://www.biomedcentral.com/sfx_links.asp?ui=1471-2105-8-210&amp;bibl=B5"><img src="sfx_links.asp" alt="OpenURL"></a></p><p><a href="javascript:turn();">Return to text</a></p>

</li>
<li>
<p><a name="B6"></a> Conrad C,  Erfle H,  Warnat P,  Daigle N,  Lorch T,  Ellenberg J,  Pepperkok R,  Eils R:
	<strong>Automatic identification of subcellular phenotypes on human cell arrays. </strong></p><p><em>Genome Research</em> 2004,  <strong>14</strong><strong>:</strong>1130-1136. <a target="_top" href="http://www.biomedcentral.com/pubmed/15173118">PubMed Abstract</a> | <a target="_top" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=15173118">Publisher Full Text</a> | <a target="_top" href="http://www.pubmedcentral.nih.gov/articlerender.fcgi?tool=pubmed&amp;pubmedid=15173118">PubMed Central Full Text</a> |  <a href="http://www.biomedcentral.com/sfx_links.asp?ui=1471-2105-8-210&amp;bibl=B6"><img src="sfx_links.asp" alt="OpenURL"></a></p><p><a href="javascript:turn();">Return to text</a></p>

</li>
<li>
<p><a name="B7"></a> Chen X,  Velliste M,  Murphy R:
	<strong>Automated Interpretation of Subcellular Patterns in Fluorescence Microscope Images for Location Proteomics. </strong></p><p><em>Cytometry</em> 2006,  <strong>69 </strong>(7)<strong>:</strong>631-640. <a target="_top" href="http://www.biomedcentral.com/pubmed/16752421">PubMed Abstract</a> | <a target="_top" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=16752421">Publisher Full Text</a> |  <a href="http://www.biomedcentral.com/sfx_links.asp?ui=1471-2105-8-210&amp;bibl=B7"><img src="sfx_links.asp" alt="OpenURL"></a></p><p><a href="javascript:turn();">Return to text</a></p>

</li>
<li>
<p><a name="B8"></a> Glory E,  Murphy R:
	<strong>Automated Subcellular Location Determination and High Throughput Microscopy. </strong></p><p><em>Developmental Cell</em> 2007,  <strong>12</strong><strong>:</strong>7-16. <a target="_top" href="http://www.biomedcentral.com/pubmed/17199037">PubMed Abstract</a> | <a target="_top" href="http://www.ncbi.nlm.nih.gov/entrez/eutils/elink.fcgi?dbfrom=pubmed&amp;cmd=prlinks&amp;retmode=ref&amp;id=17199037">Publisher Full Text</a> |  <a href="http://www.biomedcentral.com/sfx_links.asp?ui=1471-2105-8-210&amp;bibl=B8"><img src="sfx_links.asp" alt="OpenURL"></a></p><p><a href="javascript:turn();">Return to text</a></p>

</li>
<li>
<p><a name="B9"></a> Mallat S:
	<em>A Wavelet Tour of Signal Processing</em>. Academic Press;  1999. <a href="http://www.biomedcentral.com/sfx_links.asp?ui=1471-2105-8-210&amp;bibl=B9"><img src="sfx_links.asp" alt="OpenURL"></a></p><p><a href="javascript:turn();">Return to text</a></p>

</li>
<li>
<p><a name="B10"></a> Vetterli M,  Kovačević J:
	<em>Wavelets and Subband Coding</em>. Signal Processing, Englewood Cliffs, NJ: Prentice Hall;  1995. <a href="http://www.biomedcentral.com/sfx_links.asp?ui=1471-2105-8-210&amp;bibl=B10"><img src="sfx_links.asp" alt="OpenURL"></a></p><p><a href="javascript:turn();">Return to text</a></p>

</li>
<li>
<p><a name="B11"></a> Mallat S:
	<strong>Wavelets for a vision. </strong></p><p><em>Proc IEEE</em> 1996,  <strong>33</strong><strong>:</strong>604-614.  <a href="http://www.biomedcentral.com/sfx_links.asp?ui=1471-2105-8-210&amp;bibl=B11"><img src="sfx_links.asp" alt="OpenURL"></a></p><p><a href="javascript:turn();">Return to text</a></p>

</li>
<li>
<p><a name="B12"></a> Srinivasa G,  Merryman T,  Chebira A,  Mintos A,  Kovačević J:
	<strong>Adaptive multiresolution techniques for subcellular protein location image classification. </strong>In <em>Proc IEEE Int Conf Acoust., Speech and Signal Proc</em>. <em> Volume V</em>. Toulouse, France;  2006:1177-1180.  <a href="http://www.biomedcentral.com/sfx_links.asp?ui=1471-2105-8-210&amp;bibl=B12"><img src="sfx_links.asp" alt="OpenURL"></a></p><p><a href="javascript:turn();">Return to text</a></p>

</li>
<li>
<p><a name="B13"></a> Merryman T,  Williams K,  Srinivasa G,  Chebira A,  Kovačević J:
	<strong>A multiresolution enhancement to generic classifiers of subcellular protein location images. </strong>In <em>Proc. IEEE Int. Symp. Biomed. Imaging</em>. Arlington VA;  2006:570-573.  <a href="http://www.biomedcentral.com/sfx_links.asp?ui=1471-2105-8-210&amp;bibl=B13"><img src="sfx_links.asp" alt="OpenURL"></a></p><p><a href="javascript:turn();">Return to text</a></p>

</li>
<li>
<p><a name="B14"></a> Saito N,  Coifman R:
	<strong>Local discriminant bases and their applications. </strong></p><p><em>Math Imaging Vision</em> 1995,  <strong>5</strong><strong>:</strong>337-358.  <a href="http://www.biomedcentral.com/sfx_links.asp?ui=1471-2105-8-210&amp;bibl=B14"><img src="sfx_links.asp" alt="OpenURL"></a></p><p><a href="javascript:turn();">Return to text</a></p>

</li>
<li>
<p><a name="B15"></a> Huang K,  Murphy R:
	<strong>Boosting accuracy of automated classification of fluorescence microscope images for location proteomics. </strong></p><p><em>BMC Bioinformatics</em> 2004,  <strong>18</strong>(5)<strong>:</strong>78.  <a href="http://www.biomedcentral.com/sfx_links.asp?ui=1471-2105-8-210&amp;bibl=B15"><img src="sfx_links.asp" alt="OpenURL"></a></p><p><a href="javascript:turn();">Return to text</a></p>

</li>
<li>
<p><a name="B16"></a> Haralick R:
	<strong>Statistical and structural approaches to texture. </strong></p><p><em>Proc IEEE</em> 1979,  <strong>67</strong><strong>:</strong>786-804.  <a href="http://www.biomedcentral.com/sfx_links.asp?ui=1471-2105-8-210&amp;bibl=B16"><img src="sfx_links.asp" alt="OpenURL"></a></p><p><a href="javascript:turn();">Return to text</a></p>

</li>
<li>
<p><a name="B17"></a> Chebira , <em>et al</em>.:
	<strong>A multiresolution approach to automated classification of protein subcellular location images. </strong> [<a target="_top" href="http://www.andrew.cmu.edu/user/jelenak/Repository/07_ChebiraBJMSMK/07_ChebiraBJMSMK.html">http://www.andrew.cmu.edu/user/jelenak/Repository/07_ChebiraBJMSMK/07_ChebiraBJMSMK.html</a>]
	 <a href="http://www.webcitation.org/query.php?url=http://www.andrew.cmu.edu/user/jelenak/Repository/07_ChebiraBJMSMK/07_ChebiraBJMSMK.html&amp;refdoi=10.1186/1471-2105-8-210" alt="" title="Archive copy of webpage" class="xpushbutton">webcite</a></p><p> <a href="http://www.biomedcentral.com/sfx_links.asp?ui=1471-2105-8-210&amp;bibl=B17"><img src="sfx_links.asp" alt="OpenURL"></a></p><p><a href="javascript:turn();">Return to text</a></p>

</li>
<li>
<p><a name="B18"></a><strong>The Murphy Lab at Carnegie Mellon University </strong> [<a target="_top" href="http://murphylab.web.cmu.edu/">http://murphylab.web.cmu.edu</a>]
	 <a href="http://www.webcitation.org/query.php?url=http://murphylab.web.cmu.edu&amp;refdoi=10.1186/1471-2105-8-210" alt="" title="Archive copy of webpage" class="xpushbutton">webcite</a></p><p> <a href="http://www.biomedcentral.com/sfx_links.asp?ui=1471-2105-8-210&amp;bibl=B18"><img src="sfx_links.asp" alt="OpenURL"></a></p><p><a href="javascript:turn();">Return to text</a></p>

</li>
<li>
<p><a name="B19"></a> Coifman R,  Meyer Y,  Quake S,  Wickerhauser M:
	<strong>Signal Processing and Compression with Wavelet Packets. </strong>In <em>Tech rep</em>. Yale University;  1991. <a href="http://www.biomedcentral.com/sfx_links.asp?ui=1471-2105-8-210&amp;bibl=B19"><img src="sfx_links.asp" alt="OpenURL"></a></p><p><a href="javascript:turn();">Return to text</a></p>

</li>
<li>
<p><a name="B20"></a> Saito N,  Coifman R:
	<strong>Local discriminant bases. </strong></p><p><em>Proc SPIE Conf. Vis. Commun. and Image Proc</em> 1994, 2-14.  <a href="http://www.biomedcentral.com/sfx_links.asp?ui=1471-2105-8-210&amp;bibl=B20"><img src="sfx_links.asp" alt="OpenURL"></a></p><p><a href="javascript:turn();">Return to text</a></p>

</li>
<li>
<p><a name="B21"></a> Yeomans PH,  Thornton J,  Kovačević J,  Kumar B:
	<strong>Wavelet packet correlation methods in biometrics. </strong></p><p><em>Appl Opt, sp iss Biometric Recognition Systems</em> 2005,  <strong>44</strong>(5)<strong>:</strong>637-646.  <a href="http://www.biomedcentral.com/sfx_links.asp?ui=1471-2105-8-210&amp;bibl=B21"><img src="sfx_links.asp" alt="OpenURL"></a></p><p><a href="javascript:turn();">Return to text</a></p>

</li>
<li>
<p><a name="B22"></a> Chen X,  Velliste M,  Weinstein S,  Jarvik J,  Murphy R:
	<strong>Location proteomics – Building subcellular location trees from high resolution 3D fluorescence microscope images of randomly-tagged proteins. </strong>In <em>Proc SPIE</em>. <em> Volume 4962</em>. San Jose CA;  2003:298-306.  <a href="http://www.biomedcentral.com/sfx_links.asp?ui=1471-2105-8-210&amp;bibl=B22"><img src="sfx_links.asp" alt="OpenURL"></a></p><p><a href="javascript:turn();">Return to text</a></p>

</li>
<li>
<p><a name="B23"></a> Kovačević J,  Chebira A:
	<strong>Life Beyond Bases: The Advent of Frames (Parts I and II). </strong></p><p><em>IEEE Signal Processing Magazine</em> 2007. <a href="http://www.biomedcentral.com/sfx_links.asp?ui=1471-2105-8-210&amp;bibl=B23"><img src="sfx_links.asp" alt="OpenURL"></a></p><p><a href="javascript:turn();">Return to text</a></p>

</li>
</ol>
<h4 class="bottommessage"><a href="http://www.biomedcentral.com/1471-2105/8/210/postcomment">Have something to say? Post a comment on this article!</a></h4></td></tr></tbody></table></div><hr color="#cccccc" noshade="noshade" size="1">
<table cellpadding="3" width="100%"><tbody><tr valign="middle"><td align="center">


<center>

<!------ OAS AD 'Bottom' begin ------>
<a href="http://oas.biomedcentral.com/5c/www.biomedcentral.com/article/bmcbioinformatics/1924351337/Bottom/biomed/house_contra_cope_nov07may08/cope_grant_banner.bmp/63613036663366633436653634656530" target="_top"><img src="cope_grant_banner.bmp" alt="" border="0" height="60" width="468"></a>
<!------ OAS AD end ------>
</center>

</td></tr>
<tr><td align="center"><div class="smalltext">
<a href="http://www.biomedcentral.com/info/about/tandc">Terms and Conditions</a> 
<a href="http://www.biomedcentral.com/info/about/privacy">Privacy statement</a> 
<a href="http://www.biomedcentral.com/info/mediainformation/">Information for advertisers</a> 
<a href="http://www.biomedcentral.com/info/about/bmcjobs">Jobs at BMC</a> 
<a href="mailto:info@biomedcentral.com">Contact us</a> 
</div></td></tr>
</tbody></table>
<table border="0" cellpadding="0" cellspacing="0" width="100%">
<tbody><tr>
<td colspan="3" align="center" bgcolor="#003399">
<font class="smallSans" color="#ffffff" size="1">
© 1999-2008 BioMed Central Ltd
 unless otherwise stated
</font></td>
</tr>
</tbody></table>
</body>
</html>
