<html lang="en" xml:lang="en" xmlns="http://www.w3.org/1999/xhtml">
<head xmlns="">
<meta http-equiv="Content-Type" content="text/html; charset=UTF-8">

	

	

	

                































 







       
              
      



























        
                
 

 







     

 

 

        
        
  

 


 

 
 
 


 
 





	
<title>Image-based multivariate profiling of drug responses from single cells - Nature Methods</title><meta content="noarchive" name="robots"><link href="favicon.ico" rel="shortcut icon">
		










<link rel="stylesheet" type="text/css" href="index.css" media="all">
</head>
<body topmargin="0" leftmargin="0" bgcolor="#ffffff" marginheight="0" marginwidth="0" text="#000000"><a name="top">
	



<!-- header -->
</a><div id="head-er">

<div class="header-nonad-wrapper" id="top">
<a name="top">				</a><div class="accessibility-login-wrap">
<a name="top">	</a><div class="accessibility-login">
<a name="top">				<!--<a href="/" class="rollover"><img src="/common/includes/header/images/nature.com.gif" alt="Nature.com homepage" class="nature-logo" /></a>-->
		</a><a href="http://www.nature.com.ejproxy.a-star.edu.sg/" class="nature-link">nature.com<span class="hidden"> homepage</span></a>
									
<div class="logon">
	<p>Full text access provided to <span class="user">A*STAR</span><br> by <span class="inst">c/o Technical Services, Central Library</span>
</p></div>
				<div class="cleardiv"><!--  --></div>
	</div>
</div>

<div class="cleardiv"><!--  --></div>
<hr>
		
			<div class="image-user-services-wrap">
	<div class="image-user-services">
					<a href="http://www.nature.com.ejproxy.a-star.edu.sg/nmeth"><img src="nmeth_logo.gif" class="journal-header" alt="Nature Methods homepage"></a>		
								<div class="user-services div1">
								<!-- whitespace removed to deal with IE 5.0 bug --><ul class="list1"><li class="li1"><a href="http://www.nature.com.ejproxy.a-star.edu.sg/myaccount/">My account</a></li><li class="li2"><a href="http://www.nature.com.ejproxy.a-star.edu.sg/nams/svc/myaccount/save/ealert?list_id=95" title="sign up to receive Nature Methods email alerts">E-alert sign up</a></li></ul>
							</div>
						<div class="user-services div2">
								<!-- whitespace removed to deal with IE 5.0 bug --><ul class="list2"><li class="li1"><a href="http://www.nature.com.ejproxy.a-star.edu.sg/register/" title="Register with Nature.com">Register</a></li><li class="li2"><a href="https://secure.nature.com.ejproxy.a-star.edu.sg/subscribe/nmeth" title="Subscribe to Nature Methods">Subscribe</a></li></ul>
							</div>
						<div class="cleardiv"><!--  --></div>
			</div>
	<div class="cleardiv"><!--  --></div>
</div>

<div class="cleardiv"><!--  --></div>
<hr>		
			<div class="publications-search-wrap clearfix">
	<div class="publications-search">
				<ul class="publications">
			<li class="first"><a href="http://www.nature.com.ejproxy.a-star.edu.sg/siteindex/">Publications&nbsp;A-Z&nbsp;index</a></li><li><a href="http://www.nature.com.ejproxy.a-star.edu.sg/browse/">Browse&nbsp;by&nbsp;subject</a></li>		</ul>

		<hr>
		
				<div class="search">
			<form class="search" action="http://www.nature.com.ejproxy.a-star.edu.sg/search/executeSearch?" method="get">
				<div>
					<label for="searchtext">Search</label>
					<input value="" name="sp-q" id="searchtext" type="text">
					<input name="sp-c" value="10" type="hidden">
										<input name="sp-x-9" value="cat" type="hidden">
										<input name="sp-s" value="date" type="hidden">
										<select name="sp-q-9">
												<option value="NMETH">This journal</option>
												<option value="">All of nature.com</option>
											</select>
										<input name="submit" class="button" value="go" title="search now" type="submit">
										<a href="http://www.nature.com.ejproxy.a-star.edu.sg/search/adv_search?sp-a=sp1001702d&amp;sp-x-1=ujournal&amp;sp-q-1=Nature%20Methods">Advanced&nbsp;search</a>
					
										<input name="sp-a" value="sp1001702d" type="hidden">
										<input name="sp-sfvl-field" value="subject|ujournal" type="hidden">
										<input name="sp-x-1" value="ujournal" type="hidden">
										<input name="sp-p-1" value="phrase" type="hidden">
										<input name="sp-p" value="all" type="hidden">
									</div>
			</form>
		</div>
				<div class="cleardiv"><!--  --></div>
	</div>
</div>		
		</div>

	<!-- leaderboard ad -->
<div class="header-leaderboard-wrap">








										


<div class="header-leaderboard">



</div>


</div>
<!-- end leaderboard ad -->

</div>
<!-- end header -->

			<div id="constrain"><div class="constrain">
		<table border="0" cellpadding="0" cellspacing="0" width="100%"><tbody><tr xmlns=""><td align="center" height="22" valign="top"><table border="0" cellpadding="0" cellspacing="0" width="760"><tbody><tr><td bgcolor="#d1e1ea" height="22" width="145">&nbsp;</td><td class="breadcrumb" align="left" bgcolor="#d1e1ea" height="22" width="630"><a class="breadcrumblink" href="http://www.nature.com.ejproxy.a-star.edu.sg/nmeth/index.html">Journal home</a>&nbsp;&gt;
<a class="breadcrumblink" href="http://www.nature.com.ejproxy.a-star.edu.sg/nmeth/archive/index.html">Archive</a>&nbsp;&gt;
<a class="breadcrumblink" href="http://www.nature.com.ejproxy.a-star.edu.sg/nmeth/journal/v4/n5/index.html">
					
 Table of Contents</a>&nbsp;&gt;&nbsp;<a class="breadcrumblink" href="http://www.nature.com.ejproxy.a-star.edu.sg/nmeth/journal/v4/n5/index.html#af">Article</a> &gt;Full text</td></tr></tbody></table></td></tr><tr><td align="center"><table border="0" cellpadding="0" cellspacing="0" width="760"><tbody><tr><td valign="top" width="130"><input xmlns="" value="archive" id="leftnav_type" type="hidden"><input xmlns="" value="af" id="categ" type="hidden"><input xmlns="" value="" id="subcateg" type="hidden"><input xmlns="" value="orig" id="arch_categ" type="hidden">

		<table border="0" cellpadding="0" cellspacing="0" width="130"><tbody><tr><td colspan="3" height="1"><img alt="" src="spacer.gif" height="1" width="5"></td></tr><tr><td id="home_1" style="background-color: rgb(209, 225, 234);" width="5"><img alt="" src="spacer_001.gif" height="20" width="5"></td><td id="home_2" style="background-color: rgb(209, 225, 234);"><a class="leftnav" href="http://www.nature.com.ejproxy.a-star.edu.sg/nmeth/index.html" id="home_link">Journal home</a></td><td id="home_3" style="background-color: rgb(209, 225, 234);" width="5"><img alt="" src="spacer_001.gif" height="20" width="5"></td></tr><tr><td colspan="3" height="1"></td></tr><tr><td id="aop_1" style="background-color: rgb(209, 225, 234);" width="5"><img alt="" src="spacer_001.gif" height="20" width="5"></td><td id="aop_2" style="background-color: rgb(209, 225, 234);"><a class="leftnav" href="http://www.nature.com.ejproxy.a-star.edu.sg/nmeth/journal/vaop/ncurrent/index.html" id="aop_link">Advance online publication</a></td><td id="aop_3" style="background-color: rgb(209, 225, 234);" width="5"><img alt="" src="spacer_001.gif" height="20" width="5"></td></tr><tr><td colspan="3" height="1"></td></tr><tr><td id="issue_1" style="background-color: rgb(209, 225, 234);" width="5"><img alt="" src="spacer_001.gif" height="20" width="5"></td><td id="issue_2" style="background-color: rgb(209, 225, 234);"><a class="leftnav" href="http://www.nature.com.ejproxy.a-star.edu.sg/nmeth/journal/v5/n6/index.html" id="issue_link">
Current issue
</a></td><td id="issue_3" style="background-color: rgb(209, 225, 234);" width="5"><img alt="" src="spacer_001.gif" height="20" width="5"></td></tr><tr><td colspan="3" height="1"></td></tr><tr><td id="archive_1" style="background-color: rgb(110, 131, 160);" width="5"><img alt="" src="spacer_001.gif" height="20" width="5"></td><td id="archive_2" style="background-color: rgb(110, 131, 160);"><a class="leftnavOn" href="http://www.nature.com.ejproxy.a-star.edu.sg/nmeth/archive/index.html" id="archive_link">Archive</a></td><td id="archive_3" style="background-color: rgb(110, 131, 160);" width="5"><img alt="" src="spacer_001.gif" height="20" width="5"></td></tr><tr><td colspan="3" height="1"></td></tr><tr><td id="Press_releases_1" style="background-color: rgb(209, 225, 234);" width="5"><img alt="" src="spacer_001.gif" height="20" width="5"></td><td id="Press_releases_2" style="background-color: rgb(209, 225, 234);"><a class="leftnav" href="http://www.nature.com.ejproxy.a-star.edu.sg/nmeth/press_release/index.html" id="Press_releases_link">Press releases</a></td><td id="Press_releases_3" style="background-color: rgb(209, 225, 234);" width="5"><img alt="" src="spacer_001.gif" height="20" width="5"></td></tr><tr><td colspan="3" height="1"></td></tr></tbody></table>
		<table border="0" cellpadding="0" cellspacing="0" width="130"><tbody><tr><td colspan="3" height="1"><img alt="" src="spacer_001.gif" height="1" width="5"></td></tr><tr><td colspan="3" height="12"><img alt="" src="spacer_001.gif" height="12" width="5"></td></tr><tr><td id="Supplements_1" style="background-color: rgb(209, 225, 234);" width="5"><img alt="" src="spacer_001.gif" height="20" width="5"></td><td id="Supplements_2" style="background-color: rgb(209, 225, 234);"><a class="leftnav2" href="http://blogs.nature.com.ejproxy.a-star.edu.sg/nmeth/methagora/" id="Supplements_link">Methagora</a></td><td id="Supplements_3" style="background-color: rgb(209, 225, 234);" width="5"><img alt="" src="spacer_001.gif" height="20" width="5"></td></tr><tr><td colspan="3" height="1"></td></tr><tr><td id="Web_Focuses_1" style="background-color: rgb(209, 225, 234);" width="5"><img alt="" src="spacer_001.gif" height="20" width="5"></td><td id="Web_Focuses_2" style="background-color: rgb(209, 225, 234);"><a class="leftnav2" href="http://www.nature.com.ejproxy.a-star.edu.sg/nmeth/webfocus/index.html" id="Web_Focuses_link">Focuses</a></td><td id="Web_Focuses_3" style="background-color: rgb(209, 225, 234);" width="5"><img alt="" src="spacer_001.gif" height="20" width="5"></td></tr><tr><td colspan="3" height="1"></td></tr></tbody></table>
	
					 
			<!-- Molecular Cloning link for left navigation table -->
<table border="0" cellpadding="0" cellspacing="0" width="130">
<tbody><tr> 
<td colspan="3" height="1"><img src="spacer_002.gif" alt="" height="12" width="5"></td>
</tr>
<tr> 
<td class="bg-leftnav" width="5"><img src="spacer_002.gif" alt="" height="20" width="5"></td>
<td class="bg-leftnav"><a href="http://ad.doubleclick.net/clk;16135042;8858766;r?http://www.molecularcloning.com/members" class="leftnav">Molecular Cloning</a></td>
<td class="bg-leftnav" width="5"><img src="spacer_002.gif" alt="" height="20" width="5"></td>
</tr>
</tbody></table>
<!-- end Molecular Cloning link -->			
			
		<table border="0" cellpadding="0" cellspacing="0" width="130"><tbody><tr><td colspan="3" height="12"><img alt="" src="spacer_001.gif" height="12" width="5"></td></tr><tr><td id="For_authors_1" style="background-color: rgb(209, 225, 234);" width="5"><img alt="" src="spacer_001.gif" height="20" width="5"></td><td id="For_authors_2" style="background-color: rgb(209, 225, 234);"><a class="leftnav2" href="http://www.nature.com.ejproxy.a-star.edu.sg/nmeth/authors/index.html" id="For_authors_link">Guide to authors</a></td><td id="For_authors_3" style="background-color: rgb(209, 225, 234);" width="5"><img alt="" src="spacer_001.gif" height="20" width="5"></td></tr><tr><td colspan="3" height="1"></td></tr><tr><td id="Online submission_1" style="background-color: rgb(209, 225, 234);" colspan="3"><table border="0" cellpadding="0" cellspacing="0" width="130"><tbody><tr><td height="20"><img id="Online_submission_navimage" alt="Online submission" src="nmeth_icon_onlinesub.gif" align="absmiddle" border="0"></td><td><a class="leftnav2" href="http://mts-nmeth.nature.com.ejproxy.a-star.edu.sg/" target="_top" id="Online_submission_link">Online submission</a></td></tr></tbody></table></td></tr><tr><td colspan="3" height="1"></td></tr><tr><td id="Permissions_1" style="background-color: rgb(209, 225, 234);" width="5"><img alt="" src="spacer_001.gif" height="20" width="5"></td><td id="Permissions_2" style="background-color: rgb(209, 225, 234);"><a class="leftnav2" href="http://npg.nature.com.ejproxy.a-star.edu.sg/npg/servlet/Content?data=xml/01_reprints.xml&amp;style=xml/01_reprints.xsl" target="_top" id="Permissions_link">Permissions</a></td><td id="Permissions_3" style="background-color: rgb(209, 225, 234);" width="5"><img alt="" src="spacer_001.gif" height="20" width="5"></td></tr><tr><td colspan="3" height="1"></td></tr><tr><td colspan="3" height="12"><img alt="" src="spacer.gif" height="12" width="5"></td></tr><tr><td id="For_referees_1" style="background-color: rgb(209, 225, 234);" width="5"><img alt="" src="spacer_001.gif" height="20" width="5"></td><td id="For_referees_2" style="background-color: rgb(209, 225, 234);"><a class="leftnav2" href="http://www.nature.com.ejproxy.a-star.edu.sg/authors/editorial_policies/peer_review.html" id="For_referees_link">For referees</a></td><td id="For_referees_3" style="background-color: rgb(209, 225, 234);" width="5"><img alt="" src="spacer_001.gif" height="20" width="5"></td></tr><tr><td colspan="3" height="1"></td></tr><tr><td colspan="3" height="12"><img alt="" src="spacer.gif" height="12" width="5"></td></tr><tr><td id="sample_1" style="background-color: rgb(209, 225, 234);" width="5"><img alt="" src="spacer_001.gif" height="20" width="5"></td><td id="sample_2" style="background-color: rgb(209, 225, 234);"><a class="leftnav2" href="http://www.nature.com.ejproxy.a-star.edu.sg/nmeth/journal/v2/n1/index.html" id="sample_link">Free online issue</a></td><td id="sample_3" style="background-color: rgb(209, 225, 234);" width="5"><img alt="" src="spacer_001.gif" height="20" width="5"></td></tr><tr><td colspan="3" height="1"></td></tr><tr><td colspan="3" height="12"><img alt="" src="spacer.gif" height="12" width="5"></td></tr>

<!--

<tr><td id="About_the_journal_1" width="5" style="background-color:
#D1E1EA"><img alt="" height="20" width="5" src="/includes/rj_globnavimages/spacer.gif"></td><td id="About_the_journal_2" style="background-color:
#D1E1EA"><a class="leftnav2" href="/nmeth/about/index.html" id="About_the_journal_link">About the journal</a></td><td id="About_the_journal_3" width="5" style="background-color:
#D1E1EA"><img alt="" height="20" width="5" src="/includes/rj_globnavimages/spacer.gif"></td></tr><tr><td height="1" colspan="3"><script LANGUAGE="JavaScript">

var thisType = 'About_the_journal';

addSubLevel(thisType, new Array("About_the_editors","Editorial_policies","Masthead"));

var newRows = '';
if (isCurrentType(thisType) || '' == 'true'){

	
	newRows += getNewSubRow('About_the_editors', '/nmeth/about/about_eds/index.html', 'About the editors');
	
	newRows += getNewSubRow('Editorial_policies', '/nmeth/about/ed_policies/index.html', 'Editorial policies');
	
	newRows += getNewSubRow('Masthead', '/nmeth/about/masthead/index.html', 'Masthead');
	

	if (isCurrentType(thisType) && thisType != getPageType()){
		hilite(thisType);
	}
	
	
	

}

document.write (getSubRow(newRows));
</script>-->


<tr><td id="Contact_the_journal_1" style="background-color: rgb(209, 225, 234);" width="5"><img alt="" src="spacer_001.gif" height="20" width="5"></td><td id="Contact_the_journal_2" style="background-color: rgb(209, 225, 234);"><a class="leftnav2" href="http://www.nature.com.ejproxy.a-star.edu.sg/nmeth/contact/index.html" id="Contact_the_journal_link">Contact the journal</a></td><td id="Contact_the_journal_3" style="background-color: rgb(209, 225, 234);" width="5"><img alt="" src="spacer_001.gif" height="20" width="5"></td></tr><tr><td colspan="3" height="1"></td></tr><tr><td id="Subscribe_1" style="background-color: rgb(209, 225, 234);" width="5"><img alt="" src="spacer_001.gif" height="20" width="5"></td><td id="Subscribe_2" style="background-color: rgb(209, 225, 234);"><a class="leftnav2" href="https://www.sunbeltfs.com/forms/nh/subscribe.asp?eid=D50401" target="_top" id="Subscribe_link">Subscribe</a></td><td id="Subscribe_3" style="background-color: rgb(209, 225, 234);" width="5"><img alt="" src="spacer_001.gif" height="20" width="5"></td></tr><tr><td colspan="3" height="1"><img src="spacer_001.gif" alt="" height="1" width="5"></td></tr><tr><td colspan="3" height="1"><table border="0" cellpadding="0" cellspacing="0" width="100%"><tbody><tr><td style="background-color: rgb(234, 234, 234);" id="new-sub_1" width="5"><img src="spacer_001.gif" alt="" height="15" width="5"></td><td style="background-color: rgb(234, 234, 234);" id="new-sub_2" valign="top"><img src="nav_dots2.gif" id="new-sub_image" alt="" height="7" vspace="2" width="5"></td><td style="background-color: rgb(234, 234, 234);" id="new-sub_3"><img src="spacer_001.gif" alt="" height="15" width="5"></td><td style="background-color: rgb(234, 234, 234);" id="new-sub_4"><a href="http://www.sunbeltfs.com/forms/select.asp?pid=NH&amp;arq=subscribe&amp;addurl=eid=D50401" class="leftnav3" id="new-sub_link">New Subscription</a></td><td style="background-color: rgb(234, 234, 234);" id="new-sub_5" valign="top" width="5"><img src="spacer_001.gif" alt="" height="15" width="5"></td></tr><tr><td style="background-color: rgb(234, 234, 234);" id="renew-sub_1" width="5"><img src="spacer_001.gif" alt="" height="15" width="5"></td><td style="background-color: rgb(234, 234, 234);" id="renew-sub_2" valign="top"><img src="nav_dots2.gif" id="renew-sub_image" alt="" height="7" vspace="2" width="5"></td><td style="background-color: rgb(234, 234, 234);" id="renew-sub_3"><img src="spacer_001.gif" alt="" height="15" width="5"></td><td style="background-color: rgb(234, 234, 234);" id="renew-sub_4"><a href="http://www.sunbeltfs.com/forms/select.asp?pid=NH&amp;arq=renew&amp;addurl=eid=D50402" class="leftnav3" id="renew-sub_link">Renew Subscription</a></td><td style="background-color: rgb(234, 234, 234);" id="renew-sub_5" valign="top" width="5"><img src="spacer_001.gif" alt="" height="15" width="5"></td></tr><tr><td style="background-color: rgb(234, 234, 234);" id="paid-sub_1" width="5"><img src="spacer_001.gif" alt="" height="15" width="5"></td><td style="background-color: rgb(234, 234, 234);" id="paid-sub_2" valign="top"><img src="nav_dots2.gif" id="paid-sub_image" alt="" height="7" vspace="2" width="5"></td><td style="background-color: rgb(234, 234, 234);" id="paid-sub_3"><img src="spacer_001.gif" alt="" height="15" width="5"></td><td style="background-color: rgb(234, 234, 234);" id="paid-sub_4"><a href="https://secure.nature.com.ejproxy.a-star.edu.sg/subscribe/nmeth" class="leftnav3" id="paid-sub_link">Paid Subscriptions</a></td><td style="background-color: rgb(234, 234, 234);" id="paid-sub_5" valign="top" width="5"><img src="spacer_001.gif" alt="" height="15" width="5"></td></tr><tr><td style="background-color: rgb(234, 234, 234);" id="change-address_1" width="5"><img src="spacer_001.gif" alt="" height="15" width="5"></td><td style="background-color: rgb(234, 234, 234);" id="change-address_2" valign="top"><img src="nav_dots2.gif" id="change-address_image" alt="" height="7" vspace="2" width="5"></td><td style="background-color: rgb(234, 234, 234);" id="change-address_3"><img src="spacer_001.gif" alt="" height="15" width="5"></td><td style="background-color: rgb(234, 234, 234);" id="change-address_4"><a href="http://www.sunbeltfs.com/forms/select.asp?pid=NH&amp;arq=renew&amp;addurl=eid=D50403" class="leftnav3" id="change-address_link">Change of Address</a></td><td style="background-color: rgb(234, 234, 234);" id="change-address_5" valign="top" width="5"><img src="spacer_001.gif" alt="" height="15" width="5"></td></tr></tbody></table></td></tr><tr><td colspan="3" height="1"><img src="spacer_001.gif" alt="" height="1" width="5"></td></tr><tr><td id="jobs_1" style="background-color: rgb(209, 225, 234);" width="5"><img alt="" src="spacer_001.gif" height="20" width="5"></td><td id="jobs_2" style="background-color: rgb(209, 225, 234);"><a class="leftnav2" href="http://www.nature.com.ejproxy.a-star.edu.sg/naturejobs/science/jobs/" target="_top" id="jobs_link">naturejobs</a></td><td id="jobs_3" style="background-color: rgb(209, 225, 234);" width="5"><img alt="" src="spacer_001.gif" height="20" width="5"></td></tr><tr><td colspan="3" height="1"></td></tr><tr><td id="Advertising_1" style="background-color: rgb(209, 225, 234);" width="5"><img alt="" src="spacer_001.gif" height="20" width="5"></td><td id="Advertising_2" style="background-color: rgb(209, 225, 234);"><a class="leftnav2" href="http://npg.nature.com.ejproxy.a-star.edu.sg/media/nature/display/nature_methods/editorial.html" target="_top" id="Advertising_link">For Advertisers</a></td><td id="Advertising_3" style="background-color: rgb(209, 225, 234);" width="5"><img alt="" src="spacer_001.gif" height="20" width="5"></td></tr><tr><td colspan="3" height="1"></td></tr><tr><td id="Work_1" style="background-color: rgb(209, 225, 234);" width="5"><img alt="" src="spacer_001.gif" height="20" width="5"></td><td id="Work_2" style="background-color: rgb(209, 225, 234);"><a class="leftnav2" href="http://npg.nature.com.ejproxy.a-star.edu.sg/npg/forms/03_current.jsp" id="Work_link">work@npg</a></td><td id="Work_3" style="background-color: rgb(209, 225, 234);" width="5"><img alt="" src="spacer_001.gif" height="20" width="5"></td></tr><tr><td colspan="3" height="1"></td></tr><tr><td id="Reprints_1" style="background-color: rgb(209, 225, 234);" width="5"><img alt="" src="spacer_001.gif" height="20" width="5"></td><td id="Reprints_2" style="background-color: rgb(209, 225, 234);"><a class="leftnav2" href="http://www.naturereprints.com/" target="_top" id="Reprints_link">naturereprints</a></td><td id="Reprints_3" style="background-color: rgb(209, 225, 234);" width="5"><img alt="" src="spacer_001.gif" height="20" width="5"></td></tr><tr><td colspan="3" height="1"></td></tr><tr><td id="About_this_site_1" style="background-color: rgb(209, 225, 234);" width="5"><img alt="" src="spacer_001.gif" height="20" width="5"></td><td id="About_this_site_2" style="background-color: rgb(209, 225, 234);"><a class="leftnav2" href="http://www.nature.com.ejproxy.a-star.edu.sg/nmeth/site/index.html" id="About_this_site_link">About this site</a></td><td id="About_this_site_3" style="background-color: rgb(209, 225, 234);" width="5"><img alt="" src="spacer_001.gif" height="20" width="5"></td></tr><tr><td colspan="3" height="1"></td></tr><tr><td colspan="3" height="12"><img alt="" src="spacer.gif" height="12" width="5"></td></tr><tr><td id="For_librarians_1" style="background-color: rgb(209, 225, 234);" width="5"><img alt="" src="spacer_001.gif" height="20" width="5"></td><td id="For_librarians_2" style="background-color: rgb(209, 225, 234);"><a class="leftnav2" href="http://npg.nature.com.ejproxy.a-star.edu.sg/npg/servlet/Content?data=xml/07_home.xml&amp;style=xml/07_home.xsl" target="_top" id="For_librarians_link">For librarians</a></td><td id="For_librarians_3" style="background-color: rgb(209, 225, 234);" width="5"><img alt="" src="spacer_001.gif" height="20" width="5"></td></tr><tr><td colspan="3" height="1"></td></tr><tr><td colspan="3" height="12"><img alt="" src="spacer.gif" height="12" width="5"></td></tr><tr><td id="Application_notes_1" style="background-color: rgb(209, 225, 234);" width="5"><img alt="" src="spacer_001.gif" height="20" width="5"></td><td id="Application_notes_2" style="background-color: rgb(209, 225, 234);"><a class="leftnav2" href="http://www.nature.com.ejproxy.a-star.edu.sg/nmeth/appnotes/submit/index.html" target="_top" id="Application_notes_link">Application notes</a></td><td id="Application_notes_3" style="background-color: rgb(209, 225, 234);" width="5"><img alt="" src="spacer_001.gif" height="20" width="5"></td></tr><tr><td colspan="3" height="1"></td></tr></tbody></table>	
		
<table border="0" cellpadding="0" cellspacing="0" width="130"><tbody><tr><td colspan="3" height="20">&nbsp;</td></tr><tr><td bgcolor="#cc0033" width="5"><img alt="" src="spacer.gif" height="20" width="5"></td><td class="leftnavOn" bgcolor="#cc0033">NPG Resources</td><td bgcolor="#cc0033" width="5"><img alt="" src="spacer.gif" height="20" width="5"></td></tr><tr><td colspan="3" height="1"><img alt="" src="spacer.gif" height="1" width="5"></td></tr><tr><td bgcolor="#eaeaea" width="5"><img alt="" src="spacer.gif" height="20" width="5"></td><td bgcolor="#eaeaea"><a target="_top" class="leftnav3" href="http://www.nature.com.ejproxy.a-star.edu.sg/nature/">Nature</a></td><td bgcolor="#eaeaea" width="5"><img alt="" src="spacer.gif" height="20" width="5"></td></tr><tr><td colspan="3" height="1"><img alt="" src="spacer.gif" height="1" width="5"></td></tr><tr><td bgcolor="#eaeaea" width="5"><img alt="" src="spacer.gif" height="20" width="5"></td><td bgcolor="#eaeaea"><a target="_top" class="leftnav3" href="http://www.nature.com.ejproxy.a-star.edu.sg/nbt/">Nature Biotechnology</a></td><td bgcolor="#eaeaea" width="5"><img alt="" src="spacer.gif" height="20" width="5"></td></tr><tr><td colspan="3" height="1"><img alt="" src="spacer.gif" height="1" width="5"></td></tr><tr><td bgcolor="#eaeaea" width="5"><img alt="" src="spacer.gif" height="20" width="5"></td><td bgcolor="#eaeaea"><a target="_top" class="leftnav3" href="http://www.nature.com.ejproxy.a-star.edu.sg/nprot/">Nature Protocols</a></td><td bgcolor="#eaeaea" width="5"><img alt="" src="spacer.gif" height="20" width="5"></td></tr><tr><td colspan="3" height="1"><img alt="" src="spacer.gif" height="1" width="5"></td></tr><tr><td bgcolor="#eaeaea" width="5"><img alt="" src="spacer.gif" height="20" width="5"></td><td bgcolor="#eaeaea"><a target="_top" class="leftnav3" href="http://www.nature.com.ejproxy.a-star.edu.sg/ng/">Nature Genetics</a></td><td bgcolor="#eaeaea" width="5"><img alt="" src="spacer.gif" height="20" width="5"></td></tr><tr><td colspan="3" height="1"><img alt="" src="spacer.gif" height="1" width="5"></td></tr><tr><td bgcolor="#eaeaea" width="5"><img alt="" src="spacer.gif" height="20" width="5"></td><td bgcolor="#eaeaea"><a target="_top" class="leftnav3" href="http://www.nature.com.ejproxy.a-star.edu.sg/nchembio/">Nature Chemical Biology</a></td><td bgcolor="#eaeaea" width="5"><img alt="" src="spacer.gif" height="20" width="5"></td></tr><tr><td colspan="3" height="1"><img alt="" src="spacer.gif" height="1" width="5"></td></tr><tr><td bgcolor="#eaeaea" width="5"><img alt="" src="spacer.gif" height="20" width="5"></td><td bgcolor="#eaeaea"><a target="_top" class="leftnav3" href="http://www.nature.com.ejproxy.a-star.edu.sg/ncb/">Nature Cell Biology</a></td><td bgcolor="#eaeaea" width="5"><img alt="" src="spacer.gif" height="20" width="5"></td></tr><tr><td colspan="3" height="1"><img alt="" src="spacer.gif" height="1" width="5"></td></tr><tr><td bgcolor="#eaeaea" width="5"><img alt="" src="spacer.gif" height="20" width="5"></td><td bgcolor="#eaeaea"><a target="_top" class="leftnav3" href="http://www.nature.com.ejproxy.a-star.edu.sg/neuro/">Nature Neuroscience</a></td><td bgcolor="#eaeaea" width="5"><img alt="" src="spacer.gif" height="20" width="5"></td></tr><tr><td colspan="3" height="1"><img alt="" src="spacer.gif" height="1" width="5"></td></tr><tr><td bgcolor="#eaeaea" width="5"><img alt="" src="spacer.gif" height="20" width="5"></td><td bgcolor="#eaeaea"><a target="_top" class="leftnav3" href="http://www.nature.com.ejproxy.a-star.edu.sg/nrg/">Nature Reviews Genetics</a></td><td bgcolor="#eaeaea" width="5"><img alt="" src="spacer.gif" height="20" width="5"></td></tr><tr><td colspan="3" height="1"><img alt="" src="spacer.gif" height="1" width="5"></td></tr><tr><td bgcolor="#eaeaea" width="5"><img alt="" src="spacer.gif" height="20" width="5"></td><td bgcolor="#eaeaea"><a target="_top" class="leftnav3" href="http://www.nature.com.ejproxy.a-star.edu.sg/nrm/">Nature Reviews Molecular Cell Biology</a></td><td bgcolor="#eaeaea" width="5"><img alt="" src="spacer.gif" height="20" width="5"></td></tr><tr><td colspan="3" height="1"><img alt="" src="spacer.gif" height="1" width="5"></td></tr><tr><td bgcolor="#eaeaea" width="5"><img alt="" src="spacer.gif" height="20" width="5"></td><td bgcolor="#eaeaea"><a target="_top" class="leftnav3" href="http://www.nature.com.ejproxy.a-star.edu.sg/nrd/">Nature Reviews Drug Discovery</a></td><td bgcolor="#eaeaea" width="5"><img alt="" src="spacer.gif" height="20" width="5"></td></tr><tr><td colspan="3" height="1"><img alt="" src="spacer.gif" height="1" width="5"></td></tr><tr><td bgcolor="#eaeaea" width="5"><img alt="" src="spacer.gif" height="20" width="5"></td><td bgcolor="#eaeaea"><a target="_top" class="leftnav3" href="http://www.nature.com.ejproxy.a-star.edu.sg/natureconferences/">Nature Conferences</a></td><td bgcolor="#eaeaea" width="5"><img alt="" src="spacer.gif" height="20" width="5"></td></tr><tr><td colspan="3" height="1"><img alt="" src="spacer.gif" height="1" width="5"></td></tr></tbody></table>
		
		
<table border="0" cellpadding="0" cellspacing="0" width="130">
<tbody><tr>
<td colspan="2" height="20"><img src="spacer_001.gif" alt="" height="20" width="5"></td>
</tr>
<tr>
<td colspan="2" height="20">
<table bgcolor="#cc0033" border="0" cellpadding="0" cellspacing="0" width="130">
<tbody><tr>
<td width="5"><img src="spacer_001.gif" alt="" height="20" width="5"></td>
<td><a href="http:///" class="leftnavOn">NPG Subject areas</a></td>
</tr>
</tbody></table>
</td>
</tr>
<tr>
<td colspan="2" height="8"><img src="spacer_001.gif" alt="" height="8" width="5"></td>
</tr>
<tr>
<td><img src="areas_bio.gif" alt="" height="8" width="8"></td>
<td><a href="http://www.nature.com.ejproxy.a-star.edu.sg/biotech/" class="subject">Biotechnology</a></td>
</tr>
<tr>
<td colspan="2" height="5"><img src="spacer_001.gif" alt="" height="5" width="5"></td>
</tr>
<tr>
<td><img src="areas_can.gif" alt="" height="8" width="8"></td>
<td><a href="http://www.nature.com.ejproxy.a-star.edu.sg/cancer/" class="subject">Cancer</a></td>
</tr>
<tr>
<td colspan="2" height="5"><img src="spacer_001.gif" alt="" height="5" width="5"></td>
</tr>
<tr>
<td><img src="areas_chem.gif" alt="" height="8" width="8"></td>
<td><a href="http://www.nature.com.ejproxy.a-star.edu.sg/chemistry/" class="subject">Chemistry</a></td>
</tr>
<tr>
<td colspan="2" height="5"><img src="spacer_001.gif" alt="" height="5" width="5"></td>
</tr>
<tr>
<td><img src="areas_cli.gif" alt="" height="8" width="8"></td>
<td><a href="http://www.nature.com.ejproxy.a-star.edu.sg/clinicalmedicine/" class="subject">Clinical
Medicine</a></td>
</tr>
<tr>
<td colspan="2" height="5"><img src="spacer_001.gif" alt="" height="5" width="5"></td>
</tr>
<tr>
<td><img src="areas_den.gif" alt="" height="8" width="8"></td>
<td><a href="http://www.nature.com.ejproxy.a-star.edu.sg/dentistry/" class="subject">Dentistry</a></td>
</tr>
<tr>
<td colspan="2" height="5"><img src="spacer_001.gif" alt="" height="5" width="5"></td>
</tr>
<tr>
<td><img src="areas_dev.gif" alt="" height="8" width="8"></td>
<td><a href="http://www.nature.com.ejproxy.a-star.edu.sg/development/" class="subject">Development</a></td>
</tr>
<tr>
<td colspan="2" height="5"><img src="spacer_001.gif" alt="" height="5" width="5"></td>
</tr>
<tr>
<td><img src="areas_dru.gif" alt="" height="8" width="8"></td>
<td><a href="http://www.nature.com.ejproxy.a-star.edu.sg/drugdisc/" class="subject">Drug Discovery</a></td>
</tr>
<tr>
<td colspan="2" height="5"><img src="spacer_001.gif" alt="" height="5" width="5"></td>
</tr>
<tr>
<td><img src="areas_ear.gif" alt="" height="8" width="8"></td>
<td><a href="http://www.nature.com.ejproxy.a-star.edu.sg/earthsciences/" class="subject">Earth Sciences</a></td>
</tr>
<tr>
<td colspan="2" height="5"><img src="spacer_001.gif" alt="" height="5" width="5"></td>
</tr>
<tr>
<td><img src="areas_evo.gif" alt="" height="8" width="8"></td>
<td><a href="http://www.nature.com.ejproxy.a-star.edu.sg/evoeco/" class="subject">Evolution &amp; Ecology</a></td>
</tr>
<tr>
<td colspan="2" height="5"><img src="spacer_001.gif" alt="" height="5" width="5"></td>
</tr>
<tr>
<td><img src="areas_gen.gif" alt="" height="8" width="8"></td>
<td><a href="http://www.nature.com.ejproxy.a-star.edu.sg/genetics/" class="subject">Genetics</a></td>
</tr>
<tr>
<td colspan="2" height="5"><img src="spacer_001.gif" alt="" height="5" width="5"></td>
</tr>
<tr>
<td><img src="areas_imm.gif" alt="" height="8" width="8"></td>
<td><a href="http://www.nature.com.ejproxy.a-star.edu.sg/immuno/" class="subject">Immunology</a></td>
</tr>
<tr>
<td colspan="2" height="5"><img src="spacer_001.gif" alt="" height="5" width="5"></td>
</tr>
<tr>
<td><img src="areas_mat.gif" alt="" height="8" width="8"></td>
<td><a href="http://www.nature.com.ejproxy.a-star.edu.sg/materials/" class="subject">Materials Science</a></td>
</tr>
<tr>
<td colspan="2" height="5"><img src="spacer_001.gif" alt="" height="5" width="5"></td>
</tr>
<tr>
<td><img src="areas_med.gif" alt="" height="8" width="8"></td>
<td><a href="http://www.nature.com.ejproxy.a-star.edu.sg/medicalresearch/" class="subject">Medical Research</a></td>
</tr>
<tr>
<td colspan="2" height="5"><img src="spacer_001.gif" alt="" height="5" width="5"></td>
</tr>
<tr>
<td><img src="areas_mic.gif" alt="" height="8" width="8"></td>
<td><a href="http://www.nature.com.ejproxy.a-star.edu.sg/micro/" class="subject">Microbiology</a></td>
</tr>
<tr>
<td colspan="2" height="5"><img src="spacer_001.gif" alt="" height="5" width="5"></td>
</tr>
<tr>
<td><img src="areas_mol.gif" alt="" height="8" width="8"></td>
<td><a href="http://www.nature.com.ejproxy.a-star.edu.sg/molcellbio/" class="subject">Molecular Cell
Biology</a></td>
</tr>
<tr>
<td colspan="2" height="5"><img src="spacer_001.gif" alt="" height="5" width="5"></td>
</tr>
<tr>
<td><img src="areas_neu.gif" alt="" height="8" width="8"></td>
<td><a href="http://www.nature.com.ejproxy.a-star.edu.sg/neurosci/" class="subject">Neuroscience</a></td>
</tr>
<tr>
<td colspan="2" height="5"><img src="spacer_001.gif" alt="" height="5" width="5"></td>
</tr>
<tr>
<td><img src="areas_pha.gif" alt="" height="8" width="8"></td>
<td><a href="http://www.nature.com.ejproxy.a-star.edu.sg/pharma/" class="subject">Pharmacology</a></td>
</tr>
<tr>
<td colspan="2" height="5"><img src="spacer_001.gif" alt="" height="5" width="5"></td>
</tr>
<tr>
<td><img src="areas_phy.gif" alt="" height="8" width="8"></td>
<td><a href="http://www.nature.com.ejproxy.a-star.edu.sg/physics/" class="subject">Physics</a></td>
</tr>
<tr>
<td colspan="2" height="5"><img src="spacer_001.gif" alt="" height="5" width="5"></td>
</tr>
<tr>
<td colspan="2" height="6">
<table border="0" cellpadding="0" cellspacing="0" width="130">
<tbody><tr>
<td width="5"><img src="arrow_red.gif" alt="" height="5" width="3"></td>
<td><a href="http://npg.nature.com.ejproxy.a-star.edu.sg/npg/forms/04_cat.jsp" class="subjectred">Browse
all publications</a></td>
</tr>
</tbody></table>
</td>
</tr>
<tr>
<td colspan="2" height="5"><img src="spacer_001.gif" alt="" height="5" width="5"></td>
</tr>
<tr>
<td colspan="2" height="1"><img src="spacer_001.gif" alt="" height="1" width="5"></td>
</tr>
</tbody></table>
		
	</td><td width="15"><img alt="" src="spacer.gif" height="1" width="15"></td><td valign="top" width="470"><table border="0" cellpadding="0" cellspacing="0" width="470"><tbody><tr xmlns=""><td height="21" valign="top"><img alt="" src="spacer.gif" height="21" width="1"></td></tr><tr xmlns=""><td height="22" valign="top"><span class="categ_large">Article</span></td></tr><tr xmlns=""><td><img alt="" src="spacer_dotted.gif" height="1" width="470"></td></tr><tr xmlns=""><td height="14"><img alt="" src="spacer.gif" height="14" width="1"></td></tr><tr><td><span xmlns="" class="blacksml"><i>Nature Methods</i> 
		- <b>4</b>, 445 - 453 (2007)
 <br>Published online: 1 April 2007;  | doi:10.1038/nmeth1032
				
			</span>
<h2 xmlns="">Image-based multivariate profiling of drug responses from single cells</h2><p xmlns=""><span class="author">Lit-Hsin Loo, Lani F Wu&nbsp;&amp;&nbsp;Steven J Altschuler<p class="blacksml">Department of Pharmacology, University of Texas Southwestern Medical Center, 5323 Harry Hines Blvd., ND 9.214, Dallas, Texas 75390, USA.</p><span class="blacksml">
Correspondence should be addressed to Steven J Altschuler <a href="mailto:steven.altschuler@utsouthwestern.edu">steven.altschuler@utsouthwestern.edu</a></span></span></p><table xmlns="" border="0" cellpadding="0" cellspacing="0" width="450"><tbody><tr><td height="20"><img alt="" src="spacer.gif" height="20" width="1"></td></tr><tr><td height="1"><img alt="" src="spacer_dotted.gif" height="1" width="450"></td></tr><tr><td height="20"><img alt="" src="spacer.gif" height="20" width="1"></td></tr></tbody></table><a xmlns="" name="abstract"></a><span xmlns="" class="articletext"><b>Quantitative analytical approaches for discovering new compound mechanisms are required for summarizing high-throughput, image-based drug screening data. Here we present a multivariate method for classifying untreated and treated human cancer cells based on <img src="glyph.gif" style="border: 0pt none ; vertical-align: middle;" alt="approx">300 single-cell phenotypic measurements. This classification provides a score, measuring the magnitude of the drug effect, and a vector, indicating the simultaneous phenotypic changes induced by the drug. These two quantities were used to characterize compound activities and identify dose-dependent multiphasic responses. A systematic survey of profiles extracted from a 100-compound compendium of image data revealed that only 10–15% of the original features were required to detect a compound effect. We report the most informative image features for each compound and fluorescence marker set using a method that will be useful for determining minimal collections of readouts for drug screens. Our approach provides human-interpretable profiles and automatic determination of on- and off-target effects.</b></span><table xmlns="" border="0" cellpadding="0" cellspacing="0" width="470"><tbody><tr><td height="20"><img alt="" src="spacer.gif" height="20" width="1"></td></tr><tr><td height="1"><img alt="" src="spacer_dotted.gif" height="1" width="470"></td></tr><tr><td height="20"><img alt="" src="spacer.gif" height="20" width="1"></td></tr></tbody></table><span xmlns="" class="articletext">The recent increased availability of high-precision robotic liquid handling machinery, automated fluorescence microscopy and high-performance computing has enabled rapid advances in the development of high-throughput image-based assays. These assays permit quantitative observation of phenotypes from single cells, including cellular morphology, protein expression, localization and post-translational modifications. Automated image processing algorithms for cell segmentation and feature extraction offer the ability to extract objective measurements of these multidimensional phenotypes, and are particularly useful for the analysis of image data sets that are too large, or of phenotypes that are too subtle, for reliable human scoring<sup><a href="#B1">1,   
</a></sup><sup><a href="#B2">2,   
</a></sup><sup><a href="#B3">3,   
</a></sup><sup><a href="#B4">4</a></sup>. Comparisons of these measurements obtained from cells under different experimental conditions can be used to derive profiles that summarize phenotypic changes in response to different pharmacological or physiological perturbations, and presumably reveal important biological effects. Several recent pioneering studies have developed high-throughput image-based assay approaches to build profiles to characterize drug effects<sup><a href="#B5">5</a></sup>, screen for small molecules<sup><a href="#B6">6</a></sup>, classify subcellular localizations<sup><a href="#B7">7,   
</a></sup><sup><a href="#B8">8</a></sup>, and characterize whole-genome phenotypes by using RNA interference<sup><a href="#B9">9</a></sup> or gene-deletion libraries<sup><a href="#B10">10</a></sup>.</span><br xmlns=""><br xmlns=""><span xmlns="" class="articletext">Several bottlenecks in data analysis have limited the full potential of high-throughput image-based assays<sup><a href="#B2">2,   
</a></sup><sup><a href="#B4">4</a></sup>. First, one of the challenges has been to effectively transform distributions of multidimensional, phenotypic measurements from single cells into multivariate profiles that are both machine and human interpretable. Common profiling approaches based on univariate comparisons of responses miss feature correlations at the single-cell level. Multivariate profiling approaches consider dependencies among features, and may improve our ability to characterize, compare and predict cellular changes in response to external perturbations. However, profiles built on common multivariate hypothesis tests that reduce high-dimensional data to a single test statistic may have limited ability to discriminate among different classes of perturbations. Second, beyond the standard challenges of image preprocessing, cell segmentation and feature extraction, which are partially solved by available automated image analysis software<sup><a href="#B2">2</a></sup>, it is in fact not obvious which or how many features should be measured<sup><a href="#B2">2</a></sup>. Current image-based assays for drug screening usually compute a small number of features (&lt;10) selected by experts<sup><a href="#B11">11,   
</a></sup><sup><a href="#B12">12</a></sup>. An unbiased approach allowing for the discovery of unexpected phenotypes calls for the inclusion of many objective measurements. Inclusion of irrelevant features, however, not only increases the overhead of computation and storage, but also reduces the sensitivity of the data analysis. A final challenge has been to determine the effective dosage range and quantify possible multiphasic response of a compound. The traditional dose-response curve based on viable cell counts may not be able to distinguish between different responses of a compound within effective concentrations. Overcoming this challenge is essential for discovering new, dose-dependent actions of compounds by image-based assays.</span><br xmlns=""><br xmlns=""><span xmlns="" class="articletext">Here we present an approach for computing profiles that addresses the challenges discussed above. First, we used a support vector machine (SVM)<sup><a href="#B13">13</a></sup> algorithm to find 'optimal' hyperplanes separating treated and untreated populations of cells in multidimensional feature space. The hyperplane provides two easily interpretable quantities: its classification accuracy score, indicating the degree to which it separates the populations, and its normal vector, used as a multivariate profile, indicating the 'phenotypic direction' of greatest separation between the two cellular populations. Second, we used an algorithm to iteratively remove sets of features without a loss of classification accuracy and determined the most informative feature for each compound and fluorescence marker set. Unlike dimensionality-reduction approaches based on data transformation, such as principal component analysis<sup><a href="#B14">14</a></sup>, we did not transform the coordinates of our original feature space and thereby retained human interpretability of the final profiles. Finally, we developed a dosage-clustering algorithm for partitioning dosage series into distinct compound responses, and used the classification accuracy score to detect effective single or multiphasic responses from these clusters of dosage series.</span><br xmlns=""><br xmlns=""><span xmlns="" class="articletext">We applied our methods to a compendium of 100 compounds. We identified single or multiphasic responses for these compounds, and systematically compared the importance of different fluorescence marker sets and image features on characterizing these compounds. Compared with a previous univariate analysis performed on the same data set<sup><a href="#B5">5</a></sup>, our new approach provided profiles that gave better classification accuracies, and detected effects of several categories of compounds that were completely missed. Additionally, using small sets of informative features, we were able to detect phenotypic changes that could be used to group compounds almost perfectly into clusters according to their known targets and revealed interesting associations between different categories of compounds.</span><br xmlns=""><br xmlns=""><a xmlns="" name="Results"></a><span xmlns="" class="heading3">Results</span><br xmlns=""><a xmlns="" name="Overview of the multivariate profiling method"></a><span xmlns="" class="articletext"><b>Overview of the multivariate profiling method</b></span><br xmlns=""><span xmlns="" class="articletext">For a given compound, our method for constructing multivariate dose-dependent profiles consisted of five major steps (<a href="#f1">Fig. 1</a>). In the first step, we applied low-level image preprocessing, cell segmentation and image-feature extraction algorithms to fluorescence microscopy images of treated and control cells. The phenotype of each cell was represented by a vector of measured values in the multidimensional feature space. The phenotypes of the populations of treated and control cells were thereby represented as two distributions of points within the feature space (<a href="#f1">Fig. 1a</a>). These two distributions might be highly overlapping at low compound dosages but easily separable at high compound dosages.</span><br xmlns=""><br xmlns=""><a xmlns="" name="f1"></a><table xmlns="" border="0" cellpadding="10" cellspacing="0" width="470"><tbody><tr><td><img alt="" src="spacer.gif" height="2" width="1"></td></tr><tr><td bgcolor="#d1e1ea"><table border="0" cellpadding="0" cellspacing="0" width="100%"><tbody><tr><td class="articletext" colspan="2"><b><a href="http://www.nature.com.ejproxy.a-star.edu.sg/nmeth/journal/v4/n5/fig_tab/nmeth1032_F1.html">Figure&nbsp;1.</a>&nbsp;Representing multivariate changes in cellular phenotypes by human-interpretable vectors.</b></td></tr><tr><td height="15"><img alt="" src="spacer.gif" height="1" width="160"></td><td height="15"><img alt="" src="spacer.gif" height="1" width="100%"></td></tr><tr><td valign="top" width="160"><a href="http://www.nature.com.ejproxy.a-star.edu.sg/nmeth/journal/v4/n5/fig_tab/nmeth1032_F1.html"><img alt="Figure&nbsp;1 thumbnail" src="nmeth1032-f1.jpg" border="0"></a></td><td class="blacksml" valign="top" width="270"><p class="blacksml">(<b>a</b>) Multidimensional feature extraction. Immunofluorescence images of drug-treated cells were analyzed to reveal phenotypic changes. Measurements of intensity, texture and morphology were used to represent cells as points in a high-dimensional feature space (cells treated with low compound concentration, light red points; high compound concentration, dark red points; control, white points). An SVM determined the optimum hyperplane separating cells into treated and untreated classes for each compound concentration (white hyperplanes). Hyperplane orientations are specified by weight vectors (<i>W</i><sub>3</sub> and <i>W</i><sub>10</sub>, for concentration indices 3 and 10, respectively). (<b>b</b>) Multivariate profile and classification accuracy extraction. The weight vector <i>W</i><sub><i>i</i></sub> defined a profile of the compound at the concentration index <i>i</i> (top). Its performance was evaluated by the classification accuracy of the hyperplane (bottom; dashed line indicates significance threshold at <i>P</i> = 0.05). (<b>c</b>) Recursive feature removal. Features with little discriminatory information were removed from each profile (<a href="http://www.nature.com.ejproxy.a-star.edu.sg/nmeth/journal/v4/n5/suppinfo/nmeth1032_S1.html">Supplementary Note</a>). The removed features were replaced with zero values. (<b>d</b>) Titration clustering. Similar profiles within a dosage series were clustered and averaged to yield a representative profile for the cluster ('d-profile'). Profiles whose classification accuracies were not significant (<b>b</b>) were discarded from further analysis. Multiple profiles from a single compound represented dosage-dependent effects. (<b>e</b>) Application of interpretable profiles. Profiles obtained from a library of compounds were used for drug screening (left), phenotypic change discovery (middle) and category prediction (right).</p><br><br><a href="http://www.nature.com.ejproxy.a-star.edu.sg/nmeth/journal/v4/n5/fig_tab/nmeth1032_F1.html"><img alt="Full Figure" src="fullsize_icon.gif" align="bottom" border="0" height="11" width="17"></a><a class="sml" href="http://www.nature.com.ejproxy.a-star.edu.sg/nmeth/journal/v4/n5/fig_tab/nmeth1032_F1.html">Full Figure and legend (95K)</a></td></tr></tbody></table></td></tr><tr><td><img alt="" src="spacer.gif" height="2" width="1"></td></tr></tbody></table><span xmlns="" class="articletext">In the second step, for each dosage, we used an SVM algorithm<sup><a href="#B13">13</a></sup> with linear kernel function (<a href="http://www.nature.com.ejproxy.a-star.edu.sg/nmeth/journal/v4/n5/suppinfo/nmeth1032_S1.html">Supplementary Methods</a> online) to determine an 'optimal' separating hyperplane between treated and control distributions. The unit normal vector to the hyperplane indicates the direction of maximum separation between the two distributions, and its coefficients indicate the relative importance of each feature in classifying treated and control cells. The classification accuracy (ranging from 50% (random) to 100% (perfect)) reflects the separability of the two cellular distributions (<a href="#f1">Fig. 1b</a>). A single compound at different dosages, and different compounds with different targets, may induce different phenotypic changes. Thus, unit normal vectors associated with hyperplanes of significant classification accuracies could be used as multivariate compound-dosage profiles.</span><br xmlns=""><br xmlns=""><span xmlns="" class="articletext">In the third step, for each extracted profile, we removed redundant and noninformative features using an SVM recursive feature elimination (SVMRFE) algorithm<sup><a href="#B15">15</a></sup> (<a href="#f1">Fig. 1c</a>). We also evaluated several other feature-selection approaches for classification accuracy, random feature selection percentage, feature selection number and computational time (<a href="http://www.nature.com.ejproxy.a-star.edu.sg/nmeth/journal/v4/n5/suppinfo/nmeth1032_S1.html">Supplementary Note</a> online).</span><br xmlns=""><br xmlns=""><span xmlns="" class="articletext">In the fourth step, we partitioned the titration series for each compound into ranges with minimum profile dissimilarity (<a href="#f1">Fig. 1d</a>). For each partition, we obtained a representative dosage range profile (d-profile) by averaging the partition's constituent profiles that were both statistically significant (<a href="#f1">Fig. 1b</a>) and reproducible (as determined by comparing replicate experiments). Multiple d-profiles for a compound suggested possible multiphasic responses. Clusters with no d-profiles were discarded from further analysis, allowing the automated removal of poorly reproducible or ineffective dosages.</span><br xmlns=""><br xmlns=""><span xmlns="" class="articletext">In the fifth step, we used the d-profiles extracted for a library of compounds in typical applications of high-throughput image-based assays, such as drug screening, phenotypic change detection and category prediction (<a href="#f1">Fig. 1e</a>). For drug screening, we selected the compounds with d-profiles most similar to that of a reference d-profile to be lead candidates. For phenotypic change detection, we selected and compared informative features for a subset of the profiles that gave the best drug screening performance. Finally, for category prediction, we inferred the category of an 'uncharacterized' compound from previously categorized compounds with similar d-profiles.</span><br xmlns=""><br xmlns=""><span xmlns="" class="articletext">Our method provides a general and modular framework for computing multivariate profiles of drug responses. Depending upon the specific application, other cell segmentation, multivariate feature selection and clustering validation algorithms may be substituted for the ones presented here.</span><br xmlns=""><br xmlns=""><a xmlns="" name="Phenotypic profiling of the 100-compound compendium"></a><span xmlns="" class="articletext"><b>Phenotypic profiling of the 100-compound compendium</b></span><br xmlns=""><span xmlns="" class="articletext">To assess the performance of our multivariate approach, we applied our methods to a compendium of fluorescence microscopy images<sup><a href="#B5">5</a></sup> in which HeLa cells were treated with 100 compounds, dissolved in dimethyl sulfoxide (DMSO), over 13 threefold titrations (<a href="http://www.nature.com.ejproxy.a-star.edu.sg/nmeth/journal/v4/n5/suppinfo/nmeth1032_S1.html">Supplementary Methods</a>). The compounds represented <img src="glyph.gif" style="border: 0pt none ; vertical-align: middle;" alt="approx">20 categories of activities, selected to cover mechanisms of toxicity, signaling pathways and therapeutic targets in cancer and other diseases (<a href="http://www.nature.com.ejproxy.a-star.edu.sg/nmeth/journal/v4/n5/suppinfo/nmeth1032_S1.html">Supplementary Methods</a>). We assayed compound effects in duplicate on 384-well plates, using four sets of multiplexed molecular markers: DNA-SC35-anillin, DNA-p53-cFos, DNA-pp38-pERK and DNA-microtubule-actin (<a href="#f2">Fig. 2a</a>). On average, 2,413 <img src="glyph_001.gif" style="border: 0pt none ; vertical-align: baseline;" alt="plusminus"> 852 (mean <img src="glyph_001.gif" style="border: 0pt none ; vertical-align: baseline;" alt="plusminus"> s.d.) cells were captured per well, from 103,580 images per marker set, to yield a total of <img src="glyph.gif" style="border: 0pt none ; vertical-align: middle;" alt="approx">37 million individual identified cells. As controls, we used cells treated with DMSO alone.</span><br xmlns=""><br xmlns=""><a xmlns="" name="f2"></a><table xmlns="" border="0" cellpadding="10" cellspacing="0" width="470"><tbody><tr><td><img alt="" src="spacer.gif" height="2" width="1"></td></tr><tr><td bgcolor="#d1e1ea"><table border="0" cellpadding="0" cellspacing="0" width="100%"><tbody><tr><td class="articletext" colspan="2"><b><a href="http://www.nature.com.ejproxy.a-star.edu.sg/nmeth/journal/v4/n5/fig_tab/nmeth1032_F2.html">Figure&nbsp;2.</a>&nbsp;Drugs effects can be detected by a small number of features.</b></td></tr><tr><td height="15"><img alt="" src="spacer.gif" height="1" width="160"></td><td height="15"><img alt="" src="spacer.gif" height="1" width="100%"></td></tr><tr><td valign="top" width="160"><a href="http://www.nature.com.ejproxy.a-star.edu.sg/nmeth/journal/v4/n5/fig_tab/nmeth1032_F2.html"><img alt="Figure&nbsp;2 thumbnail" src="nmeth1032-f2.jpg" border="0"></a></td><td class="blacksml" valign="top" width="270"><p class="blacksml">(<b>a</b>) Immunofluorescence images of cells treated with DMSO and Taxol (concentration index 5, 5 nM). For each image, a magnified view of selected cells in mitosis and interphase is shown in an inset. All images were background subtracted and normalized to the same intensity range on each pseudocolor channel. Scale bars, 20 <img src="glyph_002.gif" style="border: 0pt none ; vertical-align: middle;" alt="mu">m. (<b>b</b>) During feature removal, a set of profiles with decrementing dimensions was produced where less important features were iteratively removed (top). The classification accuracy associated with each profile remained relatively constant as features with little discriminatory information were removed from the whole feature set (bottom). The trend continued until features critical for discriminating compound effects were removed, after which the classification accuracy started to decrease. The number of retained features was chosen to be the minimum number of features that gave classification accuracy close to the overall maxima (dashed line; <a href="http://www.nature.com.ejproxy.a-star.edu.sg/nmeth/journal/v4/n5/suppinfo/nmeth1032_S1.html">Supplementary Note</a>). The example shown is feature removal for the profile of camptothecin (concentration index 6, 16 nM). The number of retained features is 20. (<b>c</b>) Image feature category selection depends on drug category and choice of fluorescent marker set. For each feature, the mean of the absolute values of the corresponding coefficients in all d-profiles from a compound category was calculated. The importance for a feature category was calculated as the sum of the ten features with the largest mean values.</p><br><br><a href="http://www.nature.com.ejproxy.a-star.edu.sg/nmeth/journal/v4/n5/fig_tab/nmeth1032_F2.html"><img alt="Full Figure" src="fullsize_icon.gif" align="bottom" border="0" height="11" width="17"></a><a class="sml" href="http://www.nature.com.ejproxy.a-star.edu.sg/nmeth/journal/v4/n5/fig_tab/nmeth1032_F2.html">Full Figure and legend (89K)</a></td></tr></tbody></table></td></tr><tr><td><img alt="" src="spacer.gif" height="2" width="1"></td></tr></tbody></table><span xmlns="" class="articletext">To gather a comprehensive collection of phenotypic measurements for each marker set and each cell, we computed the values of 296 image features from the DNA and nonDNA regions (<a href="http://www.nature.com.ejproxy.a-star.edu.sg/nmeth/journal/v4/n5/suppinfo/nmeth1032_S1.html">Supplementary Methods</a>), including 14 morphology features (measuring shape properties of the nuclear and cellular domains), 24 intensity features (measuring the expression levels of the stained proteins in different cellular compartments), 78 Haralick texture features<sup><a href="#B16">16</a></sup> (measuring the spatial patterns of stained proteins), 13 moment features and 147 Zernike features<sup><a href="#B7">7,   
</a></sup><sup><a href="#B17">17</a></sup> (both measuring the mass distributions of stained proteins). Although most of these features were derived from the measurements of individual markers, some features measured information from more than one marker (such as the spatial correlation between the intensities of two different markers). To test the robustness of our method in removing irrelevant features, we also included 20 features with randomly generated values (uniformly distributed between –1 and 1).</span><br xmlns=""><br xmlns=""><a xmlns="" name="Importance of features and marker sets"></a><span xmlns="" class="articletext"><b>Importance of features and marker sets</b></span><br xmlns=""><span xmlns="" class="articletext">Many of the phenotypic changes were readily observable, whereas others were less obvious or too subtle to be discerned by eye (<a href="#f2">Fig. 2a</a>). For most of the compounds, automated feature elimination reduced the number of selected features needed for optimal classification to around 20–40 features (<a href="http://www.nature.com.ejproxy.a-star.edu.sg/nmeth/journal/v4/n5/suppinfo/nmeth1032_S1.html">Supplementary Fig. 1</a> online), indicating that the original feature set was highly redundant for any particular compound. Random features were consistently eliminated early in the iterative process (<a href="#f2">Fig. 2b</a>) demonstrating the effectiveness of feature selection to automatically remove features with little discriminative information.</span><br xmlns=""><br xmlns=""><span xmlns="" class="articletext">We compared the importance of all feature categories across different compounds on the same marker set. Despite the consistency in the number of selected features, the types of retained features were highly diverse (<a href="#f2">Fig. 2c</a>). For example, on the DNA-SC35-anillin marker set, texture features were more important for cholesterol inhibitors, but less important for compounds such as actin and DNA replication inhibitors. Overall, profile coefficients corresponding to texture and intensity features had the highest absolute values, whereas morphology, Zernike and moment features had comparatively lower absolute values.</span><br xmlns=""><br xmlns=""><span xmlns="" class="articletext">Next we compared the importance of all feature categories across different marker sets on the same compound. In general, texture features were more important on the DNA-SC35-anillin and DNA-MT-actin marker sets than on the DNA-cFos-p53 and DNA-pp38-pERK marker sets, whereas the reverse was true for intensity features (<a href="#f2">Fig. 2c</a>). The results suggested that spatial pattern information was most relevant on the markers measuring cytoskeleton (DNA-MT-actin) or proteins with cell cycle–dependent localization (DNA-SC35-anillin), and intensity information was most relevant on the markers measuring transcription factors (DNA-cFos-p53) or cell-signaling proteins (DNA-pp38-pERK).</span><br xmlns=""><br xmlns=""><a xmlns="" name="d-profile extraction"></a><span xmlns="" class="articletext"><b>d-profile extraction</b></span><br xmlns=""><span xmlns="" class="articletext">We considered compound effects significant only when our ability to separate treated from control cells was significantly greater than our ability to separate control cells from different replicated wells. The classification accuracy significance threshold at <i>P</i> = 0.05 estimated on every plate was much higher than 50% (<a href="#f3">Fig. 3</a>), and occasionally reached as high as 90%. The high well-to-well variability might be due in part to the microenvironment difference in each well on microtiter plates<sup><a href="#B18">18,   
</a></sup><sup><a href="#B19">19</a></sup>, and was accounted for in our null distribution estimation process.</span><br xmlns=""><br xmlns=""><a xmlns="" name="f3"></a><table xmlns="" border="0" cellpadding="10" cellspacing="0" width="470"><tbody><tr><td><img alt="" src="spacer.gif" height="2" width="1"></td></tr><tr><td bgcolor="#d1e1ea"><table border="0" cellpadding="0" cellspacing="0" width="100%"><tbody><tr><td class="articletext" colspan="2"><b><a href="http://www.nature.com.ejproxy.a-star.edu.sg/nmeth/journal/v4/n5/fig_tab/nmeth1032_F3.html">Figure&nbsp;3.</a>&nbsp;Titration clustering reveals multiphasic drug effects.</b></td></tr><tr><td height="15"><img alt="" src="spacer.gif" height="1" width="160"></td><td height="15"><img alt="" src="spacer.gif" height="1" width="100%"></td></tr><tr><td valign="top" width="160"><a href="http://www.nature.com.ejproxy.a-star.edu.sg/nmeth/journal/v4/n5/fig_tab/nmeth1032_F3.html"><img alt="Figure&nbsp;3 thumbnail" src="nmeth1032-f3.jpg" border="0"></a></td><td class="blacksml" valign="top" width="270"><p class="blacksml">(<b>a</b>–<b>d</b>) Titration clustering grouped profiles <i>W</i><sub><i>i</i></sub> obtained at different concentration index <i>i</i> into 2 or 3 clusters. Hydroxy urea, 2 clusters (<b>a</b>); oxamflatin, 3 clusters (<b>b</b>); taxol, 2 clusters (<b>c</b>); camptothecin, 3 clusters (<b>d</b>). The numbers of clusters were determined automatically using a clustering validation algorithm in the original feature space (<a href="http://www.nature.com.ejproxy.a-star.edu.sg/nmeth/journal/v4/n5/suppinfo/nmeth1032_S1.html">Supplementary Methods</a>). The relative similarities among profiles and the clustering results were visualized using multidimensional scaling plots (top; square with or without cross, profile used or not used for d-profile determination). The classification accuracy and reproducibility of each profile were used to decide if the profile will be retained (bottom; solid blue line, classification accuracy on the first replicate; error bars, standard error of the cross-validation estimation of average classification accuracy; black dashed line, classification accuracy with <i>P</i> = 0.05 on the first replicate). For each cluster, a d-profile was derived from averaging the profiles with significant classification accuracy (above the dashed line) and reproducible across two replicates (triangle). Most compounds had no effect at the lowest dosage cluster, leaving 1 or 2 d-profiles representing primary and secondary compound effects (1 d-profile, hydroxy urea and oxamflatin; 2 d-profiles, taxol and camptothecin).</p><br><br><a href="http://www.nature.com.ejproxy.a-star.edu.sg/nmeth/journal/v4/n5/fig_tab/nmeth1032_F3.html"><img alt="Full Figure" src="fullsize_icon.gif" align="bottom" border="0" height="11" width="17"></a><a class="sml" href="http://www.nature.com.ejproxy.a-star.edu.sg/nmeth/journal/v4/n5/fig_tab/nmeth1032_F3.html">Full Figure and legend (75K)</a></td></tr></tbody></table></td></tr><tr><td><img alt="" src="spacer.gif" height="2" width="1"></td></tr></tbody></table><span xmlns="" class="articletext">The classification accuracy curves of most compounds showed classical sigmoidal dose responses, with classification accuracies below the significance threshold at the lowest dosage ranges and well above the significance threshold at the highest dosage ranges. For several compounds, classification accuracies trended slightly upwards for decreasing concentrations at the lowest dosages (<a href="#f3">Fig. 3</a>), likely due to microtiter-plate-edge effects. Low dosages with significant classification accuracies were usually eliminated from computation of the final d-profiles as their reproducibility scores were mostly below threshold.</span><br xmlns=""><br xmlns=""><span xmlns="" class="articletext">The titration clustering algorithm yielded two clusters per compound on 65% of the compounds (<a href="#f3">Fig. 3a,c</a>), and three clusters per compound on 35% of the compounds (<a href="#f3">Fig. 3b,d</a>) over all four marker sets. Visualizations of the inter-profile similarities using multidimensional scaling<sup><a href="#B20">20</a></sup> showed qualitatively that the profiles largely grouped together by contiguous concentration ranges, with occasional 'jumps' to distinct regions in the feature space (<a href="#f3">Fig. 3</a>), indicating the possibility of having multiple 'clusters' in drug responses. After removing profiles that were neither significant nor reproducible, one d-profile per compound was derived for 60% of the compounds (<a href="#f3">Fig. 3a,b</a>) and two or three d-profiles per compound were derived for 18% of the compounds (<a href="#f3">Fig. 3c,d</a>), corresponding to possible distinct dosage-dependent effects. The remaining 22% of compounds did not give any d-profiles. In total, from the 100-compound compendium, we extracted 100, 100, 89 and 102 d-profiles using the DNA-SC35-anillin, DNA-p53-cFos, DNA-pp38-pERK and DNA-MT-actin marker sets, respectively.</span><br xmlns=""><br xmlns=""><span xmlns="" class="articletext">Across different marker sets, 73% of the compounds gave the same number of d-profiles on three or four marker sets. For example, taxol consistently gave 2 d-profiles (<a href="#f3">Fig. 3c</a>) across all four marker sets (concentration indices 4–5, 2–5 nM and concentration indices 7–13, 47 nM–35 <img src="glyph_002.gif" style="border: 0pt none ; vertical-align: middle;" alt="mu">M). This result, based on multivariate phenotypic change measurements, matched a previous result<sup><a href="#B21">21</a></sup> reporting a biphasic response of taxol on breast cancer cell lines based on simple viable cell counts (first phase, 5–50 nM and second phase, 5–50 <img src="glyph_002.gif" style="border: 0pt none ; vertical-align: middle;" alt="mu">M).</span><br xmlns=""><br xmlns=""><a xmlns="" name="Drug-screening performance"></a><span xmlns="" class="articletext"><b>Drug-screening performance</b></span><br xmlns=""><span xmlns="" class="articletext">To simulate a drug screen for compounds of similar target to a known compound, we selected a d-profile to be the reference profile and used all other d-profiles from the compendium as 'blinded' test profiles. Dissimilarity scores between the reference profile and all other test profiles were computed and ranked. The test profiles that were least dissimilar to the reference profile were selected as 'drug candidates'.</span><br xmlns=""><br xmlns=""><span xmlns="" class="articletext">For each reference profile, we estimated its drug screening performance by using prior target annotations as the gold standard. We defined on-target effects as d-profiles whose drug screening performance values were significant under prior target annotations (<i>P</i> &lt; 0.05, area under the receiver operating characteristic curve (AUC)), and defined all other d-profiles as off-target. Notably, many of the compounds with more than one d-profile and at least one on-target d-profile had at least one off-target d-profile (73%, 40%, 67% and 56% on the DNA-SC35-anillin, DNA-p53-cFos, DNA-pp38-pERK and DNA-MT-actin marker sets, respectively; <a href="http://www.nature.com.ejproxy.a-star.edu.sg/nmeth/journal/v4/n5/suppinfo/nmeth1032_S1.html">Supplementary Data</a> online). For example, camptothecin had one on-target effect (concentration indices 5–9) and one off-target effect (concentration indices 10–13). This observation suggested that our approach could identify dose-dependent secondary responses that were very different from the primary responses.</span><br xmlns=""><br xmlns=""><span xmlns="" class="articletext">To summarize drug-screening performance results, we averaged the performance values of the compounds that had been annotated with the same target category for each marker set (<a href="#t1">Table 1</a>). Many compound categories gave statistically significant performance values (50%, 32%, 27% and 27% of categories in DNA-SC35-anillin, DNA-p53-cFos, DNA-pp38-pERK and DNA-MT-actin, respectively), even though secondary or tertiary d-profiles were included in the averaging process. Three compound categories (cholesterol, DNA replication and MAPK/ERK pathway inhibitors) gave perfect drug screening performance (AUC value = 1). Comparisons of drug screening performance between this multivariate profiling approach and a previous univariate approach<sup><a href="#B5">5</a></sup> showed better compound categorizations (<a href="http://www.nature.com.ejproxy.a-star.edu.sg/nmeth/journal/v4/n5/suppinfo/nmeth1032_S1.html">Supplementary Note</a>).</span><br xmlns=""><br xmlns=""><a xmlns="" name="t1"></a><table xmlns="" border="0" cellpadding="10" cellspacing="0" width="470"><tbody><tr><td><img alt="" src="spacer.gif" height="2" width="1"></td></tr><tr><td bgcolor="#d1e1ea"><table border="0" cellpadding="0" cellspacing="0" width="100%"><tbody><tr><td class="articletext" colspan="2"><b><a href="http://www.nature.com.ejproxy.a-star.edu.sg/nmeth/journal/v4/n5/fig_tab/nmeth1032_T1.html">Table&nbsp;1.</a>&nbsp;Drug screening performances are significant for many compound categories</b></td></tr><tr><td height="15"><img alt="" src="spacer.gif" height="1" width="160"></td><td height="15"><img alt="" src="spacer.gif" height="1" width="100%"></td></tr><tr><td valign="top" width="160"><a href="http://www.nature.com.ejproxy.a-star.edu.sg/nmeth/journal/v4/n5/fig_tab/nmeth1032_T1.html"><img alt="Table&nbsp;1 thumbnail" src="table_thumb.gif" border="0"></a></td><td class="blacksml" valign="top" width="270"><br><br><a href="http://www.nature.com.ejproxy.a-star.edu.sg/nmeth/journal/v4/n5/fig_tab/nmeth1032_T1.html"><img alt="Full Table" src="fullsize_icon.gif" align="bottom" border="0" height="11" width="17"></a><a class="sml" href="http://www.nature.com.ejproxy.a-star.edu.sg/nmeth/journal/v4/n5/fig_tab/nmeth1032_T1.html">Full Table</a></td></tr></tbody></table></td></tr><tr><td><img alt="" src="spacer.gif" height="2" width="1"></td></tr></tbody></table><span xmlns="" class="articletext">We further evaluated the performance of a compound category across different marker sets. Some compound categories induced phenotypic changes that were highly specific for the marker set used. For example, the effects of energy metabolism, PKC, protein degradation and RNA inhibitors could only be detected by the DNA-anillin-SC35 marker set, whereas the effects of MAPK/ERK pathway inhibitors could only be detected by the DNA-pp38-pERK marker set (<a href="#t1">Table 1</a>). However, actin, cholesterol, DNA replication, histone deacetylase, microtubule and vesicle trafficking inhibitors induced phenotypic changes that could be detected by using at least three of the marker sets (<a href="#t1">Table 1</a>). Most of these compounds caused cell-cycle arrest.</span><br xmlns=""><br xmlns=""><a xmlns="" name="Common-phenotypic-change detection"></a><span xmlns="" class="articletext"><b>Common-phenotypic-change detection</b></span><br xmlns=""><span xmlns="" class="articletext">We next wanted to identify a small number of features that most discriminated compound categories. For each marker set and compound category, we selected 2–3 representative on-target d-profiles with maximum average drug screening performance. The exclusion of off-target effects permitted the selection of on-target d-profiles from five compound categories not found significant in the drug screening process previously described (<a href="http://www.nature.com.ejproxy.a-star.edu.sg/nmeth/journal/v4/n5/suppinfo/nmeth1032_S1.html">Supplementary Note</a>). We then selected 10–15 features from each compound category with the highest average absolute values. We performed a hierarchical biclustering<sup><a href="#B20">20</a></sup> of all the representative d-profiles on the union set of all the selected features (<a href="#f4">Fig. 4</a> and <a href="http://www.nature.com.ejproxy.a-star.edu.sg/nmeth/journal/v4/n5/suppinfo/nmeth1032_S1.html">Supplementary Fig. 2</a> online).</span><br xmlns=""><br xmlns=""><a xmlns="" name="f4"></a><table xmlns="" border="0" cellpadding="10" cellspacing="0" width="470"><tbody><tr><td><img alt="" src="spacer.gif" height="2" width="1"></td></tr><tr><td bgcolor="#d1e1ea"><table border="0" cellpadding="0" cellspacing="0" width="100%"><tbody><tr><td class="articletext" colspan="2"><b><a href="http://www.nature.com.ejproxy.a-star.edu.sg/nmeth/journal/v4/n5/fig_tab/nmeth1032_F4.html">Figure&nbsp;4.</a>&nbsp;Common phenotypic changes are shared by different compounds from the same category.</b></td></tr><tr><td height="15"><img alt="" src="spacer.gif" height="1" width="160"></td><td height="15"><img alt="" src="spacer.gif" height="1" width="100%"></td></tr><tr><td valign="top" width="160"><a href="http://www.nature.com.ejproxy.a-star.edu.sg/nmeth/journal/v4/n5/fig_tab/nmeth1032_F4.html"><img alt="Figure&nbsp;4 thumbnail" src="nmeth1032-f4.jpg" border="0"></a></td><td class="blacksml" valign="top" width="270"><p class="blacksml">Hierarchical biclustering was performed for representative on-target d-profiles selected from each compound category. The resulting dendrograms were reordered by a leaf-ordering algorithm for the best visualization (<a href="http://www.nature.com.ejproxy.a-star.edu.sg/nmeth/journal/v4/n5/suppinfo/nmeth1032_S1.html">Supplementary Methods</a>). A heat map of d-profile coefficients is shown for the DNA-pp38-pERK marker set (rows, features; columns, compounds). The brightness of a square indicates the importance of a feature in determining a drug effect, and should not be interpreted as the actual degree of phenotypic changes. Black squares represent removed features that were less informative than other features. Common important phenotypic changes are identified as blocks of squares with similar colors and brightness. Many compounds were clustered perfectly according to their annotated target by common phenotypic changes, such as microtubule inhibitors (i) and neurotransmitter inhibitors (ii). Similarly, different compound categories were grouped into super-clusters by sharing common phenotypic changes. For example, three kinase inhibitors (CDK, PI3K pathway and MAPK/ERK pathway) (iii) had the same features with negative coefficients (iv), zero coefficients (v) and positive coefficients (vi). The phenotypic changes detection results for the other three marker sets are available in <a href="http://www.nature.com.ejproxy.a-star.edu.sg/nmeth/journal/v4/n5/suppinfo/nmeth1032_S1.html">Supplementary Figure 2</a>.</p><br><br><a href="http://www.nature.com.ejproxy.a-star.edu.sg/nmeth/journal/v4/n5/fig_tab/nmeth1032_F4.html"><img alt="Full Figure" src="fullsize_icon.gif" align="bottom" border="0" height="11" width="17"></a><a class="sml" href="http://www.nature.com.ejproxy.a-star.edu.sg/nmeth/journal/v4/n5/fig_tab/nmeth1032_F4.html">Full Figure and legend (97K)</a></td></tr></tbody></table></td></tr><tr><td><img alt="" src="spacer.gif" height="2" width="1"></td></tr></tbody></table><span xmlns="" class="articletext">As expected, as the most discriminative features from each compound category were used, we obtained near-perfect clustering of compounds by category. Some compounds were grouped together by obvious or easily interpretable phenotypic features, such as the pERK total intensity in the DNA region over non-DNA region for compounds affecting microtubules, whereas others were grouped together by nonobvious or new phenotypic features, such as the DNA gray level co-occurrence matrix mean correlation and the pp38 gray level co-occurrence matrix mean sum average for compounds annotated as neurotransmitter inhibitors (<a href="#f4">Fig. 4</a>). More interestingly, the categories themselves formed natural 'super-clusters' based on common blocks of features, which allowed the identification of common phenotypic changes among these categories. For instance, all three categories of kinase inhibitors (CDK, PI3K and MAPK/ERK) formed a super-cluster (<a href="#f4">Fig. 4</a>).</span><br xmlns=""><br xmlns=""><span xmlns="" class="articletext">To examine whether the most discriminative features reflected changes in cell-cycle phase, we implemented a heuristic cell cycle detector based on DNA features (<a href="http://www.nature.com.ejproxy.a-star.edu.sg/nmeth/journal/v4/n5/suppinfo/nmeth1032_S1.html">Supplementary Fig. 3</a> online). In many cases, the two most discriminative features showed cell cycle–dependent responses. For example, vinblastine increased the density of two subpopulations, which had lower ratios of pERK total intensity in DNA regions to non-DNA regions than the control population (<a href="#f4">Fig. 4</a> and <a href="http://www.nature.com.ejproxy.a-star.edu.sg/nmeth/journal/v4/n5/suppinfo/nmeth1032_S1.html">Supplementary Fig. 3</a>). One of the subpopulations comprised mostly mitotic cells and another subpopulation, mostly G2 cells (<a href="http://www.nature.com.ejproxy.a-star.edu.sg/nmeth/journal/v4/n5/suppinfo/nmeth1032_S1.html">Supplementary Fig. 3</a>), suggesting a G2/M arrest. Notably, our approach also detected phenotypic changes independent of cell-cycle phase. For example, wortmannin increased the pp38 average intensity over DNA average intensity in DNA region of the whole population across all cell-cycle phases (<a href="#f4">Fig. 4</a> and <a href="http://www.nature.com.ejproxy.a-star.edu.sg/nmeth/journal/v4/n5/suppinfo/nmeth1032_S1.html">Supplementary Fig. 3</a>).</span><br xmlns=""><br xmlns=""><a xmlns="" name="Category prediction"></a><span xmlns="" class="articletext"><b>Category prediction</b></span><br xmlns=""><span xmlns="" class="articletext">The compound category of a new d-profile can be inferred by comparison to a collection of previously categorized reference d-profiles (<a href="http://www.nature.com.ejproxy.a-star.edu.sg/nmeth/journal/v4/n5/suppinfo/nmeth1032_S1.html">Supplementary Data</a>). For instance, comparison of d-profiles indicated that oxamflatin is most similar to trichostatin, scriptaid and apicidin on the DNA-pp38-pERK marker set (<a href="#f5">Fig. 5a</a>). Although all of these compounds are histone deacetylase inhibitors, oxamflatin, trichostatin and scriptaid are hydroxamic acids having very different chemical structures than apicidin, a cyclic tetrapeptide<sup><a href="#B22">22</a></sup>. Similarly for the DNA-p53-cFos marker set, the DNA replication inhibitor, hydroxy urea-2, was found to be most similar to aphidicolin and methotrexate, both DNA replication inhibitors, as well as to a replicate of hydroxyl urea-2 with different starting stock concentration (<a href="#f5">Fig. 5b</a>).</span><br xmlns=""><br xmlns=""><a xmlns="" name="f5"></a><table xmlns="" border="0" cellpadding="10" cellspacing="0" width="470"><tbody><tr><td><img alt="" src="spacer.gif" height="2" width="1"></td></tr><tr><td bgcolor="#d1e1ea"><table border="0" cellpadding="0" cellspacing="0" width="100%"><tbody><tr><td class="articletext" colspan="2"><b><a href="http://www.nature.com.ejproxy.a-star.edu.sg/nmeth/journal/v4/n5/fig_tab/nmeth1032_F5.html">Figure&nbsp;5.</a>&nbsp;The category of a new d-profile can be inferred from its closest characterized d-profiles.</b></td></tr><tr><td height="15"><img alt="" src="spacer.gif" height="1" width="160"></td><td height="15"><img alt="" src="spacer.gif" height="1" width="100%"></td></tr><tr><td valign="top" width="160"><a href="http://www.nature.com.ejproxy.a-star.edu.sg/nmeth/journal/v4/n5/fig_tab/nmeth1032_F5.html"><img alt="Figure&nbsp;5 thumbnail" src="nmeth1032-f5.jpg" border="0"></a></td><td class="blacksml" valign="top" width="270"><p class="blacksml">(<b>a</b>–<b>d</b>) The three least dissimilar d-profiles to the d-profiles of oxamflatin (<b>a</b>), hydroxy urea-2 (<b>b</b>), camptothecin (<b>c</b>) and taxol (<b>d</b>) are visualized using multidimensional scaling (diamond, d-profile obtained from averaging significant and reproducible profiles in a titration cluster). The measured dissimilarity values were listed at the link between two d-profiles, and the bold link represented the least dissimilar d-profile. The 5 least dissimilar d-profiles to every d-profile considered are listed in the <a href="http://www.nature.com.ejproxy.a-star.edu.sg/nmeth/journal/v4/n5/suppinfo/nmeth1032_S1.html">Supplementary Data</a>.</p><br><br><a href="http://www.nature.com.ejproxy.a-star.edu.sg/nmeth/journal/v4/n5/fig_tab/nmeth1032_F5.html"><img alt="Full Figure" src="fullsize_icon.gif" align="bottom" border="0" height="11" width="17"></a><a class="sml" href="http://www.nature.com.ejproxy.a-star.edu.sg/nmeth/journal/v4/n5/fig_tab/nmeth1032_F5.html">Full Figure and legend (97K)</a></td></tr></tbody></table></td></tr><tr><td><img alt="" src="spacer.gif" height="2" width="1"></td></tr></tbody></table><span xmlns="" class="articletext">Category prediction for compounds with multiple d-profiles typically matched prior target annotations for at least one of their d-profiles. For camptothecin, its first d-profile was closest to another topoisomerase inhibitor, etoposide, whereas its second d-profile was closest to a CDK inhibitor, alsterpaullone (<a href="#f5">Fig. 5c</a>), in remarkable agreement with a centrosome profiling study on Chinese Hamster Ovarian (CHO) cells<sup><a href="#B23">23</a></sup>. For taxol, its first d-profile was closest to sulindac sulfide, a cyclooxygenase inhibitor, whereas its second d-profile was closest to epothilone B and griseofulvin, which stabilize microtubule assemblies similarly to taxol despite dissimilarity in chemical structures<sup><a href="#B24">24,   
</a></sup><sup><a href="#B25">25</a></sup> (<a href="#f5">Fig. 5d</a>). Microtubule depolymerizing compounds, such as 105D, nocodazole and vinblastine were further away from this group of microtubule stabilizing compounds. These results indicated that our approach has the sensitivity to distinguish compounds affecting the same target, but through different mechanisms.</span><br xmlns=""><br xmlns=""><a xmlns="" name="Discussion"></a><span xmlns="" class="articletext"><b>Discussion</b></span><br xmlns=""><span xmlns="" class="articletext">Our approach of profiling compound-dosage responses reduced <img src="glyph.gif" style="border: 0pt none ; vertical-align: middle;" alt="approx">300 unbiased single-cell phenotypic features to <img src="glyph.gif" style="border: 0pt none ; vertical-align: middle;" alt="approx">20 informative features for each compound and marker set. The large reduction in dimensionality came with greatly enhanced human interpretability of the drug response profiles and improved detection of cellular phenotypic changes, yet at little loss of classification accuracy. Analysis of these selected features suggested informative marker and feature set combinations for detecting and discriminating treated versus control-cell phenotypic changes.</span><br xmlns=""><br xmlns=""><span xmlns="" class="articletext">Determination of these feature sets will potentially streamline future screening of compounds with similar effects to a known compound or category of compounds under the same experimental conditions. But detecting new drug effects using new experimental conditions or marker sets may require the reselection of features that cannot be identified <i>a priori</i>. Although we used the SVM recursive feature elimination, other feature-selection algorithms may provide beneficial tradeoffs in computational runtime, classification accuracy and false positive rate appropriate for other data sets and applications.</span><br xmlns=""><br xmlns=""><span xmlns="" class="articletext">Separating compound dosage effects into multiple d-profiles resulted in more sensitive screening and raised the possibility of identifying new dosage-dependent mechanisms, even for previously characterized compounds. Our approach to building compound profiles is computationally and experimentally scalable; compound profiles are created independently of each other and allow for incremental growth of a compound compendium.</span><br xmlns=""><br xmlns=""><span xmlns="" class="articletext">Cytological profiling provides accurate quantification of complex phenotypic changes that are complementary to other high-throughput approaches, such as transcript profiling, and offers the potential to bring the use of cell-based assays earlier into the drug discovery process<sup><a href="#B2">2,   
</a></sup><sup><a href="#B26">26,   
</a></sup><sup><a href="#B27">27</a></sup>. Our approach is generally applicable for characterizing single-cell phenotypic changes owing to a broad range of external perturbations and internal cellular states. This work provides a platform for future studies, including identification of new drug targets, drug-combination effects and heterogeneous cellular drug responses.</span><br xmlns=""><br xmlns=""><table xmlns="" border="0" cellpadding="0" cellspacing="0" width="470"><tbody><tr><td rowspan="2" height="20"><img alt="" src="spacer.gif" height="20" width="1"></td><td align="right"><img alt="" src="arrow_black_up.gif" height="3" width="5">&nbsp;<a class="top" href="#top">Top</a></td></tr><tr><td align="right" height="7"><img alt="" src="spacer.gif" height="7" width="1"></td></tr><tr><td colspan="2" height="1"><img alt="" src="spacer_dotted.gif" height="1" width="470"></td></tr><tr><td colspan="2" height="20"><img alt="" src="spacer.gif" height="20" width="1"></td></tr></tbody></table><a xmlns="" name="Methods"></a><span xmlns="" class="heading3">Methods</span><br xmlns=""><a xmlns="" name="Data collection."></a><span xmlns="" class="articletext"><b>Data collection.</b></span><br xmlns=""><span xmlns="" class="articletext">Detailed protocols for the preparation of the samples and images for the 100-compound compendium are as previously described<sup><a href="#B5">5</a></sup>.</span><br xmlns=""><br xmlns=""><a xmlns="" name="Data analysis platform."></a><span xmlns="" class="articletext"><b>Data analysis platform.</b></span><br xmlns=""><span xmlns="" class="articletext">We performed all of the image and data analysis by using custom software written in Matlab v14.3 (Mathworks), unless otherwise indicated. The software is available to academic laboratories upon request.</span><br xmlns=""><br xmlns=""><a xmlns="" name="Image analysis."></a><span xmlns="" class="articletext"><b>Image analysis.</b></span><br xmlns=""><span xmlns="" class="articletext">We subtracted background intensities from the images to remove nonuniform background illumination by using the National Institutes of Health ImageJ software. The DNA and nonDNA regions for each and every cell were identified by a watershed segmentation algorithm (<a href="http://www.nature.com.ejproxy.a-star.edu.sg/nmeth/journal/v4/n5/suppinfo/nmeth1032_S1.html">Supplementary Methods</a>). We extracted a total of 296 features from these regions. We used previously published software<sup><a href="#B7">7</a></sup> to extract texture and Zernike features. We used a semi-automated algorithm to identify potentially bad wells (<a href="http://www.nature.com.ejproxy.a-star.edu.sg/nmeth/journal/v4/n5/suppinfo/nmeth1032_S1.html">Supplementary Methods</a>).</span><br xmlns=""><br xmlns=""><a xmlns="" name="Data resampling."></a><span xmlns="" class="articletext"><b>Data resampling.</b></span><br xmlns=""><span xmlns="" class="articletext">We resampled the treated and control wells to fixed sampling sizes before computing profiles from them. For treated wells, we resampled cells with replacement from each well at one of the nine common sampling sizes (ranging from 100 to 3,500) closest to the actual well cell count, and discarded wells with &lt;100 cells. For the control wells, we resampled a pooled population of cells with replacement from all the wells at the common sampling sizes in proportion to weights assigned to every well. We computed the weights as follows: for <i>t<sub>ij</sub></i> = 100 minus pairwise classification accuracy between control wells <i>i</i> and <i>j</i>, the eigenvector <i>v</i> corresponding to the largest eigen value of <i>T</i> = {<i>t<sub>ij</sub></i>} was computed. The components of <i>v</i>, normalized to have unit sum, provided weights for every well. Wells with higher weights tended have low pairwise classification accuracies and had higher probabilities of being sampled<sup><a href="#B28">28</a></sup>.</span><br xmlns=""><br xmlns=""><a xmlns="" name="Profile and classification accuracy computation."></a><span xmlns="" class="articletext"><b>Profile and classification accuracy computation.</b></span><br xmlns=""><span xmlns="" class="articletext">We estimated a multivariate profile and a classification accuracy score by comparing each treated population resampled from a well to a resampled control population of the same sampling size. We performed the estimation using a twofold cross-validation with 3 randomized fold divisions for a total of 6 trials. The computation of the hyperplane was performed by libsvm v2.81 (<a target="_top" href="http://www.csie.ntu.edu.tw/%7Ecjlin/libsvm">http://www.csie.ntu.edu.tw/~cjlin/libsvm</a>). To reduce the computational runtime, we eliminated subsets of 20 features at each iteration (or halved if the number of feature was &lt;20; <a href="#f2">Fig. 2b</a>). Profiles were oriented toward the direction of the treated population and scaled to have unit sum of the absolute values of their coefficients. We estimated final profile and classification accuracies from the average of the trials.</span><br xmlns=""><br xmlns=""><a xmlns="" name="Feature preprocessing."></a><span xmlns="" class="articletext"><b>Feature preprocessing.</b></span><br xmlns=""><span xmlns="" class="articletext">During each trial of the above cross-validation, for each feature, we set the outliers outside the 1st and 99th percentile values determined from the training fold to the 1st and 99th percentile values respectively, and scaled the values of each feature linearly to the range of (-1, 1).</span><br xmlns=""><br xmlns=""><a xmlns="" name="Profile dissimilarity."></a><span xmlns="" class="articletext"><b>Profile dissimilarity.</b></span><br xmlns=""><span xmlns="" class="articletext">We measured the dissimilarity between profiles <i>x<sub>r</sub></i> and <i>x<sub>s</sub></i> by:</span><br xmlns=""><br xmlns=""><span xmlns="" class="articletext"><a name="eq1"><!-- . --></a><img src="nmeth1032-m1.gif"></span><br xmlns=""><br xmlns=""><span xmlns="" class="articletext">where <i>x</i>' is the vector transpose of <i>x</i>.</span><br xmlns=""><br xmlns=""><a xmlns="" name="Profile reproducibility."></a><span xmlns="" class="articletext"><b>Profile reproducibility.</b></span><br xmlns=""><span xmlns="" class="articletext">We flagged a profile as 'irreproducible' if its dissimilarity to its replicates is more than its dissimilarities to 5% of all other profiles associated with significant classification accuracies.</span><br xmlns=""><br xmlns=""><a xmlns="" name="Combinatorial titration clustering."></a><span xmlns="" class="articletext"><b>Combinatorial titration clustering.</b></span><br xmlns=""><span xmlns="" class="articletext">For a given number of clusters, <i>k</i>, we used a combinatorial titration clustering algorithm to partition the titration series into <i>k</i> ranges having a minimum between-cluster point scatter<sup><a href="#B20">20</a></sup>. We determined the optimum number of clusters automatically by choosing the <i>k</i> giving the most stable clustering (<a href="http://www.nature.com.ejproxy.a-star.edu.sg/nmeth/journal/v4/n5/suppinfo/nmeth1032_S1.html">Supplementary Methods</a>). Owing to the small number of profiles (<img src="glyph.gif" style="border: 0pt none ; vertical-align: middle;" alt="approx">13), we only considered <i>k</i> = {2, 3}.</span><br xmlns=""><br xmlns=""><a xmlns="" name="Representative d-profiles selection."></a><span xmlns="" class="articletext"><b>Representative d-profiles selection.</b></span><br xmlns=""><span xmlns="" class="articletext">We selected representative d-profiles from compound categories having d-profiles from at least two distinct compounds. Each compound was only allowed to contribute one d-profile. For a category with <i>n<sub>d</sub></i> d-profiles, we searched exhaustively for the largest d-profile subset with significant AUC (average <i>P</i> value &lt; 0.05). For ties, we selected the subset with the minimum average <i>P</i> value. When <i>n<sub>d</sub></i> &gt; 15, we used a suboptimal procedure to select representative d-profiles (<a href="http://www.nature.com.ejproxy.a-star.edu.sg/nmeth/journal/v4/n5/suppinfo/nmeth1032_S1.html">Supplementary Methods</a>). Finally, if the selected subset had more than 3 d-profiles, we selected only the 3 d-profiles with the minimum <i>P</i> values; otherwise, we selected all the d-profiles.</span><br xmlns=""><br xmlns=""><a xmlns="" name="Significance analysis."></a><span xmlns="" class="articletext"><b>Significance analysis.</b></span><br xmlns=""><span xmlns="" class="articletext">For each sampling size, we estimated the null distribution of classification accuracy between the pooled control population and each control well. Each control well was resampled similar to the treated wells (see data resampling above). We estimated the null distribution of classification accuracy by a Parzen-window approach<sup><a href="#B20">20</a></sup> with Gaussian kernel and s.d. of 0.02. We adjusted the number of resamplings from 1 (for sampling size of 3,500) to 128 (for sampling size of 100). The significance threshold was determined at <i>P</i> = 0.05 (one tailed).</span><br xmlns=""><br xmlns=""><span xmlns="" class="articletext">Given a reference d-profile and its dissimilarity scores to a set of test d-profiles, we scanned the drug candidate selection threshold from 0 to the maximum dissimilarity and plotted the corresponding receiver operating characteristic curve (true versus false positive rates). The area under the receiver operating characteristic curve (AUC) was estimated using the trapezoidal integration, and is equal to the Wilcoxon statistic<sup><a href="#B29">29</a></sup>. For a large number of samples, the null distribution of Wilcoxon statistic approximates the<i>z</i>-distribution. The significance threshold was determined at <i>P</i> = 0.05 (one tailed).</span><br xmlns=""><br xmlns=""><a xmlns="" name="Additional methods."></a><span xmlns="" class="articletext"><b>Additional methods.</b></span><br xmlns=""><span xmlns="" class="articletext">Information on the stock plate layout, compound and feature lists, image processing, feature selection and clustering validation algorithms, and quality control process is available in <a href="http://www.nature.com.ejproxy.a-star.edu.sg/nmeth/journal/v4/n5/suppinfo/nmeth1032_S1.html">Supplementary Methods</a> and <a href="http://www.nature.com.ejproxy.a-star.edu.sg/nmeth/journal/v4/n5/suppinfo/nmeth1032_S1.html">Supplementary Note</a>.</span><br xmlns=""><br xmlns=""><a xmlns="" name="Author contributions"></a><span xmlns="" class="articletext"><b>Author contributions</b></span><br xmlns=""><span xmlns="" class="articletext">L.-H.L. designed, implemented and performed the profiling methods. All authors contributed to the conception of the overall approach, statistical analysis of the methods and writing of the manuscript.</span><br xmlns=""><br xmlns=""><span xmlns="" class="articletext"><i>Note: <a href="http://www.nature.com.ejproxy.a-star.edu.sg/nmeth/journal/v4/n5/suppinfo/nmeth1032_S1.html">Supplementary information</a> is available on the Nature Methods website</i>.</span><br xmlns=""><br xmlns=""><table xmlns="" border="0" cellpadding="0" cellspacing="0" width="470"><tbody><tr><td rowspan="2" height="20"><img alt="" src="spacer.gif" height="20" width="1"></td><td align="right"><img alt="" src="arrow_black_up.gif" height="3" width="5">&nbsp;<a class="top" href="#top">Top</a></td></tr><tr><td align="right" height="7"><img alt="" src="spacer.gif" height="7" width="1"></td></tr><tr><td colspan="2" height="1"><img alt="" src="spacer_dotted.gif" height="1" width="470"></td></tr><tr><td colspan="2" height="20"><img alt="" src="spacer.gif" height="20" width="1"></td></tr></tbody></table><span xmlns="" class="articletext">Received 25 August 2006; </span><span xmlns="" class="articletext">Accepted 21 February 2007; </span><span xmlns="" class="articletext">Published online: 1 April 2007.</span><br xmlns=""><br xmlns=""><a xmlns="" name="References"></a><span xmlns="" class="heading3">REFERENCES </span><ol xmlns="" class="articletext">
<li id="B1"><a name="B1"><!-- . --></a>Boland, M.V. &amp; Murphy, R.F. <span class="atl"> After sequencing: quantitative analysis of protein localization</span>. <span class="journalname">IEEE Eng. Med. Biol. Mag.</span> <span class="journalnumber">18</span>, 115–119 (<span class="cite-month-year">1999</span>).&nbsp;|&nbsp;<a href="http://dx.doi.org.ejproxy.a-star.edu.sg/10.1109/51.790995" class="reftxt" title="After sequencing: quantitative analysis of protein localization">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.ejproxy.a-star.edu.sg/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=10497746&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - After sequencing: quantitative analysis of protein localization">PubMed</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:STN:280:DyaK1MvisleqtA%3D%3D&amp;pissn=1548-7091&amp;pyear=2007&amp;md5=4fdd075552d0d3f7fd583d72bcce40fe" class="reftxt" title="Article on ChemPort - After sequencing: quantitative analysis of protein localization">ChemPort</a>&nbsp;|</li><li id="B2"><a name="B2"><!-- . --></a>Lang, P., Yeow, K., Nichols, A. &amp; Scheer, A. <span class="atl"> Cellular imaging in drug discovery</span>. <span class="journalname">Nat. Rev. Drug Discov.</span> <span class="journalnumber">5</span>, 343–356 (<span class="cite-month-year">2006</span>).&nbsp;|&nbsp;<a href="http://www.nature.com.ejproxy.a-star.edu.sg/doifinder/10.1038/nrd2008" class="reftxt" title="Article on Article - Cellular imaging in drug discovery">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.ejproxy.a-star.edu.sg/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=16582878&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - Cellular imaging in drug discovery">PubMed</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD28XjtVaksr0%3D&amp;pissn=1548-7091&amp;pyear=2007&amp;md5=24ee39fcef62be8613d3a43e1c63fb7d" class="reftxt" title="Article on ChemPort - Cellular imaging in drug discovery">ChemPort</a>&nbsp;|</li><li id="B3"><a name="B3"><!-- . --></a>Price, J.H. <i>et al</i>. <span class="atl"> Advances in molecular labeling, high throughput imaging and machine intelligence portend powerful functional cellular biochemistry tools</span>. <span class="journalname">J. Cell. Biochem. Suppl.</span> <span class="journalnumber">39</span>, 194–210 (<span class="cite-month-year">2002</span>).&nbsp;|&nbsp;<a href="http://dx.doi.org.ejproxy.a-star.edu.sg/10.1002/jcb.10448" class="reftxt" title="Advances in molecular labeling, high throughput imaging and machine intelligence portend powerful functional cellular biochemistry tools">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.ejproxy.a-star.edu.sg/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=12552619&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - Advances in molecular labeling, high throughput imaging and machine intelligence portend powerful functional cellular biochemistry tools">PubMed</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD3sXhtFKjur8%3D&amp;pissn=1548-7091&amp;pyear=2007&amp;md5=603fe15e0fdbc1db53da0cc6a234fbcf" class="reftxt" title="Article on ChemPort - Advances in molecular labeling, high throughput imaging and machine intelligence portend powerful functional cellular biochemistry tools">ChemPort</a>&nbsp;|</li><li id="B4"><a name="B4"><!-- . --></a>Zhou, X. &amp; Wong, S.T.C. <span class="atl"> Informatics challenges of high-throughput microscopy</span>. <span class="journalname">IEEE Signal Process. Mag.</span> <span class="journalnumber">23</span>, 63–72 (<span class="cite-month-year">2006</span>).&nbsp;|&nbsp;<a href="http://dx.doi.org.ejproxy.a-star.edu.sg/10.1109/MSP.2006.1628879" class="reftxt" title="Informatics challenges of high-throughput microscopy">Article</a>&nbsp;|</li><li id="B5"><a name="B5"><!-- . --></a>Perlman, Z.E. <i>et al</i>. <span class="atl"> Multidimensional drug profiling by automated microscopy</span>. <span class="journalname">Science</span> <span class="journalnumber">306</span>, 1194–1198 (<span class="cite-month-year">2004</span>).&nbsp;|&nbsp;<a href="http://dx.doi.org.ejproxy.a-star.edu.sg/10.1126/science.1100709" class="reftxt" title="Multidimensional drug profiling by automated microscopy">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.ejproxy.a-star.edu.sg/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=15539606&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - Multidimensional drug profiling by automated microscopy">PubMed</a>&nbsp;|&nbsp;<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?&amp;GWVersion=2&amp;SrcAuth=Nature&amp;SrcApp=Nature&amp;DestLinkType=FullRecord&amp;KeyUT=000225193100053&amp;DestApp=WOS_CPL" class="reftxt" title="Article on ISI - Multidimensional drug profiling by automated microscopy">ISI</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD2cXpsF2ktLo%3D&amp;pissn=1548-7091&amp;pyear=2007&amp;md5=79d2bf1ba0da63c984da474e32f7e816" class="reftxt" title="Article on ChemPort - Multidimensional drug profiling by automated microscopy">ChemPort</a>&nbsp;|</li><li id="B6"><a name="B6"><!-- . --></a>Tanaka, M. <i>et al</i>. <span class="atl"> An unbiased cell morphology-based screen for new, biologically active small molecules</span>. <span class="journalname">PLoS Biol.</span> <span class="journalnumber">3</span>, e128 (<span class="cite-month-year">2005</span>).&nbsp;|&nbsp;<a href="http://dx.doi.org.ejproxy.a-star.edu.sg/10.1371/journal.pbio.0030128" class="reftxt" title="An unbiased cell morphology-based screen for new, biologically active small molecules">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.ejproxy.a-star.edu.sg/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=15799708&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - An unbiased cell morphology-based screen for new, biologically active small molecules">PubMed</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD2MXktlaltrY%3D&amp;pissn=1548-7091&amp;pyear=2007&amp;md5=4cf6859bdcc85ae86a6aa09e6e600fd2" class="reftxt" title="Article on ChemPort - An unbiased cell morphology-based screen for new, biologically active small molecules">ChemPort</a>&nbsp;|</li><li id="B7"><a name="B7"><!-- . --></a>Boland, M.V. &amp; Murphy, R.F. <span class="atl"> A neural network classifier capable of recognizing the patterns of all major subcellular structures in fluorescence microscope images of HeLa cells</span>. <span class="journalname">Bioinformatics</span> <span class="journalnumber">17</span>, 1213–1223 (<span class="cite-month-year">2001</span>).&nbsp;|&nbsp;<a href="http://dx.doi.org.ejproxy.a-star.edu.sg/10.1093/bioinformatics/17.12.1213" class="reftxt" title="A neural network classifier capable of recognizing the patterns of all major subcellular structures in fluorescence microscope images of HeLa cells">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.ejproxy.a-star.edu.sg/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=11751230&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - A neural network classifier capable of recognizing the patterns of all major subcellular structures in fluorescence microscope images of HeLa cells">PubMed</a>&nbsp;|&nbsp;<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?&amp;GWVersion=2&amp;SrcAuth=Nature&amp;SrcApp=Nature&amp;DestLinkType=FullRecord&amp;KeyUT=000173050500015&amp;DestApp=WOS_CPL" class="reftxt" title="Article on ISI - A neural network classifier capable of recognizing the patterns of all major subcellular structures in fluorescence microscope images of HeLa cells">ISI</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:STN:280:DC%2BD38%2FgtFOmsA%3D%3D&amp;pissn=1548-7091&amp;pyear=2007&amp;md5=b139732b72b8c6c52197b779dcb893df" class="reftxt" title="Article on ChemPort - A neural network classifier capable of recognizing the patterns of all major subcellular structures in fluorescence microscope images of HeLa cells">ChemPort</a>&nbsp;|</li><li id="B8"><a name="B8"><!-- . --></a>Conrad, C. <i>et al</i>. <span class="atl"> Automatic identification of subcellular phenotypes on human cell arrays</span>. <span class="journalname">Genome Res.</span> <span class="journalnumber">14</span>, 1130–1136 (<span class="cite-month-year">2004</span>).&nbsp;|&nbsp;<a href="http://dx.doi.org.ejproxy.a-star.edu.sg/10.1101/gr.2383804" class="reftxt" title="Automatic identification of subcellular phenotypes on human cell arrays">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.ejproxy.a-star.edu.sg/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=15173118&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - Automatic identification of subcellular phenotypes on human cell arrays">PubMed</a>&nbsp;|&nbsp;<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?&amp;GWVersion=2&amp;SrcAuth=Nature&amp;SrcApp=Nature&amp;DestLinkType=FullRecord&amp;KeyUT=000221852400015&amp;DestApp=WOS_CPL" class="reftxt" title="Article on ISI - Automatic identification of subcellular phenotypes on human cell arrays">ISI</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD2cXkvFGht7w%3D&amp;pissn=1548-7091&amp;pyear=2007&amp;md5=483e32b91e7badbd37170ddde5095a58" class="reftxt" title="Article on ChemPort - Automatic identification of subcellular phenotypes on human cell arrays">ChemPort</a>&nbsp;|</li><li id="B9"><a name="B9"><!-- . --></a>Neumann, B. <i>et al</i>. <span class="atl"> High-throughput RNAi screening by time-lapse imaging of live human cells</span>. <span class="journalname">Nat. Methods</span> <span class="journalnumber">3</span>, 385–390 (<span class="cite-month-year">2006</span>).&nbsp;|&nbsp;<a href="http://www.nature.com.ejproxy.a-star.edu.sg/doifinder/10.1038/nmeth876" class="reftxt" title="Article on Article - High-throughput RNAi screening by time-lapse imaging of live human cells">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.ejproxy.a-star.edu.sg/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=16628209&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - High-throughput RNAi screening by time-lapse imaging of live human cells">PubMed</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD28XjslSktr8%3D&amp;pissn=1548-7091&amp;pyear=2007&amp;md5=c0606a9a3197aed080130fdfb949367b" class="reftxt" title="Article on ChemPort - High-throughput RNAi screening by time-lapse imaging of live human cells">ChemPort</a>&nbsp;|</li><li id="B10"><a name="B10"><!-- . --></a>Ohya, Y. <i>et al</i>. <span class="atl"> High-dimensional and large-scale phenotyping of yeast mutants</span>. <span class="journalname">Proc. Natl. Acad. Sci. USA</span> <span class="journalnumber">102</span>, 19015–19020 (<span class="cite-month-year">2005</span>).&nbsp;|&nbsp;<a href="http://dx.doi.org.ejproxy.a-star.edu.sg/10.1073/pnas.0509436102" class="reftxt" title="High-dimensional and large-scale phenotyping of yeast mutants">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.ejproxy.a-star.edu.sg/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=16365294&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - High-dimensional and large-scale phenotyping of yeast mutants">PubMed</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD28XivV2itA%3D%3D&amp;pissn=1548-7091&amp;pyear=2007&amp;md5=e86ce99c634c460feaf9e752ce2806fd" class="reftxt" title="Article on ChemPort - High-dimensional and large-scale phenotyping of yeast mutants">ChemPort</a>&nbsp;|</li><li id="B11"><a name="B11"><!-- . --></a>Gasparri, F., Mariani, M., Sola, F. &amp; Galvani, A. <span class="atl"> Quantification of the proliferation index of human dermal fibroblast cultures with the ArrayScan high-content screening reader</span>. <span class="journalname">J. Biomol. Screen.</span> <span class="journalnumber">9</span>, 232–243 (<span class="cite-month-year">2004</span>).&nbsp;|&nbsp;<a href="http://dx.doi.org.ejproxy.a-star.edu.sg/10.1177/1087057103262836" class="reftxt" title="Quantification of the proliferation index of human dermal fibroblast cultures with the ArrayScan high-content screening reader">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.ejproxy.a-star.edu.sg/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=15146854&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - Quantification of the proliferation index of human dermal fibroblast cultures with the ArrayScan high-content screening reader">PubMed</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD2cXjvFOkt7Y%3D&amp;pissn=1548-7091&amp;pyear=2007&amp;md5=74163cee572719285f86027def25306d" class="reftxt" title="Article on ChemPort - Quantification of the proliferation index of human dermal fibroblast cultures with the ArrayScan high-content screening reader">ChemPort</a>&nbsp;|</li><li id="B12"><a name="B12"><!-- . --></a>Giuliano, K.A. <i>et al</i>. <span class="atl"> Systems cell biology knowledge created from high content screening</span>. <span class="journalname">Assay Drug Dev. Technol.</span> <span class="journalnumber">3</span>, 501–514 (<span class="cite-month-year">2005</span>).&nbsp;|&nbsp;<a href="http://dx.doi.org.ejproxy.a-star.edu.sg/10.1089/adt.2005.3.501" class="reftxt" title="Systems cell biology knowledge created from high content screening">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.ejproxy.a-star.edu.sg/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=16305307&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - Systems cell biology knowledge created from high content screening">PubMed</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD2MXht1GisLvM&amp;pissn=1548-7091&amp;pyear=2007&amp;md5=baefe7814f9a82a8d16c8b47075c26c9" class="reftxt" title="Article on ChemPort - Systems cell biology knowledge created from high content screening">ChemPort</a>&nbsp;|</li><li id="B13"><a name="B13"><!-- . --></a>Vapnik, V.N. <span class="booktitle">Statistical Learning Theory</span>. (John Wiley &amp; Sons, New York, <span class="cite-month-year">1998</span>).</li><li id="B14"><a name="B14"><!-- . --></a>Duda, R.O., Hart, P.E. &amp; Stork, D.G. <span class="booktitle">Pattern Classification</span> 2<sup>nd</sup> edn. (John Wiley &amp; Sons, New York, <span class="cite-month-year">2001</span>).</li><li id="B15"><a name="B15"><!-- . --></a>Guyon, I., Weston, J., Barnhill, S. &amp; Vapnik, V. <span class="atl"> Gene selection for cancer classification using support vector machines</span>. <span class="journalname">Mach. Learn.</span> <span class="journalnumber">46</span>, 389–422 (<span class="cite-month-year">2002</span>).&nbsp;|&nbsp;<a href="http://dx.doi.org.ejproxy.a-star.edu.sg/10.1023/A:1012487302797" class="reftxt" title="Gene selection for cancer classification using support vector machines">Article</a>&nbsp;|</li><li id="B16"><a name="B16"><!-- . --></a>Haralick, R.M. <span class="atl"> Statistical and structural approaches to texture</span>. <span class="journalname">Proc. IEEE</span> <span class="journalnumber">67</span>, 786–804 (<span class="cite-month-year">1979</span>).</li><li id="B17"><a name="B17"><!-- . --></a>Teh, C.H. &amp; Chin, R.T. <span class="atl"> On image analysis by the methods of moments</span>. <span class="journalname">IEEE Trans. Pattern Anal. Mach. Intell.</span> <span class="journalnumber">10</span>, 496–513 (<span class="cite-month-year">1988</span>).&nbsp;|&nbsp;<a href="http://dx.doi.org.ejproxy.a-star.edu.sg/10.1109/34.3913" class="reftxt" title="On image analysis by the methods of moments">Article</a>&nbsp;|</li><li id="B18"><a name="B18"><!-- . --></a>Lundholt, B.K., Scudder, K.M. &amp; Pagliaro, L. <span class="atl"> A simple technique for reducing edge effect in cell-based Assays</span>. <span class="journalname">J. Biomol. Screen.</span> <span class="journalnumber">8</span>, 566–570 (<span class="cite-month-year">2003</span>).&nbsp;|&nbsp;<a href="http://dx.doi.org.ejproxy.a-star.edu.sg/10.1177/1087057103256465" class="reftxt" title="A simple technique for reducing edge effect in cell-based Assays">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.ejproxy.a-star.edu.sg/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=14567784&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - A simple technique for reducing edge effect in cell-based Assays">PubMed</a>&nbsp;|&nbsp;<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?&amp;GWVersion=2&amp;SrcAuth=Nature&amp;SrcApp=Nature&amp;DestLinkType=FullRecord&amp;KeyUT=000185924700009&amp;DestApp=WOS_CPL" class="reftxt" title="Article on ISI - A simple technique for reducing edge effect in cell-based Assays">ISI</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD3sXptVChs7w%3D&amp;pissn=1548-7091&amp;pyear=2007&amp;md5=4aa1cbf236e208646b927a41c1d01599" class="reftxt" title="Article on ChemPort - A simple technique for reducing edge effect in cell-based Assays">ChemPort</a>&nbsp;|</li><li id="B19"><a name="B19"><!-- . --></a>Malo, N., Hanley, J.A., Cerquozzi, S., Pelletier, J. &amp; Nadon, R. <span class="atl"> Statistical practice in high-throughput screening data analysis</span>. <span class="journalname">Nat. Biotechnol.</span> <span class="journalnumber">24</span>, 167–175 (<span class="cite-month-year">2006</span>).&nbsp;|&nbsp;<a href="http://www.nature.com.ejproxy.a-star.edu.sg/doifinder/10.1038/nbt1186" class="reftxt" title="Article on Article - Statistical practice in high-throughput screening data analysis">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.ejproxy.a-star.edu.sg/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=16465162&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - Statistical practice in high-throughput screening data analysis">PubMed</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD28XhtFGqs70%3D&amp;pissn=1548-7091&amp;pyear=2007&amp;md5=3d3fd56deb7399d0a8040b7639ee6d5f" class="reftxt" title="Article on ChemPort - Statistical practice in high-throughput screening data analysis">ChemPort</a>&nbsp;|</li><li id="B20"><a name="B20"><!-- . --></a>Hastie, T., Tibshirani, R. &amp; Friedman, J. <span class="booktitle">The Elements of Statistical Learning: Data Mining, Inference, and Prediction</span>. (Springer-Verlag, New York, <span class="cite-month-year">2001</span>).</li><li id="B21"><a name="B21"><!-- . --></a>Yeung, T.K., Germond, C., Chen, X. &amp; Wang, Z. <span class="atl"> The mode of action of taxol: apoptosis at low concentration and necrosis at high concentration</span>. <span class="journalname">Biochem. Biophys. Res. Commun.</span> <span class="journalnumber">263</span>, 398–404 (<span class="cite-month-year">1999</span>).&nbsp;|&nbsp;<a href="http://dx.doi.org.ejproxy.a-star.edu.sg/10.1006/bbrc.1999.1375" class="reftxt" title="The mode of action of taxol: apoptosis at low concentration and necrosis at high concentration">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.ejproxy.a-star.edu.sg/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=10491305&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - The mode of action of taxol: apoptosis at low concentration and necrosis at high concentration">PubMed</a>&nbsp;|&nbsp;<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?&amp;GWVersion=2&amp;SrcAuth=Nature&amp;SrcApp=Nature&amp;DestLinkType=FullRecord&amp;KeyUT=000083016400024&amp;DestApp=WOS_CPL" class="reftxt" title="Article on ISI - The mode of action of taxol: apoptosis at low concentration and necrosis at high concentration">ISI</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DyaK1MXlvFantL0%3D&amp;pissn=1548-7091&amp;pyear=2007&amp;md5=d962545ba7bf521ac2179c7e8059c971" class="reftxt" title="Article on ChemPort - The mode of action of taxol: apoptosis at low concentration and necrosis at high concentration">ChemPort</a>&nbsp;|</li><li id="B22"><a name="B22"><!-- . --></a>Yoo, C.B. &amp; Jones, P.A. <span class="atl"> Epigenetic therapy of cancer: past, present and future</span>. <span class="journalname">Nat. Rev. Drug Discov.</span> <span class="journalnumber">5</span>, 37–50 (<span class="cite-month-year">2006</span>).&nbsp;|&nbsp;<a href="http://www.nature.com.ejproxy.a-star.edu.sg/doifinder/10.1038/nrd1930" class="reftxt" title="Article on Article - Epigenetic therapy of cancer: past, present and future">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.ejproxy.a-star.edu.sg/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=16485345&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - Epigenetic therapy of cancer: past, present and future">PubMed</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD28Xmt1yqsA%3D%3D&amp;pissn=1548-7091&amp;pyear=2007&amp;md5=9ec9ccd592fd2cc370019673a6c37e75" class="reftxt" title="Article on ChemPort - Epigenetic therapy of cancer: past, present and future">ChemPort</a>&nbsp;|</li><li id="B23"><a name="B23"><!-- . --></a>Perlman, Z.E., Mitchison, T.J. &amp; Mayer, T.U. <span class="atl"> High-content screening and profiling of drug activity in an automated centrosome-duplication assay</span>. <span class="journalname">ChemBioChem</span> <span class="journalnumber">6</span>, 145–151 (<span class="cite-month-year">2005</span>).&nbsp;|&nbsp;<a href="http://dx.doi.org.ejproxy.a-star.edu.sg/10.1002/cbic.200400266" class="reftxt" title="High-content screening and profiling of drug activity in an automated centrosome-duplication assay">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.ejproxy.a-star.edu.sg/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=15568197&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - High-content screening and profiling of drug activity in an automated centrosome-duplication assay">PubMed</a>&nbsp;|</li><li id="B24"><a name="B24"><!-- . --></a>Bollag, D.M. <i>et al</i>. <span class="atl"> Epothilones, a new class of microtubule-stabilizing agents with a taxol-like mechanism of action</span>. <span class="journalname">Cancer Res.</span> <span class="journalnumber">55</span>, 2325–2333 (<span class="cite-month-year">1995</span>).&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.ejproxy.a-star.edu.sg/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=7757983&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - Epothilones, a new class of microtubule-stabilizing agents with a taxol-like mechanism of action">PubMed</a>&nbsp;|&nbsp;<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?&amp;GWVersion=2&amp;SrcAuth=Nature&amp;SrcApp=Nature&amp;DestLinkType=FullRecord&amp;KeyUT=A1995RA14500021&amp;DestApp=WOS_CPL" class="reftxt" title="Article on ISI - Epothilones, a new class of microtubule-stabilizing agents with a taxol-like mechanism of action">ISI</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DyaK2MXlvFeqsr0%3D&amp;pissn=1548-7091&amp;pyear=2007&amp;md5=3a7a51f549902c738691e1878f56858e" class="reftxt" title="Article on ChemPort - Epothilones, a new class of microtubule-stabilizing agents with a taxol-like mechanism of action">ChemPort</a>&nbsp;|</li><li id="B25"><a name="B25"><!-- . --></a>Panda, D., Rathinasamy, K., Santra, M.K. &amp; Wilson, L. <span class="atl"> Kinetic suppression of microtubule dynamic instability by griseofulvin: implications for its possible use in the treatment of cancer</span>. <span class="journalname">Proc. Natl. Acad. Sci. USA</span> <span class="journalnumber">102</span>, 9878–9883 (<span class="cite-month-year">2005</span>).&nbsp;|&nbsp;<a href="http://dx.doi.org.ejproxy.a-star.edu.sg/10.1073/pnas.0501821102" class="reftxt" title="Kinetic suppression of microtubule dynamic instability by griseofulvin: implications for its possible use in the treatment of cancer">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.ejproxy.a-star.edu.sg/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=15985553&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - Kinetic suppression of microtubule dynamic instability by griseofulvin: implications for its possible use in the treatment of cancer">PubMed</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD2MXmsFaru78%3D&amp;pissn=1548-7091&amp;pyear=2007&amp;md5=073aaa41e5e6881c2c086002e7b8b5ab" class="reftxt" title="Article on ChemPort - Kinetic suppression of microtubule dynamic instability by griseofulvin: implications for its possible use in the treatment of cancer">ChemPort</a>&nbsp;|</li><li id="B26"><a name="B26"><!-- . --></a>Clemons, P.A. <span class="atl"> Complex phenotypic assays in high-throughput screening</span>. <span class="journalname">Curr. Opin. Chem. Biol.</span> <span class="journalnumber">8</span>, 334–338 (<span class="cite-month-year">2004</span>).&nbsp;|&nbsp;<a href="http://dx.doi.org.ejproxy.a-star.edu.sg/10.1016/j.cbpa.2004.04.002" class="reftxt" title="Complex phenotypic assays in high-throughput screening">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.ejproxy.a-star.edu.sg/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=15183333&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - Complex phenotypic assays in high-throughput screening">PubMed</a>&nbsp;|&nbsp;<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?&amp;GWVersion=2&amp;SrcAuth=Nature&amp;SrcApp=Nature&amp;DestLinkType=FullRecord&amp;KeyUT=000222407600017&amp;DestApp=WOS_CPL" class="reftxt" title="Article on ISI - Complex phenotypic assays in high-throughput screening">ISI</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DC%2BD2cXkslWqs7k%3D&amp;pissn=1548-7091&amp;pyear=2007&amp;md5=ef51c3a6fadbc8c411ee1aedba4768f5" class="reftxt" title="Article on ChemPort - Complex phenotypic assays in high-throughput screening">ChemPort</a>&nbsp;|</li><li id="B27"><a name="B27"><!-- . --></a>Dove, A. <span class="atl"> Drug screening-beyond the bottleneck</span>. <span class="journalname">Nat. Biotechnol.</span> <span class="journalnumber">17</span>, 859–863 (<span class="cite-month-year">1999</span>).&nbsp;|&nbsp;<a href="http://www.nature.com.ejproxy.a-star.edu.sg/doifinder/10.1038/12845" class="reftxt" title="Article on Article - Drug screening-beyond the bottleneck">Article</a>&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.ejproxy.a-star.edu.sg/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=10471925&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - Drug screening-beyond the bottleneck">PubMed</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:CAS:528:DyaK1MXlslOgtL8%3D&amp;pissn=1548-7091&amp;pyear=2007&amp;md5=2f9edea1d7288cb746bb72760854d66d" class="reftxt" title="Article on ChemPort - Drug screening-beyond the bottleneck">ChemPort</a>&nbsp;|</li><li id="B28"><a name="B28"><!-- . --></a>Maciag, K. <i>et al</i>. <span class="atl"> Systems-level analyses identify extensive coupling among gene expression machines</span>. <span class="journalname">Mol. Syst. Biol. [online]</span> <span class="journalnumber">2</span>, 0003 (<span class="cite-month-year">2006</span>) (doi:doi: 10.1038/msb4100045).</li><li id="B29"><a name="B29"><!-- . --></a>Hanley, J.A. &amp; McNeil, B.J. <span class="atl"> The meaning and use of the area under a receiver operating characteristic (ROC) curve</span>. <span class="journalname">Radiology</span> <span class="journalnumber">143</span>, 29–36 (<span class="cite-month-year">1982</span>).&nbsp;|&nbsp;<a href="http://www.ncbi.nlm.nih.gov.ejproxy.a-star.edu.sg/entrez/query.fcgi?holding=npg&amp;cmd=Retrieve&amp;db=PubMed&amp;list_uids=7063747&amp;dopt=Abstract" class="reftxt" title="Article on PubMed - The meaning and use of the area under a receiver operating characteristic (ROC) curve">PubMed</a>&nbsp;|&nbsp;<a href="http://links.isiglobalnet2.com/gateway/Gateway.cgi?&amp;GWVersion=2&amp;SrcAuth=Nature&amp;SrcApp=Nature&amp;DestLinkType=FullRecord&amp;KeyUT=A1982NG95400006&amp;DestApp=WOS_CPL" class="reftxt" title="Article on ISI - The meaning and use of the area under a receiver operating characteristic (ROC) curve">ISI</a>&nbsp;|&nbsp;<a href="http://chemport.cas.org/cgi-bin/sdcgi?APP=ftslink&amp;action=reflink&amp;origin=npg&amp;version=1.0&amp;coi=1:STN:280:Bi2C2M7oslc%3D&amp;pissn=1548-7091&amp;pyear=2007&amp;md5=fcc72cd25a092d3fe97371df884e6cdd" class="reftxt" title="Article on ChemPort - The meaning and use of the area under a receiver operating characteristic (ROC) curve">ChemPort</a>&nbsp;|</li></ol><table xmlns="" border="0" cellpadding="0" cellspacing="0" width="470"><tbody><tr><td rowspan="2" height="20"><img alt="" src="spacer.gif" height="20" width="1"></td><td align="right"><img alt="" src="arrow_black_up.gif" height="3" width="5">&nbsp;<a class="top" href="#top">Top</a></td></tr><tr><td align="right" height="7"><img alt="" src="spacer.gif" height="7" width="1"></td></tr><tr><td colspan="2" height="1"><img alt="" src="spacer_dotted.gif" height="1" width="470"></td></tr><tr><td colspan="2" height="20"><img alt="" src="spacer.gif" height="20" width="1"></td></tr></tbody></table><a xmlns="" name="Ack"></a><span xmlns="" class="heading3">Acknowledgments</span><br xmlns=""><span xmlns="" class="articletext">We thank S. Johnson from the Texas Advanced Computing Center for high-performance computing technical assistance, and all members of the Altschuler and Wu lab at the UT Southwestern Medical Center for stimulating discussions. This research was supported by the Endowed Scholars program at UT Southwestern Medical Center and the Welch Foundation (I-1619, I-1644).</span><br xmlns=""><br xmlns=""><p xmlns="" class="articletext"><b>Competing interests statement:</b>&nbsp;
The authors declare
				
				 no competing financial interests.</p></td></tr></tbody></table></td><td width="15"><img alt="" src="spacer.gif" height="1" width="15"></td><td valign="top" width="130"><table xmlns="" border="0" cellpadding="0" cellspacing="0" width="130"><tbody><tr><td><img alt="" src="spacer.gif" height="34" width="1"></td></tr><tr><td colspan="2" height="22"><img alt="" src="spacer.gif" height="22" width="1"></td></tr></tbody></table>
<!-- Molecular cloning ad at top of right nav -->


<table style="margin: 0px 0px 8px;" border="0" cellpadding="0" cellspacing="0" width="130">
<tbody><tr>
<td class="sponsors-td" align="left" valign="top">
<a href="http://ad.doubleclick.net/clk;16138647;8858766;f?http://www.molecularcloning.com/members" target="_top">
<img src="molecular_cloning.gif" alt="Molecularcloning.com" style="border: 0px none ;" height="54" width="130">
</a>
</td>
</tr>
</tbody></table>

<!-- end Molecular cloning ad -->
<table xmlns="" border="0" cellpadding="0" cellspacing="0" width="130"><tbody><tr><td colspan="2" class="whitesml" align="center" bgcolor="#6e83a0" height="20"><b>FULL  TEXT</b></td></tr><tr><td colspan="2" class="blacksml2" align="center" bgcolor="#d1e1ea" height="17" valign="middle"><table border="0" cellpadding="1" cellspacing="0" width="106"><tbody><tr><td width="3"><img alt="" src="arrow_black_prev.gif" border="0" height="5" width="3"></td><td align="center" width="100"><a class="breadcrumblink" href="http://www.nature.com.ejproxy.a-star.edu.sg/nmeth/journal/v4/n5/full/nmeth1035.html">
									Previous							
								</a><a class="breadcrumblink" href="http://www.nature.com.ejproxy.a-star.edu.sg/nmeth/journal/v4/n5/full/nmeth0507-455.html">
										 | 
																		
									Next							
								</a></td><td height="10" width="3"><img alt="" src="arrow_black.gif" height="5" width="3"></td></tr></tbody></table></td></tr><tr><td colspan="2"><img alt="" src="spacer_003.gif" height="1" width="130"></td></tr><tr><td class="blacksml2" align="center" bgcolor="#d1e1ea" height="15" width="15"><img alt="" src="arrow_black.gif" border="0" height="5" width="3"></td><td class="blacksml2" bgcolor="#d1e1ea" height="17" width="115"><a href="http://www.nature.com.ejproxy.a-star.edu.sg/nmeth/journal/v4/n5/index.html" class="breadcrumblink">Table of contents</a></td></tr><tr><td colspan="2" class="blacksml2" align="center" height="15"><table border="0" cellpadding="1" cellspacing="0" width="130"><tbody><tr><td bgcolor="#d1e1ea"><table border="0" cellpadding="0" cellspacing="0" width="128"><tbody><tr><td colspan="2" align="center" bgcolor="#ffffff"><img alt="" src="spacer.gif" height="5" width="1"></td></tr><tr><td align="center" bgcolor="#ffffff" width="23"><img alt="Download PDF" src="icon_pdf.gif" height="14" width="14"></td><td bgcolor="#ffffff" height="14" width="105"><a class="breadcrumblink" href="http://www.nature.com.ejproxy.a-star.edu.sg/nmeth/journal/v4/n5/pdf/nmeth1032.pdf">Download PDF</a></td></tr><tr><td colspan="2" align="center" bgcolor="#ffffff" valign="middle"><img alt="" src="spacer.gif" height="6" width="1"></td></tr><tr><td align="center" bgcolor="#ffffff" valign="middle" width="23"><img alt="Send to a friend" src="icon_email.gif" height="10" width="13"></td><td bgcolor="#ffffff" height="12" valign="middle" width="105"><a class="breadcrumblink" href="http://www.nature.com.ejproxy.a-star.edu.sg/foxtrot/svc/mailform?file=/nmeth/journal/v4/n5/full/nmeth1032.html">Send to a friend</a></td></tr>
		
		

 


<tr><td colspan="2" align="center" bgcolor="#ffffff" valign="middle"><img alt="" src="spacer_003.gif" height="5" width="1"></td></tr>
<tr><td align="center" bgcolor="#ffffff" valign="middle" width="23"><img alt="rights and permissions" src="permissions.gif" height="10" width="13"></td><td bgcolor="#ffffff" height="12" valign="middle" width="105"><a class="breadcrumblink" href="http://s100.copyright.com/AppDispatchServlet?publisherName=NPG&amp;publication=Nature+Methods&amp;title=Image-based%20multivariate%20profiling%20of%20drug%20responses%20from%20single%20cells&amp;author=Lit-Hsin%20Loo%20,%20Lani%20F%20Wu%20and%20Steven%20J%20Altschuler%20&amp;contentID=10.1038/nmeth1032&amp;publicationDate=05/01/2007&amp;Vol=4&amp;Issue=5">Rights and permissions</a></td></tr>
<tr><td align="center" bgcolor="#ffffff" valign="middle" width="23"><img alt="Order commercial reprints" src="ocr.gif" height="10" width="13"></td><td bgcolor="#ffffff" height="12" valign="middle" width="105"><a class="breadcrumblink" href="https://s100.copyright.com/AppDispatchServlet?publisherName=NPGR&amp;publication=Nature+Methods&amp;title=Image-based%20multivariate%20profiling%20of%20drug%20responses%20from%20single%20cells&amp;author=Lit-Hsin%20Loo%20,%20Lani%20F%20Wu%20and%20Steven%20J%20Altschuler%20&amp;contentID=10.1038/nmeth1032&amp;publicationDate=05/01/2007&amp;volumeNum=4&amp;issueNum=5&amp;numPages=9&amp;pageNumbers=pp445-453&amp;orderBeanReset=true"> Order commercial reprints</a></td></tr> 

	<tr><td colspan="2" align="center" bgcolor="#ffffff" valign="middle"><img alt="" src="spacer_003.gif" height="5" width="1"></td></tr><tr><td align="center" bgcolor="#ffffff" valign="middle" width="23"><img alt="Save this link" src="connotea.gif" height="10" width="13"></td><td bgcolor="#ffffff" height="12" valign="middle" width="105"><a class="breadcrumblink" href="http://www.connotea.org/add?uri=http://www.nature.com.ejproxy.a-star.edu.sg/nmeth/journal/v4/n5/full/nmeth1032.html">Save this link</a></td></tr><tr><td colspan="2" align="center" bgcolor="#ffffff" valign="middle"><img alt="" src="spacer_003.gif" height="5" width="1"></td></tr></tbody></table></td></tr></tbody></table></td></tr><tr><td colspan="2"><table border="0" cellpadding="0" cellspacing="0" width="130"><tbody><tr><td height="10"><img alt="" src="spacer_003.gif" height="1" width="15"></td><td height="10"><img alt="" src="spacer_003.gif" height="1" width="115"></td></tr><tr><td class="blacksml2" align="center" width="15"><img alt="" src="arrow_black_dwn.gif" border="0" height="3" width="5"></td><td class="blacksml2" height="15" width="115"><a class="breadcrumblink" href="#abstract">Abstract</a></td></tr><tr><td class="blacksml2" align="center" width="15"><img alt="" src="arrow_black_dwn.gif" border="0" height="3" width="5"></td><td class="blacksml2" height="15" width="115"><a class="breadcrumblink" href="#Results">Results</a></td></tr><tr><td class="blacksml2" align="center" width="15"><img alt="" src="arrow_black_dwn.gif" border="0" height="3" width="5"></td><td class="blacksml2" height="15" width="115"><a class="breadcrumblink" href="#Discussion">Discussion</a></td></tr><tr><td class="blacksml2" align="center" width="15"><img alt="" src="arrow_black_dwn.gif" border="0" height="3" width="5"></td><td class="blacksml2" height="15" width="115"><a class="breadcrumblink" href="#Methods">Methods</a></td></tr><tr><td class="blacksml2" align="center" valign="top" width="15"><img alt="" src="arrow_black.gif" border="0" height="5" vspace="4" width="3"></td><td class="blacksml2" height="15" width="115"><a class="breadcrumblink" href="http://www.nature.com.ejproxy.a-star.edu.sg/nmeth/journal/v4/n5/fig_tab/nmeth1032_ft.html">Figures &amp; Tables</a></td></tr><tr><td class="blacksml2" align="center" width="15"><img alt="" src="arrow_black_dwn.gif" border="0" height="3" width="5"></td><td class="blacksml2" height="15" width="115"><a class="breadcrumblink" href="#Ack">Acknowledgments</a></td></tr><tr><td class="blacksml2" align="center" width="15"><img alt="" src="arrow_black_dwn.gif" border="0" height="3" width="5"></td><td class="blacksml2" height="15" width="115"><a class="breadcrumblink" href="#References">References</a></td></tr><tr><td class="blacksml2" align="center" valign="top" width="15"><img alt="" src="arrow_black.gif" border="0" height="5" vspace="4" width="3"></td><td class="blacksml2" height="15" width="115"><a href="http://www.nature.com.ejproxy.a-star.edu.sg/nmeth/journal/v4/n5/suppinfo/nmeth1032_S1.html" class="breadcrumblink">Supplementary info</a></td></tr><tr><td colspan="2" class="blacksml2" align="center"><img alt="" src="spacer.gif" height="5" width="1"></td></tr><tr><td colspan="2" class="blacksml2" align="center"><img alt="" src="spacer_dotted.gif" height="1" width="130"></td></tr><tr><td class="blacksml2" align="center" width="15"><img src="arrow_black_001.gif" alt="" border="0" height="5" width="3"></td><td class="blacksml2" height="15" width="115"><a href="http://www.nature.com.ejproxy.a-star.edu.sg/nmeth/journal/v4/n5/ris/nmeth1032.ris" target="_top" class="breadcrumblink">Export citation</a></td></tr><tr><td class="blacksml2" align="center" width="15"><img src="arrow_black_001.gif" alt="" border="0" height="5" width="3"></td><td class="blacksml2" height="15" width="115"><a href="http://www.nature.com.ejproxy.a-star.edu.sg/nmeth/journal/v4/n5/ris/nmeth1032refs.ris" target="_top" class="breadcrumblink">Export references</a></td></tr><tr><td colspan="2" class="blacksml2" align="center"><img alt="" src="spacer.gif" height="5" width="1"></td></tr>
			 <tr><td class="blacksml2" align="center" width="15"><img src="arrow_black.gif" alt="" border="0" height="5" width="3"></td><td class="blacksml2" height="15" width="115">
<a class="" href="http://sherlock.nature.com.ejproxy.a-star.edu.sg/arls/Query?prod=NMETH&amp;query=http%3A%2F%2Fxml.nature.com%2Fnmeth%2Fjournal%2Fv4%2Fn5%2Fxml%2Fnmeth1032.xml&amp;type=AF&amp;src_title=Image-based+multivariate+profiling+of+drug+responses+from+single+cells&amp;src_jrnl=Nature+Methods&amp;src_vol=4&amp;src_pp=445-453&amp;src_date=2007&amp;target=popup">More articles like this</a></td></tr><tr><td colspan="2" class="blacksml2" align="center" height="5"><img alt="" src="spacer_004.gif" height="5" width="1"></td></tr><tr><td colspan="2" class="blacksml2" align="center"><img alt="" src="spacer_dotted.gif" height="1" width="130"></td></tr></tbody></table></td></tr></tbody></table><table xmlns="" border="0" cellpadding="0" cellspacing="0" width="130"><tbody><tr><td>
		
<div class="box" id="natjob">
<h2 class="natjob"><a href="http://www.nature.com.ejproxy.a-star.edu.sg/naturejobs/"><span class="first">nature</span><span class="hidden"> </span><span class="second">jobs</span></a></h2>

	<ul class="natjob">

	<li><h3 class="first"><a href="http://nature.com.ejproxy.a-star.edu.sg/naturejobs/science/jobs/55648">Postdoctoral Research Assistant in Vascular Biology</a></h3>
	<ul>
	<li>Royal Veterinary College</li>
	<li>London United Kingdom</li>
	</ul>
	</li>

			<li><h3><a href="http://nature.com.ejproxy.a-star.edu.sg/naturejobs/science/jobs/53990">Entry-Level Bio-Manufacturing Technician: Be Part of the New Wave of Talent! - Glendale, CA</a></h3>
		<ul>
		<li>KSR</li>
		<li>Woodland Hills, CA, USA</li>
		</ul>
		</li>
	</ul>


<div class="natjobmore"><a href="http://www.nature.com.ejproxy.a-star.edu.sg/naturejobs/science/jobs">More science jobs</a></div>
<!--
<div class="natjobmore"><a href="/naturejobs/science/jobs/new">Post a job for free</a></div>
-->
</div>

<form method="post" action="http://www.biocompare.com/nature/search.asp" name="search">
<table bgcolor="#000000" border="0" cellpadding="1" cellspacing="0" width="100%">
<tbody><tr><td valign="top">
<table bgcolor="#ffffff" border="0" cellpadding="0" cellspacing="0" width="100%">
<tbody><tr><td colspan="3"><img src="spacer_005.gif" alt="" height="3" width="5"></td></tr>
<tr><td><img src="spacer_005.gif" alt="" height="1" width="5"></td>
<td valign="top"><img src="logo.gif" alt="natureproducts" border="0" height="16" width="88"></td>
<td bgcolor="#ffffff"><img src="spacer_005.gif" alt="" height="1" width="5"></td></tr>
<tr><td colspan="3"><img src="spacer_005.gif" alt="" height="2" width="1"></td></tr>
<tr><td colspan="3" bgcolor="#000000" valign="top"><img src="spacer_005.gif" alt="" border="0" height="1" width="1"></td></tr>
<tr><td colspan="3" bgcolor="#f3f2f2"><img src="spacer_005.gif" alt="" height="9" width="5"></td></tr>
<tr><td bgcolor="#f3f2f2"><img src="spacer_005.gif" alt="" height="1" width="5"></td>
<td bgcolor="#f3f2f2" valign="top">
<p class="prdnorm">Search buyers guide:<br><br>
<input value="" name="search" size="10" type="text">
<input src="go.gif" type="image"></p></td><td bgcolor="#f3f2f2"><img src="spacer_005.gif" alt="" height="1" width="5"></td></tr>
<tr><td colspan="3" bgcolor="#f3f2f2"><img src="spacer_005.gif" alt="" height="9" width="5"></td></tr>

</tbody></table>
</td></tr></tbody></table></form>



<!-- skyscraper ad -->
<div class="ad-rh skyscraper narrow">
<p>ADVERTISEMENT</p>








										



<ilayer id="layer1" style="visibility: hidden;"></ilayer><nolayer><iframe style="display: none;" src="index_1.html" marginwidth="0" marginheight="0" frameborder="0" height="600" scrolling="no" width="120">&lt;a href="http://ad.doubleclick.net/jump/nmeth.nature.com/article;abr=!ie4;abr=!ie5;type=af;artid=nmeth1032;date=20070401;issue=5;pos=right;fidid=0;posid=0;plaid=0;sz=120x600;ptile=2;ord=1212645044100?"&gt;&lt;img src="http://ad.doubleclick.net/ad/nmeth.nature.com/article;abr=!ie4;abr=!ie5;type=af;artid=nmeth1032;date=20070401;issue=5;pos=right;fidid=0;posid=0;plaid=0;sz=120x600;ptile=2;ord=1212645044100?" alt="Advertisement" /&gt;&lt;/a&gt;</iframe></nolayer>



<layer src="http://ad.doubleclick.net/adl/nmeth.nature.com/article;abr=%21NN2;type=af;artid=nmeth1032;date=20070401;issue=5;pos=right;fidid=0;posid=0;plaid=0;sz=120x600;ptile=2;ord=1212645044100?" visibility="hidden" style="max-width: 120px; height: 600px;"></layer>





</div>
<!-- end skyscraper ad -->
</td></tr></tbody></table></td></tr></tbody></table></td></tr></tbody></table><table border="0" cellpadding="0" cellspacing="0" width="100%"><tbody><tr><td align="center">&nbsp;</td></tr>
<tr><td class="footerline" align="center" height="4"><img alt="" src="spacer.gif" height="4" width="1"></td></tr><tr><td align="center"><table border="0" cellpadding="0" cellspacing="0" width="760"><tbody><tr><td valign="top"><img alt="" src="spacer_002.gif" height="10" width="1"></td></tr><tr><td class="norm" valign="top"><em><b>Nature Methods</b></em><br><span class="blacksmltext">
ISSN:&nbsp;1548-7091<br>
EISSN:&nbsp;1548-7105</span></td></tr><tr><td class="blacknorm" align="left" valign="top" width="615">
<a class="footer" href="http://www.nature.com.ejproxy.a-star.edu.sg/nmeth/index.html">Journal home</a> | <a class="footer" href="http://www.nature.com.ejproxy.a-star.edu.sg/nmeth/journal/v4/n2/index.html">
Current issue
</a> | <a class="footer" href="http://www.nature.com.ejproxy.a-star.edu.sg/nmeth/archive/index.html">Archive</a> | <a class="footer" href="http://www.nature.com.ejproxy.a-star.edu.sg/nmeth/press_release/index.html">Press releases</a> | </td></tr><tr><td valign="top"><img alt="" src="spacer_002.gif" height="10" width="1"></td></tr></tbody></table></td></tr><tr><td class="footerline" align="center" height="2"><img alt="" src="spacer.gif" height="2" width="1"></td></tr><tr><td align="center"><table xmlns="" border="0" cellpadding="0" cellspacing="0" width="760"><tbody><tr><td colspan="3" valign="top"><img alt="" src="spacer_001.gif" height="10" width="1"></td></tr><tr><td align="right" valign="top" width="130"><a href="http://npg.nature.com.ejproxy.a-star.edu.sg/npg/servlet/Content?data=xml/02_welcome.xml&amp;style=xml/02_welcome.xsl"><img alt="Nature Publishing Group, publisher of Nature, and other science journals and reference works" src="npg_footer_logo.gif" border="0" height="26" width="26"></a></td><td align="right" valign="top" width="15"><img alt="" src="spacer_001.gif" height="1" width="15"></td><td class="blacksmltext" valign="middle" width="615"><a href="http://www.nature.com.ejproxy.a-star.edu.sg/common/copyright_statement.html" class="copyright">©2007&nbsp;Nature Publishing Group</a> | <a target="_top" class="privacy" href="http://npg.nature.com.ejproxy.a-star.edu.sg/npg/servlet/Content?data=xml/08_privacy.xml&amp;style=xml/08_privacy.xsl">Privacy policy</a></td></tr><tr><td colspan="3" valign="top"><img alt="" src="spacer_001.gif" height="10" width="1"></td></tr></tbody></table></td></tr></tbody></table>
<!--extra code for 468 x 60 ad-->
<layer src="http://ad.doubleclick.net/adl/nmeth.nature.com/article;type=;artid=nmeth1032;date=20070401;pos=right;lg=no;sz=468x60;ord=1212645044100?" visibility="hidden" width="468" height="60"></layer>

<!--end of extra code--></div></div></body>
</html>
